The bacterial community is reported to vary between case and control for a number of diseases [@Turnbaugh2008; @Baxter2016; @Sze2015; @Bonfili2017]. However, for diseases like obesity, the taxa identified have varied widely depending on the study [@Turnbaugh2008; @Zupancic2012]. Although some of the differences could be attributed to study design, error introduced during the 16S rRNA gene sequencing workflow could also be another source of variation. A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, PCR, and sequencing steps. The preservation and extraction stages of the 16S rRNA gene sequencing workflow have been the most extensively studied [@contamination_Salter2014; @preservation_Song_2016; @storage_Bassis_2017; @review_Kim_2017]. For preservation and extraction stages of the workflow, it has been consistently found that there are errors in the observed bacterial community composition based on the kits used, but that these differences are smaller than the overall biological difference measured between samples with different kits [@preservation_Song_2016; @storage_Bassis_2017]. Since these studies use the same PCR approach while varying preservation or extraction method, the contribution of PCR error to this overall workflow is not as well characterized.


There is a large body of literature that shows there are errors due to primer and number of cycles chosen for the PCR stage of 16S rRNA gene sequencing [@Eckert1991; @Burkardt2000]. Primers have variable region dependent binding affinities which causes an inability to detect specific bacteria (e.g. V1-V3 does not detect *Haemophilus influenzae* and V3-V5 does not detect *Propionibacterium acnes*) [@Meisel2016; @Sze2015 (Table S4)]. Another source of error is the selective amplification of AT-rich over GC-rich sequences which exaggerate the difference between 16S rRNA genes higher in AT versus those higher in GC content [@polz_bias_1998]. Many of these sources of error are made worse as the number of cycles increases [@Wang1996; @Haas2011; @Kebschull2015]. For example, both amplification error and non-specific amplification (e.g. incorrect amplicon size products) also can increase as the number of cycles increases. This will increase the number of Operational Taxonomic Units (OTUs) observed and drastically change the values obtained from commonly used diversity measures [@BautistadelosSantos2016; @Acinas2005]. Additionally, as the number of cycles increases more chimeras can form from an aborted extension step that causes a priming error and subsequent secondary extension [@Haas2011]. These chimeras will artificially increase community diversity by increasing the number of OTUs that are observed [@Haas2011]. In addition to these sources of errors, there also are multiple families of DNA polymerases that have their own error rate and proof reading capacity [@polymerase_Ishino_2014]. 
