\documentclass[12pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{}
  \pretitle{\vspace{\droptitle}}
  \posttitle{}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{amsmath}

\usepackage{mathtools}

\begin{document}

\section{What the Taq? The Influence of Different Hi-Fidelity Taq
Polymerase on 16S rRNA
Sequencing}\label{what-the-taq-the-influence-of-different-hi-fidelity-taq-polymerase-on-16s-rrna-sequencing}

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

\begin{itemize}
\tightlist
\item
  \href{mailto:marcsze@med.umich.edu}{\nolinkurl{marcsze@med.umich.edu}}
\end{itemize}

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

\textbf{Background.}

\textbf{Methods.}

\textbf{Results.}

\textbf{Conclusions.}

\newpage

\subsection{Introduction}\label{introduction}

\newpage

\subsection{Materials \& Methods}\label{materials-methods}

\textbf{\emph{Human and Mock Samples:}} A single fecal sample was
obtained from 4 individuals who were part of the Enterics Research
Investigational Network (ERIN) and the processing and storage of these
samples have been published previously (Seekatz et al., 2016). Clinical
data and other types of meta data were not utilized or accessed for this
study. All samples were extracted using the MOBIO\textsuperscript{TM}
PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The
ZymoBIOMICS\textsuperscript{TM} Microbial Community DNA Standard (Zymo,
CA, USA) was used in this study and is made up of \emph{Pseudomonas
aeruginosa}, \emph{Escherichia coli}, \emph{Salmonella enterica},
\emph{Lactobacillus fermentum}, \emph{Enterococcus faecalis},
\emph{Staphylococcus aureus}, \emph{Listeria monocytogenes}, and
\emph{Bacillus subtilis} at equal genomic DNA abundance
(\url{http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards}).

\textbf{\emph{PCR Protocol:}} The five different high fidelity (HiFi)
Taq DNA polymerase that were tested were AccuPrime\textsuperscript{TM}
(ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion
(ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New
England Biolabs, MA, USA). The PCR cycle conditions were the same for
every primer (Kozich et al., 2013)
(\url{https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md}).
If the HiFi Taq had a specific activation time that was different then 2
minutes that was used instead. The 30 cycle default was used but the
cycle conditions started at 15 and increased by 5 up to 35 cycles and
was used for both fecal and mock samples. The fecal PCR consisted of all
4 samples at 15, 20, 25, 30, and 35 cycles for each Taq (total samples =
100). Although, the mock communities also had 4 replicates used for 15,
20, 25, and 35 cycles, 10 replicates were used for 30 cycles for all Taq
(total samples = 130). For all the mock community samples there was not
enough PCR product at 15 cycles for adequate sequencing.

\textbf{\emph{Sequence Processing:}} The mothur software program was
utilized for all sequence processing steps (Schloss et al., 2009). The
protocol followed was similar to what has been previously publsihed
(Kozich et al., 2013) (\url{https://www.mothur.org/wiki/MiSeq_SOP}). Two
major differences from the stated protocol were the use VSEARCH instead
of UCHIME for chimera detection and the use of the OptiClust algorithmn
instead of average neighbor for Operational Taxonomic Unit (OTU)
generation (Edgar et al., 2011; Rognes et al., 2016; Westcott \&
Schloss, 2017). Sequence error was determined using the seq.error
command on mock samples after chimera removal and classification to the
RDP to remove non-bacterial sequences (Schloss et al., 2009; Cole et
al., 2013; Rognes et al., 2016).

\textbf{\emph{Statistical Analysis:}} All analysis was done with the R
(v 3.4.2) software package (R Core Team, 2017). Data tranformation and
graphing was completed using the tidyverse package (v 1.1.1) and colors
selected using the viridis package (v 0.4.0) (Garnier, 2017; Wickham,
2017). The total number of OTUs were analyzed using an ANOVA with a
tukey post-hoc test. For the fecal samples the data was normalized to
each individual by cycle number to account for the biological variation
between different people. For both error and chimera analysis, samples
were tested using Kruskal-Wallis with a Dunns post-hoc test. Where
applicable correction for multiple comparison utilized the
Benjamini-Hochberg method (Benjamini \& Hochberg, 1995).

\textbf{\emph{Analysis Workflow:}} The total number of OTUs after
sub-sampling was analyzed for both the fecal and mock community samples.
From these observations we wanted to next analyze potential reasons as
to why some of these differences may have occured. First, analysis of
general sequence error rate, number of sequences with an error, and base
subsitution were assessed in the mock community for each Taq. After
assessing these errors, the total number of chimeras was determined
after sequence processing. The fecal samples were analyzed at 4
different sub-sampling levels, 1000, 5000, 10000, and 15000 while the
mock community samples were analysed at 3 levels, 1000, 5000, 10000.

\textbf{\emph{Reproducible Methods:}} The code and analysis can be found
here \url{https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017}.
The raw sequences can be found in the SRA at the following accesssion
number \textbf{need to upload still}.

\newpage

\subsection{Results}\label{results}

\textbf{\emph{The Number of OTUs is Dependent on HiFi Taq Used:}} After
normalization by individual, for each cycle number, we observed that for
fecal samples the number of OTUs identified was dependent upon the HiFi
Taq used and this difference increased as the depth of sub-sampling
increased {[}Figure 1{]}. Lower cycle numbers (15-20) resulted in less
differences between Taq while cycle numbers of 25, 30, and 35 had larger
clearer defined differences {[}Figure 1{]}. Only 35 cycles had HiFi Taq
differences that were significantly different at all sub-sampling levels
(P-value \textless{} 0.0001) {[}Table S1{]}. At sub-sampling depth of
5000 or higher 25 and 30 cycles had HiFi Taq differences (P-value
\textless{} 0.05) {[}Table S1{]}. Using a Tukey post-hoc test only at 35
cycles were significant differences found to be mainly driven by
Platinum being different than all other Taq across sub-sampling depth
(P-value \textless{} 0.05) {[}Table S2{]}.

This Taq dependent difference in the number of OTUs was also observed in
the mock community samples with the same Taq polymerases being high
(Platinum) and low (Accuprime) respectively {[}Figure 2 \& Table S3{]}.
Conversely, differences between HiFi Taq were observed as early as 20
cycles and a sub-sampling depth of 1000 sequences (P-value = 0.002)
{[}Table S3{]}. Using a Tukey post-hoc test differences between Platinum
and the other HiFi Taqs was the major driver of the differences seen at
different cycle numbers and sub-sampling depths {[}Table S4{]}. Both
fecal and mock samples consistently showed that across sub-sampling
depth and cycle number the lowest number of OTUs identified was from
Accuprime\textsuperscript{TM} while the highest was from Platinum for
both fecal and mock samples {[}Figure 1 \& 2{]}.

\textbf{\emph{Per Base Error Rate is Dependent on both Taq and Cycle
Number Used:}} Differences by HiFi Taq in the median average per base
error varied without a clear pattern across sub-sampling depth {[}{]}.
Generally, the highest values were for the Kappa HiFI Taq {[}Figure
3A{]}. Sub-sampling depth seems to have little effect on this rate with
both 5000 and 10000 sub-sampled sequences showing similar results
{[}Figure 3B \& C{]}. Generally, there were small differences between
the various HiFi Taq at lower cycle number but larger differences at
higher cycle number {[}Figure 3B \& C{]}. Platinum HiFi Taq consistently
had the highes median average per base error rate while Phusion and
Accuprime had the lowest {[}Figure 3B \& C{]}. These differences though
were realtively small between the different HiFi Taq {[}Figure 3{]}. The
total seqeunces with at least one error matched the median average per
base error except for the Accuprime HiFi Taq {[}Figure S1{]}. Although
it had the lowest per base error rate it had either the largest or
second largest number of sequences with at least one error regardless of
cycle number of sub-sampling depth {[}Figure S1{]}. Investigation of
whether there were Taq dependent effects on base subsitution found that
there was no clear bias and this was indpendent of sub sampling depth
{[}Figure S2-S4{]}. Further, the variation in subsitution error seems to
reduce as the sequencing depth increases {[}Figure S2-S4{]}.

\textbf{\emph{Chimeric Sequences Corelate with OTUs and are HiFi Taq
Dependent:}} After chimera removal using VSEARCH and removal of
sequences that did not classify as bacteria we assessed the percentage
of sequences that were still chimeric within our mock community. From
this we observed that Platinum HiFi conssitently had the highest
percentage of chimeric reads regardless of amplification cycle number
and sub-sampling depth {[}Figure 4{]}. For all Taqs a positive
correlation was observed between chimeric sequences and number of OTUs
and this correlation was strongest for Platinum and Phusion HiFi Taq
{[}Figure 5{]}. In general the correlations between the number of OTUs
and chimeric sequences became stronger as sub-sampling depth increased
{[}Figure 5{]}.

\newpage

\subsection{Discussion}\label{discussion}

\newpage

\subsection{Conclusion}\label{conclusion}

\newpage

\subsection{Acknowledgements}\label{acknowledgements}

The authors would like to thank all the study participants ERIN whose
samples were utilized. We would also like to thank Judy Opp and April
Cockburn for their effort in sequencing the samples as part of the
Microbiome Core Facility at the University of Michigan. Salary support
for Marc Sze came from the Canadian Institue of Health Research and the
Michigan Institute for Clinical and Health Research Postdoctoral
Translational Scholar Program.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-benjamini_controlling_1995}{}
Benjamini Y., Hochberg Y. 1995. Controlling the false discovery rate: A
practical and powerful approach to multiple testing. \emph{Journal of
the Royal Statistical Society. Series B (Methodological)} 57:289--300.

\hypertarget{ref-rdp_Cole_2013}{}
Cole JR., Wang Q., Fish JA., Chai B., McGarrell DM., Sun Y., Brown CT.,
Porras-Alfaro A., Kuske CR., Tiedje JM. 2013. Ribosomal database
project: Data and tools for high throughput rRNA analysis. \emph{Nucleic
Acids Research} 42:D633--D642. DOI:
\href{https://doi.org/10.1093/nar/gkt1244}{10.1093/nar/gkt1244}.

\hypertarget{ref-uchime_Edgar_2011}{}
Edgar RC., Haas BJ., Clemente JC., Quince C., Knight R. 2011. UCHIME
improves sensitivity and speed of chimera detection.
\emph{Bioinformatics} 27:2194--2200. DOI:
\href{https://doi.org/10.1093/bioinformatics/btr381}{10.1093/bioinformatics/btr381}.

\hypertarget{ref-viridis_citation_2017}{}
Garnier S. 2017. \emph{Viridis: Default color maps from 'matplotlib'}.

\hypertarget{ref-protocol_Kozich_2013}{}
Kozich JJ., Westcott SL., Baxter NT., Highlander SK., Schloss PD. 2013.
Development of a dual-index sequencing strategy and curation pipeline
for analyzing amplicon sequence data on the MiSeq illumina sequencing
platform. \emph{Applied and Environmental Microbiology} 79:5112--5120.
DOI: \href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-r_citation_2017}{}
R Core Team. 2017. \emph{R: A language and environment for statistical
computing}. Vienna, Austria: R Foundation for Statistical Computing.

\hypertarget{ref-vsearch_Rognes_2016}{}
Rognes T., Flouri T., Nichols B., Quince C., Mahé F. 2016. VSEARCH: A
versatile open source tool for metagenomics. \emph{PeerJ} 4:e2584. DOI:
\href{https://doi.org/10.7717/peerj.2584}{10.7717/peerj.2584}.

\hypertarget{ref-mothur_schloss_2009}{}
Schloss PD., Westcott SL., Ryabin T., Hall JR., Hartmann M., Hollister
EB., Lesniewski RA., Oakley BB., Parks DH., Robinson CJ., Sahl JW.,
Stres B., Thallinger GG., Horn DJV., Weber CF. 2009. Introducing mothur:
Open-source, platform-independent, community-supported software for
describing and comparing microbial communities. \emph{Applied and
Environmental Microbiology} 75:7537--7541. DOI:
\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-erin_seekatz_2016}{}
Seekatz AM., Rao K., Santhosh K., Young VB. 2016. Dynamics of the fecal
microbiome in patients with recurrent and nonrecurrent clostridium
difficile infection. \emph{Genome Medicine} 8. DOI:
\href{https://doi.org/10.1186/s13073-016-0298-8}{10.1186/s13073-016-0298-8}.

\hypertarget{ref-opticlust_Westcott_2017}{}
Westcott SL., Schloss PD. 2017. OptiClust, an improved method for
assigning amplicon-based sequence data to operational taxonomic units.
\emph{mSphere} 2:e00073--17. DOI:
\href{https://doi.org/10.1128/mspheredirect.00073-17}{10.1128/mspheredirect.00073-17}.

\hypertarget{ref-tidyverse_2017}{}
Wickham H. 2017. \emph{Tidyverse: Easily install and load 'tidyverse'
packages}.

\newpage

\textbf{Table 1: }

\footnotesize

\normalsize
\newpage

\textbf{Table 2: }

\footnotesize

\normalsize
\newpage

\textbf{Figure 1: .}

\textbf{Figure 2: .}

\textbf{Figure 3: .}

\textbf{Figure 4: .}

\textbf{Figure 5: .}

\textbf{Figure 6: .}

\textbf{Figure 7: .}

\newpage

\textbf{Figure S1: .}

\textbf{Figure S2: .}

\textbf{Figure S3: .}

\textbf{Figure S4: .}

\textbf{Figure S5: .}

\textbf{Figure S6: .}

\textbf{Figure S7: .}

\newpage


\end{document}
