---
title: ''
csl: peerj.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

# Read in fecal sample number of OTU results
fs_anova_summary <- read_csv("../data/process/tables/fecal_overall_anova_results.csv")
fs_tukey_summary <- read_csv("../data/process/tables/fecal_overall_tukey_results.csv")

# Read in mock sample number of OTU results
mock_anova_summary <- read_csv("../data/process/tables/mock_overall_anova_results.csv")
mock_tukey_summary <- read_csv("../data/process/tables/mock_overall_tukey_results.csv")

# Read in error rate results
krsukal_error_rate <- read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")
dunn_error_rate <- read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")

# Read in sequence error count results
kruskal_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")
dunn_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")

# Read in chimera prevalence results
kruskal_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv")
dunn_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")

```



# What the Taq? The Influence of Different Hi-Fidelity Taq Polymerase on 16S rRNA Sequencing

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** 

**Methods.**

**Results.** 

**Conclusions.** 




\newpage


## Introduction




\newpage

## Materials & Methods

***Human and Mock Samples:***
A single fecal sample was obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN) and the processing and storage of these samples have been published previously [@erin_seekatz_2016]. Clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used in this study and is made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance  (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards).

***PCR Protocol:***
The five different high fidelity Taq DNA polymerase that were tested were AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA). 

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. 


\newpage


## Results

***Insert Sub title here***




\newpage

## Discussion



\newpage

## Conclusion



\newpage



## Acknowledgements

The authors would like to thank all the study participants who were a part of each of the individual studies utilized. We would also like to thank each of the study authors for making their data available for use. Finally we would like to thank the members of the Schloss lab for valuable feed back and proof reading during the formulation of this manuscript.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: **

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}


```

\normalsize
\newpage

**Table 2: **

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}


```

\normalsize
\newpage



**Figure 1: .** 

**Figure 2: .** 

**Figure 3: .**

**Figure 4: .** 

**Figure 5: .** 

**Figure 6: .** 

**Figure 7: .**  


\newpage


**Figure S1: .**

**Figure S2: .** 

**Figure S3: .** 

**Figure S4: .** 

**Figure S5: .** 

**Figure S6: .** 

**Figure S7: .** 

\newpage
