\documentclass[12pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{}
  \pretitle{\vspace{\droptitle}}
  \posttitle{}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{amsmath}

\usepackage{mathtools}

\begin{document}

\section{High Fidelity DNA Polymerase Introduces Bias into 16S rRNA Gene
Sequencing
Results}\label{high-fidelity-dna-polymerase-introduces-bias-into-16s-rrna-gene-sequencing-results}

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

\begin{itemize}
\tightlist
\item
  \href{mailto:marcsze@med.umich.edu}{\nolinkurl{marcsze@med.umich.edu}}
\end{itemize}

\newpage

\linenumbers

\subsection{Abstract}\label{abstract}

\textbf{Background.} Many of the different steps used during 16S rRNA
gene sequencing can introduce bias and by extension change the observed
microbial community results. Although cycle number and high fidelity
(HiFi) DNA polymerase are studied less often, they are still important
sources of bias in this workflow. Here, we critically examine both cycle
number and HiFi DNA polymerase for biases that may influence downstream
bacterial community results.

\textbf{Methods.} DNA from fecal samples (n = 4) were extracted using a
PowerMag DNA extraction kit with a 10 minute bead beating step and
amplified at 15, 20, 25, 30, and 35 cycles using Accuprime, Kappa,
Phusion, Platinum, or Q5 HiFi DNA polymerase. Mock communities
(technical replicates n = 4) consisting of previously isolated whole
genomes of 8 different bacteria were also amplified using the same
approach. First, the number of OTUs (Operational Taxonomic Units) were
examined for both fecal samples and mock communities. Next, Bray-Curtis
index, the error rate, sequence error prevalence, and chimera prevalence
were investigated. Finally, the chimera prevalence correlation with
number of OTUs was assessed.

\textbf{Results.} At 35 cycles there were significant differences
between HiFi DNA polymerase for fecal samples (P-value \textless{}
0.0001). These HiFi dependent differences in the number of OTUs could be
identified as early as 20 cycles in the mock communities (P-value =
0.002). Chimera prevalence varied by HiFi DNA polymerase and these
differences were still observed after chimera removal using VSEARCH.
Additionally, the chimera prevalence had a strong positive correlation
with the number of OTUs and this association was not changed by chimera
removal with VSEARCH.

\textbf{Conclusions.} Due to HiFi DNA polymerase dependent differences
in the number of OTUs and chimera prevalence, common diversity metrics
could have values that are not comparable across studies.

\newpage

\subsection{Introduction}\label{introduction}

This study will specifically address how cycle number and high fidelity
(HiFi) DNA polymerases can bias observed bacterial community results
derived from 16S rRNA gene sequencing. First, it is important to
differentiate between bias, reproducibility, and standardization since
often times these three can be confused and used interchangeably with
each other. Bias can change the observed results in a way that is
reproducible and standardized. For example, if one group uses one brand
of DNA extraction kit for their 16S rRNA gene sequencing, their results
may be biased versus another group not using the same brand kit but
within their group they can still have reproducible results. Therefore,
standardization of 16S rRNA gene sequencing methods to increase
reproducibility can still be problematic due to bias. Determining how
different aspects of a 16S rRNA gene sequencing workflow could bias the
observed results is critical for the interpretation of specific studies
in the borader context of the overall field.

A typical 16S rRNA gene sequencing workflow can be divided into
preservation, extraction, PCR, and sequencing steps. Generally, not
using a preservation media and leaving samples at room temperature has
been shown to cause overgrowth of low abundance members of the fecal
bacterial community {[}ref{]}. Similarily, this overgrowth could still
occur if the preservation media does not adequately inhibit growth
{[}ref{]}. Reports have also shown that changes in specific community
members might occur due to differing susceptibility to freeze thaw
cycles amongst microbes {[}ref{]}. Additionally, reagent contamination
has been shown to add community members and the contribution of these
contaminant members grows larger with lower biomass samples (Salter et
al., 2014). Recent studies have shown that the majority of these biases
due to either preservation or extraction tend to be smaller than the
overall biological signal being measured {[}refs{]}. However, the
contribution of PCR bias to this overall workflow is not well
characterized since these studies use the same PCR approach while
varying preservation or extraction method.

Identifying the biases in the PCR stage of 16S rRNA gene sequencing is
important because a large body of literature shows that there are a
variety of steps during PCR that can change the observed results
{[}refs{]}. Many of these sources of biases are made worse as cycle
number increases {[}ref{]}. For example, the selective amplification of
the dominant sequence can increase the difference between the most and
next abundant sequence, over-emphasizing the actual difference
{[}ref{]}. Both amplification error and non-specific amplification
(e.g.~incorrect amplicon size products) can also increase as cycle
number increases which can drastically change commonly used diversity
measures {[}refs{]}. Additionally, chimeras can form from an aborted
extension followed by subsequent priming error and secondary extension
and even if it has not been explicitly shown to increase with cycle
number it is easy to postulate that this does happen and will also
artificially increase community diversity (Haas et al., 2011).

There are also intrinsic properties to primers and DNA polymerases
chosen that can introduce bias. Primers have variable region dependent
binding affinities for different bacteria and depending on the primer
pair do not detect specific bacteria (e.g.~V1-V3 does not detect
\emph{Haemophilus Influenzae} and V3-V5 does not detect
\emph{Propionibacterium acnes}) {[}refs{]}. Additionally, there are
multiple families of DNA polymerases that have their own error rate and
proof reading capcity (Ishino \& Ishino, 2014). Interestingly, the
influence that these different DNA polymerases can have on the observed
results have not been well studied like some of the other previously
mentioned sources of PCR-based bias.

A recent study found clear differences between normal and high fidelity
(HiFi) DNA polymerase and that optimization of the PCR protocol could
reduce error and chimera generation (Gohl et al., 2016). This study also
found that regardless of DNA polymerase, the number of Operational
Taxonomic Units (OTUs) or taxa generated were not easily reduced using
the authors chosen bioinformatic pipeline (Gohl et al., 2016). It is
natural to extend this line of inquiry and ask if biases in the number
of OTUs and chimeras are also dependent on HiFi DNA polymerases. There
is some reason to think that this may be the case since many of these
HiFi DNA polymerases come from different families (e.g. \emph{Taq}
belongs to the family A polymerases) and may intrinsically have
different error rates that cannot be completely removed with
modifications (Ishino \& Ishino, 2014). In this study, we critically
examine if any of five different HiFi DNA polymerases introduce
significant biases into 16S rRNA gene surveys, if this is a cycle
dependent phenomenon, and whether they can be removed using a standard
bioinformatic pipeline.

\newpage

\subsection{Materials \& Methods}\label{materials-methods}

\textbf{\emph{Human and Mock Samples:}} A single fecal sample was
obtained from 4 individuals who were part of the Enterics Research
Investigational Network (ERIN). The processing and storage of these
samples have been previously published (Seekatz et al., 2016). Other
than confirmation that none of these individuals had a \emph{Clostridium
difficle} infection, clinical data and other types of meta data were not
utilized or accessed for this study. All samples were extracted using
the MOBIO\textsuperscript{TM} PowerMag Microbiome RNA/DNA extraction kit
(now Qiagen, MD, USA). The ZymoBIOMICS\textsuperscript{TM} Microbial
Community DNA Standard (Zymo, CA, USA) was used in this study and is
made up of \emph{Pseudomonas aeruginosa}, \emph{Escherichia coli},
\emph{Salmonella enterica}, \emph{Lactobacillus fermentum},
\emph{Enterococcus faecalis}, \emph{Staphylococcus aureus},
\emph{Listeria monocytogenes}, and \emph{Bacillus subtilis} at equal
genomic DNA abundance
(\url{http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards}).

\textbf{\emph{PCR Protocol:}} The five different HiFi DNA polymerases
that were tested included AccuPrime\textsuperscript{TM} (ThermoFisher,
MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA),
Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA).
The PCR cycle conditions for Platinum and Accuprime followed a
previously published protocol (Kozich et al., 2013)
(\url{https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md}).
The HiFi DNA polymerase activation time was 2 minutes, unless a
different activation was specified. For Kappa and Q5, the protocol
previously published by Gohl and colleagues was used (Gohl et al.,
2016). For Phusion, the company defined conditions were used except for
extension time, where the Accuprime and Platinum settings were used.

Both fecal and mock samples cycle conditions started at 15 and increased
by 5 up to 35 cycles with amplicons used at each 5-step increase for
sequencing. The fecal PCR consisted of all 4 samples at 15, 20, 25, 30,
and 35 cycles for each HiFi DNA polymerase (total samples = 100). The
mock communities had 4 replicates at 15, 20, 25, and 35 cycles and 10
replicates for 30 cycles for all HiFi DNA polymerase (total samples =
130). No mock community sample had enough PCR product at 15 cycles for
adequate 16S rRNA gene sequencing.

\textbf{\emph{Sequence Processing:}} The mothur software program was
utilized for all sequence processing steps (Schloss et al., 2009).
Generally, the protocol followed what has been previously published
(Kozich et al., 2013) (\url{https://www.mothur.org/wiki/MiSeq_SOP}). Two
major differences from the stated protocol were the use of VSEARCH
instead of UCHIME for chimera detection and the use of the OptiClust
algorithm instead of average neighbor for OTU generation at 97\%
similarity (Edgar et al., 2011; Rognes et al., 2016; Westcott \&
Schloss, 2017). Sequence error was determined using the `seq.error'
command on mock samples after chimera removal and classification to the
RDP to remove non-bacterial sequences (Schloss et al., 2009; Cole et
al., 2013; Rognes et al., 2016).

\textbf{\emph{Analysis Workflow:}} The total number of OTUs after
sub-sampling was analyzed for both the fecal and mock community samples.
Cycle dependent affects on Bray-Curtis indices were next assessed for
the fecal samples looking at both overall cycle differences and within
individual differences for the previous cycle (e.g.~20 versus 25, 25
versus 30, etc.). For these community based measures, the fecal samples
were analyzed at 4 different sub-sampling sequence depths (1000, 5000,
10000, and 15000) while the mock community samples were analysed at 3
levels (1000, 5000, 10000). Next, based on these observations we wanted
to analyze potential reasons for these differences. First, analysis of
general sequence error rate, number of sequences with an error, and base
substitution were assessed in the mock community for each HiFi DNA
polymerase. After assessing these errors, the total number of chimeras
was determined after sequence processing and correlations to the number
of OTUs also assessed.

\textbf{\emph{Statistical Analysis:}} All analysis was done with the R
(v 3.4.3) software package (R Core Team, 2017). Data transformation and
graphing was completed using the tidyverse package (v 1.2.1) and colors
selected using the viridis package (v 0.4.1) (Garnier, 2017; Wickham,
2017). Differences in the total number of OTUs were analyzed using an
ANOVA with a tukey post-hoc test. For the fecal samples the data was
normalized to each individual by cycle number to account for the
biological variation between people. Bray-Curtis distance matrices were
generated using mothur after 100 sub-samplings at 1000, 5000, 10000, and
15000 sequence depth. The distance matrix data was analyzed using
PERMANOVA with the vegan package (v 2.4.5) (Oksanen et al., 2017) and
Kruskal-Wallis tests within R. For both error and chimera analysis,
samples were tested using Kruskal-Wallis with a Dunns post-hoc test.
Where applicable correction for multiple comparison utilized the
Benjamini-Hochberg method (Benjamini \& Hochberg, 1995).

\textbf{\emph{Reproducible Methods:}} The code and analysis can be found
here \url{https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017}.
The raw sequences can be found in the SRA at the following accession
number SRP132931.

\newpage

\subsection{Results}\label{results}

\textbf{\emph{The Number of OTUs is Dependent on HiFi DNA Polymerase:}}
There was a HiFi DNA polymerase dependent difference that was consistent
across sub-sampling {[}Figure 1{]}. Lower cycle numbers (15-20) tended
to result in less differences between HiFI DNA polymerase versus higher
cycle numbers (25, 30, and 35) {[}Figure 1{]}. At 35 cycles, all
sub-sampling levels had significant differences between HiFi DNA
polymerases (P-value \textless{} 0.0001) {[}Table S1{]}. Some
differences between HiFi DNA polymerase we identifed at 25 and 30 cycles
but the sub-sampling depth had to be 5000 or higher (P-value \textless{}
0.05) {[}Table S1{]}. Most of the differences observed at 35 cycles were
between Platinum and other HiFi DNA polymerases, based on a Tukey
post-hoc test (P-value \textless{} 0.05) {[}Table S2{]}

This HiFi DNA polymerase dependent difference in the number of OTUs was
also observed in the mock community samples with the same DNA
polymerases having high (Platinum) and low (Accuprime) number of OTUs
{[}Figure 2 \& Table S3{]}. In contrast to fecal samples, differences
between HiFi DNA polymerase were observed as early as 20 cycles and as
low as a sub-sampling depth of 1000 sequences (P-value = 0.002) {[}Table
S3{]}. For both different cycle numbers and sub-sampling depths, the
majority of differences were between Platinum and the other HiFi DNA
polymerases {[}Table S4{]}. The lowest number of OTUs identified was
from Accuprime\textsuperscript{TM} while the highest was from Platinum
and this was consistent for both fecal and mock samples across
sub-sampling depth and cycle number {[}Figure 1 \& 2{]}. Based on these
observations it is clear that different HiFi DNA polymerase result in
different total number of OTUs observed within a sample.

\textbf{\emph{Minimal Bray-Curtis Differences are Detected by Cycle
Number:}} Overall, there were a few sub-sampling dependent differences
in bacterial communities between cycle number . First, independent of
HiFi DNA polymerase there were differences for the same fecal sample
between 20 versus 25 cycles that was dependent on sub-sampling depth
(sub-sampled to 1000 = 0.51 (0.4 - 0.79) (median (25\% - 75\%
quantile)), sub-sampled to 5000 = 0.43 (0.33 - 0.63), sub-sampled to
10000 = 0.4 (0.24 - 0.43)) {[}Figure 3A{]}. Second, where data is
available for the mock communities, there were larger difference between
20 and 25 cycles (sub-sampled to 1000 = 0.88 (0.42 - 0.91)) {[}Figure
3B{]}. Third, using PERMANOVA to test for differences within HiFi DNA
polymerase groups based on cycle number, only Phusion had cycle
dependent differences at 1000 and 5000 sub-sampling depth (P-value =
0.03 and 0.01, respectively). Phusion was one of only two HiFi DNA
polymerases that that had fecal samples for the 1000 sub-sampling depth
at 15 cycles. These stated differences between the next 5-cycle
increment do not persist once 25 cycles are reached {[}Figure 3{]}.

Next, we assessed whether there were any differences between 5-cycle
increments within each individual or sample replicate. For both fecal
and mock samples, we found that there was no detectable difference in
Bray-Curtis index when comparing to the previous 5-cycle increment
(P-value \textgreater{} 0.05). However, Phusion at 1000 sub-sampling
depth had a P-value = 0.02 before multiple comparison correction.
Similar to the PERMANOVA analysis, at higher sub-sampling depths these
differences in Bray-Curtis indices disappear. Overall, HiFi DNA
polymerase differences in Bray-Curtis index are dependent on both
sub-sampling depth and cycle number.

\textbf{\emph{Sequence Error is Dependent on both HiFi DNA Polymerase
and Cycle Number:}} Differences by HiFi DNA polymerase in the median
average per base error varied without a clear pattern across
sub-sampling depth {[}Table S5{]}. The highest per base median average
error rates were for the Kappa HiFi DNA polymerase {[}Figure 4{]}. This
error rate was minimally affected by both the `pre.cluster' step and
chimera removal by VSEARCH {[}Figure 4{]}. The differences in the per
base error rate between the various HiFi DNA polymerase was cycle
dependent with Platinum having the largest differences versus other HiFi
DNA polymerases {[}Figure 4B-C and Table S6{]}. The total sequences with
at least one error had was also cycle number dependent and mostly
alleviated by the use of the `pre.cluster' step {[}Figure S1{]}.
Differences in sequences with an error before this `pre.cluster' step
were due to large differences in Accuprime\textsuperscript{TM} and
Platinum versus the other HiFi DNA polymerases {[}Figure S1 \& Table S7
\& S8{]}. Investigation of whether there were HiFi DNA polymerase
dependent effects on base substitution found that there were no biases
in the types of substitution made {[}Figure S2{]}. Although though
sequence error is dependent on HiFi DNA polymerase it can be corrected
using exsiting bioinformatic approaches.

\textbf{\emph{Chimeric Sequences are HiFi DNA Polymerase Dependent and
Correlate with Number of OTUs:}} At all levels of sub-sampling and cycle
numbers there were significant differences in the chimera prevalence
based on HiFi DNA polymerase used (P-value \textless{} 0.05) {[}Table
S9{]}. Differences between Platinum and all other HiFi DNA polymerases
accounted for the majority of these differences and was independent of
cycle number and sub-sampling depth {[}Table S10{]}. Across sub-sampling
depth and cycle number Accuprime\textsuperscript{TM} had the lowest
chimera prevalence of all the HiFi DNA Polymerases regardless of whether
`pre.cluster' or chimera removal with VSEARCH was used {[}Figure 5{]}.

For all HiFi DNA polymerases, a positive correlation was observed
between chimeric sequences and number of OTUs, with this correlation
being strongest for Accuprime, Platinum and Phusion HiFi DNA Polymerase
{[}Figure 6{]}. The R\textsuperscript{2} value between the number of
OTUs and chimeric sequences did not change with the use of `pre.cluster'
or with the removal of chimeras using VSEARCH {[}Figure 6{]}. Taken
together, this data suggests that a strong correlation exists between
the number of OTUs and the prevalence of chimeric sequences.

\newpage

\subsection{Discussion}\label{discussion}

These observations build upon previous studies (Gohl et al., 2016) by
showing that different HiFi DNA polymerases have significant differences
in the number of OTUs and that changes in total OTUs correlate with
chimeras not removed after sequence processing {[}Figure 1-2 \& 5{]}.
HiFi DNA polymerase dependent differences in total number of OTUs are
important to consider since many diversity metrics rely on this measure
as part of their calculations. Our observations show that HiFi DNA
polymerase can have a noticeable affect on the OTUs generated and these
differences are consistent across sub-sampling depth and PCR cycle
number {[}Figure 2-4{]}. Importantly, high biomass samples may
underestimate the biases present within low biomass samples. An
indication that this may be the case is the earlier detection of
differences in total number of OTUs between HiFi DNA polymerases in the
mock versus fecal samples.

Although we did not observe strong differences in the Bray-Curtis index
the data suggests that there may be differences between 15 and 20 cycles
versus higher cycle numbers, such as 30 cycles, that are commonly used.
There was few differences within individuals between corresponding
5-cycle increments (e.g.~15 to 20, 20 to 25, etc.). However, there is a
clear trend that suggests that the community at 20 cycles is different
then the community at 25 cycles {[}Figure 3{]}. These findings, in
conjunction with the PERMANOVA results, suggest that cycle number can
change bacterial community calculations but that these differences are
minimal once 25 cycles are reached. Further, for some HiFI DNA
polymerases, increasing the sub-sampling depth may reduce some of these
observed community differences at lower cycle numbers.

Increasing the cycle number also exacerbated chimera prevalence
differences between the different HiFi DNA polymerases {[}Figure 5{]}.
The chimera prevalence was strongly correlated with the number of OTUs
which is relied upon heavily for many community metric calculations.
However, Bray-Curtis analysis with PERMANOVA showed few differences
based on HiFi DNA polymerase. It is possible that many of the increased
number of OTUs are not highly abundant allowing the Bray-Curtis index to
be able to successfully down-weight these respective OTUs (Minchin,
1987). The choice of downstream diversity metric could be an important
consideration in helping to mitigate these observed changes due to high
chimera prevalence in specific HiFi DNA polymerases.

Our observations suggest that there are clear HiFi DNA polymerase
dependent differences in both per base error rate and chimera prevalence
that cannot be removed using bioinformatic approaches {[}Figure 4 \&
5{]}. Although it may be a natural assumption that the variation may be
due to the DNA polymerase family, the highest chimera rate, from
Platinum, was a family A polymerase while the lowest, from Accuprime,
was also an A family polymerase (Ishino \& Ishino, 2014). In fact, from
the material safety data sheet (MSDS), the differences between the two
is not immediately apparent. Both Accuprime and Platinum contain a
recombinant \emph{Taq} DNA polymerase, a \emph{Pyrococcus} spp GB-D
polymerase and a platinum \emph{Taq} antibody. It is possible that
differences in how the recombinant \emph{Taq} was generated could be the
main reason for the differences in chimera rate.

\newpage

\subsection{Conclusion}\label{conclusion}

Our findings show that the number of OTUs will be dependent on both HiFi
DNA polymerase and cycle number chosen. Care should be taken when
choosing a HiFi DNA polymerase for 16S rRNA gene surveys since intrinsic
differences can change the number of OTUs observed and influence
diversity based metrics that do not down-weight rare observations.

\newpage

\subsection{Acknowledgements}\label{acknowledgements}

The authors would like to thank all the study participants in ERIN whose
samples were utilized. We would also like to thank Judy Opp and April
Cockburn for their effort in sequencing the samples as part of the
Microbiome Core Facility at the University of Michigan. Salary support
for Marc Sze came from the Canadian Institute of Health Research and the
Michigan Institute for Clinical and Health Research Postdoctoral
Translational Scholar Program.

\newpage

\subsection{References}\label{references}

\hypertarget{refs}{}
\hypertarget{ref-benjamini_controlling_1995}{}
Benjamini Y., Hochberg Y. 1995. Controlling the false discovery rate: A
practical and powerful approach to multiple testing. \emph{Journal of
the Royal Statistical Society. Series B (Methodological)} 57:289--300.

\hypertarget{ref-rdp_Cole_2013}{}
Cole JR., Wang Q., Fish JA., Chai B., McGarrell DM., Sun Y., Brown CT.,
Porras-Alfaro A., Kuske CR., Tiedje JM. 2013. Ribosomal database
project: Data and tools for high throughput rRNA analysis. \emph{Nucleic
Acids Research} 42:D633--D642. DOI:
\href{https://doi.org/10.1093/nar/gkt1244}{10.1093/nar/gkt1244}.

\hypertarget{ref-uchime_Edgar_2011}{}
Edgar RC., Haas BJ., Clemente JC., Quince C., Knight R. 2011. UCHIME
improves sensitivity and speed of chimera detection.
\emph{Bioinformatics} 27:2194--2200. DOI:
\href{https://doi.org/10.1093/bioinformatics/btr381}{10.1093/bioinformatics/btr381}.

\hypertarget{ref-viridis_citation_2017}{}
Garnier S. 2017. \emph{Viridis: Default color maps from 'matplotlib'}.

\hypertarget{ref-taq_Gohl_2016}{}
Gohl DM., Vangay P., Garbe J., MacLean A., Hauge A., Becker A., Gould
TJ., Clayton JB., Johnson TJ., Hunter R., Knights D., Beckman KB. 2016.
Systematic improvement of amplicon marker gene methods for increased
accuracy in microbiome studies. \emph{Nature Biotechnology} 34:942--949.
DOI: \href{https://doi.org/10.1038/nbt.3601}{10.1038/nbt.3601}.

\hypertarget{ref-Haas2011}{}
Haas BJ., Gevers D., Earl AM., Feldgarden M., Ward DV., Giannoukos G.,
Ciulla D., Tabbaa D., Highlander SK., Sodergren E., Methe B., DeSantis
TZ., Petrosino JF., Knight R., and BWB. 2011. Chimeric 16S rRNA sequence
formation and detection in sanger and 454-pyrosequenced PCR amplicons.
\emph{Genome Research} 21:494--504. DOI:
\href{https://doi.org/10.1101/gr.112730.110}{10.1101/gr.112730.110}.

\hypertarget{ref-polymerase_Ishino_2014}{}
Ishino S., Ishino Y. 2014. DNA polymerases as useful reagents for
biotechnology â the history of developmental research in the field.
\emph{Frontiers in Microbiology} 5. DOI:
\href{https://doi.org/10.3389/fmicb.2014.00465}{10.3389/fmicb.2014.00465}.

\hypertarget{ref-protocol_Kozich_2013}{}
Kozich JJ., Westcott SL., Baxter NT., Highlander SK., Schloss PD. 2013.
Development of a dual-index sequencing strategy and curation pipeline
for analyzing amplicon sequence data on the MiSeq illumina sequencing
platform. \emph{Applied and Environmental Microbiology} 79:5112--5120.
DOI: \href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\hypertarget{ref-bc_index_Minchin1987}{}
Minchin PR. 1987. An evaluation of the relative robustness of techniques
for ecological ordination. \emph{Vegetatio} 69:89--107. DOI:
\href{https://doi.org/10.1007/bf00038690}{10.1007/bf00038690}.

\hypertarget{ref-vegan_citation}{}
Oksanen J., Blanchet FG., Friendly M., Kindt R., Legendre P., McGlinn
D., Minchin PR., O'Hara RB., Simpson GL., Solymos P., Stevens MHH.,
Szoecs E., Wagner H. 2017. \emph{Vegan: Community ecology package}.

\hypertarget{ref-r_citation_2017}{}
R Core Team. 2017. \emph{R: A language and environment for statistical
computing}. Vienna, Austria: R Foundation for Statistical Computing.

\hypertarget{ref-vsearch_Rognes_2016}{}
Rognes T., Flouri T., Nichols B., Quince C., Mahé F. 2016. VSEARCH: A
versatile open source tool for metagenomics. \emph{PeerJ} 4:e2584. DOI:
\href{https://doi.org/10.7717/peerj.2584}{10.7717/peerj.2584}.

\hypertarget{ref-contamination_Salter2014}{}
Salter SJ., Cox MJ., Turek EM., Calus ST., Cookson WO., Moffatt MF.,
Turner P., Parkhill J., Loman NJ., Walker AW. 2014. Reagent and
laboratory contamination can critically impact sequence-based microbiome
analyses. \emph{BMC Biology} 12. DOI:
\href{https://doi.org/10.1186/s12915-014-0087-z}{10.1186/s12915-014-0087-z}.

\hypertarget{ref-mothur_schloss_2009}{}
Schloss PD., Westcott SL., Ryabin T., Hall JR., Hartmann M., Hollister
EB., Lesniewski RA., Oakley BB., Parks DH., Robinson CJ., Sahl JW.,
Stres B., Thallinger GG., Horn DJV., Weber CF. 2009. Introducing mothur:
Open-source, platform-independent, community-supported software for
describing and comparing microbial communities. \emph{Applied and
Environmental Microbiology} 75:7537--7541. DOI:
\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\hypertarget{ref-erin_seekatz_2016}{}
Seekatz AM., Rao K., Santhosh K., Young VB. 2016. Dynamics of the fecal
microbiome in patients with recurrent and nonrecurrent clostridium
difficile infection. \emph{Genome Medicine} 8. DOI:
\href{https://doi.org/10.1186/s13073-016-0298-8}{10.1186/s13073-016-0298-8}.

\hypertarget{ref-opticlust_Westcott_2017}{}
Westcott SL., Schloss PD. 2017. OptiClust, an improved method for
assigning amplicon-based sequence data to operational taxonomic units.
\emph{mSphere} 2:e00073--17. DOI:
\href{https://doi.org/10.1128/mspheredirect.00073-17}{10.1128/mspheredirect.00073-17}.

\hypertarget{ref-tidyverse_2017}{}
Wickham H. 2017. \emph{Tidyverse: Easily install and load 'tidyverse'
packages}.

\newpage

\textbf{Figure 1: Normalized Fecal Number of OTUs.} The x-axis
represents the different sub-sampling depths used and the y-axis is the
normalized within individual number of OTUs. The red line represents the
overall mean Z-score normalized number of OTUs for each respective HiFi
DNA polymerase. The dashed black line represents the overall Z-score
normalized mean number of OTUs.

\textbf{Figure 2: Mock Sample Variability in Number of OTUs based on
HiFi DNA Polymerase.} A) Sub-sampled to 1000 reads. B) Sub-sampled to
5000 reads. C) Sub-sampled to 10000 reads. The dotted line represents
the number of OTUs generated when the mock reference sequences are run
through the pipeline.

\textbf{Figure 3: Community Differences by Five-Cycle Intervals and
Sub-sampling Depth.} A) Fecal samples within person difference based on
the next 5-cycle PCR interval. B) Mock samples within replicate
difference based on the next 5-cycle PCR interval.

\textbf{Figure 4: HiFi DNA Polymerase Per Base Error Rate in Mock
Samples.} A) Error rate before the merger of sequences with pre.cluster
and the removal of chimeras with VSEARCH. B) Error rate before the
removal of chimeras with VSEARCH. C) Full pipeline. The error bars
represent the 75\% interquartile range of the median.

\textbf{Figure 5: HiFi DNA Polymerase Chimera Prevalence in Mock
Samples.} A) Chimera sequence percentage before the merger of sequences
with pre.cluster and the removal of chimeras with VSEARCH. B) Chimera
sequence percentage before the removal of chimeras with VSEARCH. C) Full
pipeline. The error bars represent the 75\% interquartile range of the
median.

\textbf{Figure 6: The Correlation between Number of OTUs and Chimeras.}
A) Correlation before the merger of sequences with pre.cluster and the
removal of chimeras with VSEARCH. B) Correlation before the removal of
chimeras with VSEARCH. C) Correlation with full pipeline.

\newpage

\textbf{Figure S1: HiFi DNA Polymerase Sequence Error Prevalence in Mock
Samples.} A) Sequence error prevalence before the merger of sequences
with pre.cluster and the removal of chimeras with VSEARCH. B) Sequence
error prevalence before the removal of chimeras with VSEARCH. C) Full
pipeline. The error bars represent the 75\% interquartile range of the
median.

\textbf{Figure S2: HiFi DNA Polymerase Nucleotide Subsitutions in Mock
Samples.}


\end{document}
