---
title: ''
csl: peerj.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read in the meta data file
metadata <- read_csv("../data/process/tables/meta_data.csv")

# Read in fecal count data
fecal_numOTU_data <- read_csv("../data/process/tables/fecal_sub_sample_1000_count_table.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5")), 
         cycles = factor(cycles, 
                            levels = c("15x", "20x", "25x", "30x", "35x"), 
                            labels = c("15x", "20x", "25x", "30x", "35x"))) %>% 
  group_by(taq, cycles) %>% 
  summarise(taq_median = median(numOTUs, na.rm = T), 
            taq_min = min(numOTUs, na.rm = T), 
            taq_max = max(numOTUs, na.rm = T)) %>% 
  mutate(range_diff = taq_max - taq_min)
  
# Read in fecal sample number of OTU results
fs_anova_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_anova_results.csv"))
fs_tukey_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_tukey_results.csv")) %>% 
  rename(padj = `p adj`)


# Read in the Mock count data
mock_num_otu_data <- read_csv("../data/process/tables/mock_sub_sample_1000_count_table.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5"))) %>% 
  group_by(taq, cycles) %>% 
  summarise(taq_median = median(numOTUs, na.rm = T), 
            taq_min = min(numOTUs, na.rm = T), 
            taq_max = max(numOTUs, na.rm = T)) %>% 
  mutate(range_diff = taq_max - taq_min)

ref_mock_otu_num <- read_tsv("../data/process/zymo_mock_only.shared") %>% 
  select(numOtus)


# Read in mock sample number of OTU results
mock_anova_summary <- as.data.frame(read_csv("../data/process/tables/mock_overall_anova_results.csv"))
mock_tukey_summary <- read_csv("../data/process/tables/mock_overall_tukey_results.csv")

# Read in error rate results
krsukal_error_rate <- read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")
dunn_error_rate <- read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")

# Read in sequence error count results
kruskal_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")
dunn_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")

# Read in chimera prevalence results
kruskal_chimera_prev <- as.data.frame(read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv"))
dunn_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")

# Read in results from PERMANOVA and bray analysis
bray_sim_to_prev_cycle_kruskal_results <- as.data.frame(read_csv("../data/process/tables/bray_sim_to_prev_cycle_kruskal_results.csv"))
bray_permanova_by_taq_results <- as.data.frame(read_csv("../data/process/tables/bray_permanova_by_taq_results.csv"))

# Read in actual error data
error_data <- read_csv("../data/process/tables/mock_chimera_error_summary.csv") %>% 
                       filter(total_seqs >= 1000)

error_summary <- error_data %>% 
  group_by(taq, cycles) %>% 
  summarise_at(vars(mean_error, chimera_prevalence, seq_error_prevalence), funs(median, min, max)) %>% 
  select(taq, cycles, contains("chimera"), contains("mean_error"), contains("seq_error"))
  

# Vector of sub samples used
sub_sample_level <- c("1000", "5000", "10000")

dist_data <- sapply(sub_sample_level, 
                    function(x) as.data.frame(read_csv(paste("../data/process/tables/bray_5_cycle_dist_",                                                x, "_data.csv", sep = ""))), simplify = F)

mock_dist_data <- sapply(sub_sample_level, 
                    function(x) as.data.frame(read_csv(paste("../data/process/tables/mock_bray_5_cycle_dist_", 
                                               x, "_data.csv", sep = ""))), simplify = F)

# Read in the regression data
r2_table <- read_csv("../data/process/tables/r2_otu_vs_chimera_summary.csv")

```



# Error Introduced into 16S rRNA Gene Sequencing Results Varies by High Fidelity DNA Polymerase Used

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** It is challenging to compare 16S rRNA gene sequencing data across studies and one of the reasons for this is due to error. There are many different places throughout the workflow where error can be introduced into the pipeline. Here, we focus on studying how the number of cycles and high fidelity (HiFi) DNA polymerase introduce error by varying cycle number and polymerase used to amplify 16S rRNA genes in human fecal and mock community samples.  

**Methods.** We extracted DNA from fecal samples (n=4) using a PowerMag DNA extraction kit with a 10 minute bead beating step and amplified at 15, 20, 25, 30, and 35 cycles using Accuprime, Kappa, Phusion, Platinum, or Q5 HiFi DNA polymerase. Amplification of mock communities (technical replicates n=4) consisting of previously isolated whole genomes of 8 different bacteria used the same approach. The analysis initially examined the number of Operational Taxonomic Units (OTUs) for fecal samples and mock communities. It also assessed polymerase dependent differences in the Bray-Curtis index, error rate, sequence error prevalence, chimera prevalence, and the  correlation between chimera prevalence and number of OTUs. 

**Results.** When analyzing fecal samples we observed that the range in the number of OTUs detected was not consistent between HiFi DNA polymerases at 35 cycles (Accuprime = `r filter(fecal_numOTU_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_max)` (min - max) versus Phusion = `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_max)`). Additionally, the median number of OTUs vaired by HiFi DNA polymerase used (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001). When analyzing mock community samples the variation in the number of OTUs detected by the polymerases was observable as early as 20 cycles (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`). There also was a large range in the number of OTUs amplified by the polymerases at 35 cycles (Accuprime = `r filter(mock_num_otu_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_min)` - `r filter(mock_num_otu_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_max)` versus Phusion = `r filter(mock_num_otu_data, cycles == "35x", taq == "Phusion") %>% pull(taq_min)` - `r filter(mock_num_otu_data, cycles == "35x", taq == "Phusion") %>% pull(taq_max)`). Chimera prevalence in mock communities varied by polymerase with differences being most notable at 35 cycles (Kappa = `r round((filter(error_summary, cycles == "35x", taq == "K") %>% pull(chimera_prevalence_median))*100, digits = 2)`% (median) versus Platinum = `r round((filter(error_summary, cycles == "35x", taq == "PL") %>% pull(chimera_prevalence_median))*100, digits = 2)`%) and this variation persisted after chimera removal using VSEARCH. We also observed positive correlations between chimera prevalence and the number of OTUs with Platinum having the highest (R^2 = `r max(pull(r2_table, r2))`) and Kappa having the worst (R^2 = `r min(pull(r2_table, r2))`).   

**Conclusions.** Although the variation in the number of OTUs in fecal samples could be due to certain polymerases capturing the biological variability better than others, this is unlikely to be the main reason for our observed differences. In mock community samples, the strong correlation between chimera prevalence and the number of OTUs suggests that this is the main reason for differences between the polymerases. Ultimately, this variation makes comparison across studies difficult and care should be exercised when choosing the polymerase and number of cycles in 16S rRNA gene sequencing studies. 


\newpage


## Introduction

The bacterial community is reported to vary between case and control for a number of diseases [@Turnbaugh2008; @Baxter2016; @Sze2015; @Bonfili2017]. However, for diseases like obesity, the taxa identified have varied widely depending on the study [@Turnbaugh2008; @Zupancic2012]. Some of this variation could be due to error introduced during the 16S rRNA gene sequencing workflow. Yet, standardizing a 16S rRNA gene sequencing workflow will ultimately result in a standardized and reproducible bias due to choices made on the methods used for preservation, extraction, PCR, and sequencing. Within this context, all 16S rRNA gene sequencing methods are biased even when these workflows are standardized to increase reproducibility. In order to interpret specific studies within the broader context of the overall field, assessing error at different parts of the 16S rRNA gene sequencing workflow is critical.

A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, PCR, and sequencing steps. The preservation and extraction stages of the 16S rRNA gene sequencing workflow have been the most extensively studied [@contamination_Salter2014; @preservation_Song_2016; @storage_Bassis_2017; @review_Kim_2017]. For preservation and extraction stages of the workflow, it has been consistently found that there are biases based on the kits used, but that these differences are smaller than the overall biological difference measured between samples with different kits [@preservation_Song_2016; @storage_Bassis_2017]. Since these studies use the same PCR approach while varying preservation or extraction method, the contribution of PCR bias to this overall workflow is not well characterized.

There is a large body of literature that shows there are biases due to primer and number of cycles chosen for the PCR stage of 16S rRNA gene sequencing [@Eckert1991; @Burkardt2000]. Primers have variable region dependent binding affinities which causes an inability to detect specific bacteria (e.g. V1-V3 does not detect *Haemophilus influenzae* and V3-V5 does not detect *Propionibacterium acnes*) [@Meisel2016; @Sze2015 (Table S4)]. Another source of error is the selective amplification of AT-rich over GC-rich sequences which exaggerate the difference between 16S rRNA genes higher in AT versus those higher in GC content [@polz_bias_1998]. Many of these sources of biases are made worse as the number of cycles increases [@Wang1996; @Haas2011; @Kebschull2015]. For example, both amplification error and non-specific amplification (e.g. incorrect amplicon size products) also can increase as the number of cycles increases. This will increase the number of Operational Taxonomic Units (OTUs) observed and drastically change the values obtained from commonly used diversity measures [@BautistadelosSantos2016; @Acinas2005]. Additionally, as the number of cycles increases more chimeras can form from an aborted extension step that causes a priming error and subsequent secondary extension [@Haas2011]. These chimeras will artificially increase community diversity by increasing the number of OTUs that are observed [@Haas2011]. In addition to these sources of errors, there also are multiple families of DNA polymerases that have their own error rate and proof reading capacity [@polymerase_Ishino_2014]. Interestingly, the influence that these different DNA polymerases can have on the observed 16S rRNA gene sequencing results have not been well studied like some of the other sources of PCR-based bias.     

A recent study found differences in the number of OTUS and chimeras between normal and high fidelity DNA polymerases [@taq_Gohl_2016]. The authors could reduce the difference between the two polymerases by optimizing the annealing and extension steps within the PCR protocol [@taq_Gohl_2016]. Yet, within this study there was no comparison made between different high fidelity DNA polymerases. Due to this gap, it is natural to extend this line of inquiry and test if biases in the number of OTUs and chimeras also are dependent on the type of high fidelity DNA polymerase. This study will investigate how high fidelity DNA polymerases can bias observed bacterial community results derived from 16S rRNA gene sequencing. We will accomplish this by examining the number of OTUs, error rate, number of sequences with an error, and chimera prevalence at varying number of cycles in five different high fidelity DNA polymerases



\newpage

## Materials & Methods

***Human and Mock Samples:***
Fecal samples were obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN). The processing and storage of these samples were previously published [@erin_seekatz_2016]. Other than confirmation that none of these individuals had a *Clostridium difficle* infection, clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used for mock communities and was made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards).

***PCR Protocol:***
The five different high fidelity DNA polymerases (hereto referred to as polymerases) that were tested included AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA). The polymerases activation time was 2 minutes, unless a different activation was specified by the manufacturer. The annealing and extension time for Platinum and Accuprime followed a previously published protocol [@protocol_Kozich_2013] (https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md). For Kappa and Q5, the annealing and extension time were from a previously published protocol [@taq_Gohl_2016]. For Phusion, the company defined activation and annealing times were used while the extension time followed the Accuprime and Platinum settings. 

The number of cycles in the PCR for fecal and mock samples started at 15 and increased by 5 up to 35 cycles, with amplicons used at each 5-step increase for sequencing. The PCR of fecal DNA samples consisted of all 4 samples at 15, 20, 25, 30, and 35 cycles for each polymerase (total sample n=100). The mock communities had 4 replicates at 15, 20, 25, and 35 cycles and 10 replicates for 30 cycles for all polymerases (total samples n=130). No mock community sample had enough PCR product at 15 cycles for adequate 16S rRNA gene sequencing.    

***Sequence Processing:***
The mothur software program was used for all sequence processing steps [@mothur_schloss_2009]. The protocol has been previously published [@protocol_Kozich_2013] (https://www.mothur.org/wiki/MiSeq_SOP). Two major differences from the published protocol were the use of VSEARCH instead of UCHIME for chimera detection and the use of the OptiClust algorithm instead of average neighbor for OTU generation at 97% similarity [@vsearch_Rognes_2016; @uchime_Edgar_2011; @opticlust_Westcott_2017]. Sequence error was determined using the 'seq.error' command on mock samples to compare back to the reference 16S sequences [@mothur_schloss_2009; @vsearch_Rognes_2016; @rdp_Cole_2013].  

***Analysis Workflow:***
To adjust for unequal sequencing, all samples were rarefied to 1000 sequences for downstream analysis. The total number of OTUs was analyzed for both the fecal and mock community samples. For fecal samples, cycle dependent affects on Bray-Curtis indices were assessed for cycle group and within individual differences from the previous cycle (e.g. 20 versus 25, 25 versus 30). Based on these observations we analyzed potential reasons for these differences. Analysis of the mock community of each polymerase for sequence error rate, number of sequences with an error, base substitution, and numbers of chimeras before and after chimera removal with VSEARCH was assessed. Additionally, the correlation between the number of chimeras and the number of OTUs was also assessed.

***Statistical Analysis:***
All analysis was done with the R (v `r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. Data transformation and graphing was completed using the tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) and colors selected using the viridis package (v `r paste(packageVersion("viridis")[1])`) [@tidyverse_2017; @viridis_citation_2017]. Differences in the total number of OTUs were analyzed using an ANOVA with a tukey post-hoc test. For the fecal samples the data was normalized to each individual by cycle number to account for the biological variation between people. Bray-Curtis distance matrices were generated using mothur after 100 sub-samplings at 1000, 5000, 10000, and 15000 sequence depth. The distance matrix data was analyzed using PERMANOVA with the vegan package (v `r paste(packageVersion("vegan")[1])`) [@vegan_citation] and Kruskal-Wallis tests within R. For both error and chimera analysis, samples were tested using Kruskal-Wallis with a Dunns post-hoc test. Where applicable correction for multiple comparison utilized the Benjamini-Hochberg method [@benjamini_controlling_1995].    

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. The raw sequences can be found in the SRA at the following accession number SRP132931.


\newpage


## Results

***The Number of OTUs are Dependent on Polymerase:***
Differences in the range of the number of OTUs detected for fecal samples is dependent on the polymerase used [Figure 1]. Additionally, there is a trend for lower number of cycles (15-20) to result in a reduced range in the number of OTUs versus higher number of cycles (25, 30, and 35) for all polymerases [Figure 1]. There is an overall difference in the number of OTUs detected within fecal samples between polymerases at 35 cycles (F-stat > `r round(min(filter(fs_anova_summary, cycle == "35x", sub_sample_level == 1000)[, "F.value"]), digits = 2)`, P-value = `r format(filter(fs_anova_summary, cycle == "35x", sub_sample_level == 1000)[, "bh"], digits = 2, nsmall = 2, scientific = T)`) [Table S1]. No differences between the polymerases was found based on a Tukey post-hoc test (P-value `r ifelse(min(filter(fs_tukey_summary, sub_sample_level == 1000)[, "padj"]) > 0.05,  invisible(">"), invisible("check data"))` 0.05) [Table S2]. The polymerase dependent difference in the range of the number of OTUs also was observed in the mock community samples [Figure 2]. Regardless if fecal or mock communities were used, the same polymerases had high (Platinum) and low (Accuprime) number of OTUs and this was consistent across the number of cycles used [Figure 1-2 & Table S1-S4]. In contrast to the results obtained with fecal samples, differences between polymerases were observed as early as 20 cycles in the mock community (F-stat = `r round(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "F.value"], digits = 2)`, P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`) [Table S1]. The majority of differences for the number of OTUs detected in the Mock community is largely due to Kappa and Platinum differences to the other polymerases across the different number of cycles [Table S2]. Based on these observations in fecal and mock communities, it is clear that different polymerases result in a different total number of OTUs within a sample.


***Minimal Bray-Curtis Differences are Detected and are Dependent on Cycle Number:***
A few small differences based on the number of cycles were detected in overall bacterial community composition. Within the same fecal sample and independent of polymerases, there are large differences in the community composition between 20 and 25 cycles [Figure 3A]. Additionally, when data was available for the mock communities, there are also large observed differences between 20 and 25 cycles [Figure 3B]. These community differences within a sample disappear when comparing 25 to 30 cycles and do not persist past 25 cycles [Figure 3]. Although these trends are clearly noticeable, there is no detectable difference in Bray-Curtis index when comparing to the previous 5-cycle increment for both fecal and mock communities after multiple comparison correction (P-value `r ifelse(min(filter(bray_sim_to_prev_cycle_kruskal_results, sub_sample_level == 1000)[, "bh"], na.rm = T) > 0.05, invisible(">"), invisible("check data"))` 0.05). Using PERMANOVA to test for community differences based on the number of cycles within polymerases, only Phusion had cycle dependent differences (P-value = `r format(filter(bray_permanova_by_taq_results, sub_sample_level == 1000, taq == "PHU")[, "bh"], digits = 1, nsmall = 1)`. For fecal samples, Phusion was one of two polymerases that had enough sequences to be rarefied to 1000 at 15 cycles. Overall, these data suggest that there are small polymerase differences in Bray-Curtis index that are dependent on the number of cycles.     


***Sequence Error is Dependent on both Polymerase and Cycle Number:***
Differences in the median error rate varied by polymerase, with Kappa being the highest [Figure 4 & Table S3]. The differences in the median error rate between the polymerases also is dependent on the number of cycles, with Platinum and Kappa having the most differences versus other polymerases [Figure 4 and Table S4]. The total sequences with at least one error also is dependent on the number of cycles [Table S3 & S4]. These differences in sequences with at least one error is mostly due to differences in Accuprime^TM^ and Kappa versus the other polymerases [Table S3 & S4]. The differences in error rates were not due to polymerase dependent differences in base substitution rate [Figure S1]. These results suggest that sequence error is dependent on polymerase and the number of cycles, but is not attributable to a specific type. Based on these results, we examined whether chimeras also were dependent on polymerase and whether this also could affect the number of OTUs. 


***Prevalence of Chimeric Sequences are Polymerase Dependent and Correlate with the Number of OTUs:***
There is significant differences in the chimera prevalence based on polymerase at all the number of cycles used (P-value `r ifelse(max(filter(kruskal_chimera_prev)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S3]. Differences in chimera prevalence between Platinum and all other polymerases accounted for the majority of these differences [Table S4]. Accuprime^TM^ had the lowest chimera prevalence of all polymerases regardless of whether chimera removal with VSEARCH was used [Figure 5A & 5B]. Additionally, there was a plateau in the total percent of chimeras that were removed that was similar for all polymerases [Figure 5C]. A positive correlation was observed between chimeric sequences and the number of OTUs for all polymerases [Figure 6]. This positive correlation was strongest for Accuprime^TM^, Platinum, and Phusion [Figure 6]. This data suggests that chimera prevalence depends on polymerase used and confirms that the number of OTUs is dependent on the prevalence of these chimeric sequences. 

\newpage

## Discussion

In this study we show that the number of OTUs, error rate, and chimera prevalence depends on polymerase used [Figure 1-5]. These differences are important because many diversity metrics rely on the number of OTUs or other measures dependent on error rate and chimera prevalence as part of their metric calculations (e.g. richness). Additionally, the earlier detection of differences in total number of OTUs between polymerases in the mock versus fecal samples might indicate that high biomass samples may underestimate the biases present within low biomass samples. We observed that undetected chimeras that were not identified and removed using standard bioinformatic approaches cause many of these differences. This suggests that some of the specific diversity differences between studies can be attributed to differences in polyermase used. Based on these observations, metrics that measure within sample diversity like richness depend on polymerase but this may not be the case for metrics that assess between sample diversity.    

There were few differences that depend on polymerase for between sample diversity, as measured by the Bray-Curtis index. Our observations generally found no differences in the overall bacterial community composition for the number of cycles used. One possible reason for this outcome is that our study did not have enough power to detect differences due to low sample number in each group. Another reason is that many of the OTUs are likely not highly abundant, allowing the Bray-Curtis index to be able to successfully down-weight chimeric OTUs [@bc_index_Minchin1987]. The choice of downstream diversity metric could be an important consideration in helping to mitigate these observed polymerase dependent differences in chimera prevalence. Metrics that solely use presence/absence of OTUs (e.g. Jaccard [@Real1996]) may be less robust to chimera prevalence and by extension total number of OTU differences in polymerases. When choosing a distance metric, careful consideration of the biases introduced from the PCR step of the 16S rRNA gene sequencing workflow need to be taken into account. With differences in the number of OTUs and chimera prevalence depending on polymerase used, it might be easier to avoid specific DNA polymerase families altogether.

Although the variation in error rate and chimera prevalence may be due to the DNA polymerase family, this is unlikely to be the only contributor. For example, the highest and lowest chimera rates both belonged to a family A polymerase (Platinum and Accuprime^TM^ respectively) [@polymerase_Ishino_2014]. Additionally, based on the material safety data sheet the differences between the two polymerases are not immediately apparent. Both polymerases contain a recombinant *Taq* DNA polymerase, a *Pyrococcus* spp GB-D polymerase and a platinum *Taq* antibody. With everything else being equal, it is possible that differences in how the recombinant *Taq* was generated could be a contributing factor for the differences observed between the polymerases. We are unlikely to avoid adding polymerase dependent bias to 16S rRNA gene sequencing results, however, these differences may not be large enough to mask the actual biological signal. 

The majority of polymerases we studied add small increases in the number of OTUs and chimera prevalence and may be masked by biological differences. The sequence error introduced by the polymerase is also small and likely to be smaller than the biological variation within a specific study, which would be consistent with previous findings for preservation and DNA extraction methods [@preservation_Luo_2016; @preservation_Song_2016; @contamination_Salter2014]. The choice of polymerase should be an important consideration in the creation of a 16S rRNA gene sequencing workflow because using different polymerases can add error and bias to the downstream observations. Although standardization of the workflow may partially solve this problem by introducing a consistent bias to all samples, it does come at a cost. 

Methods can be standardized but they commonly contain bias that is reproducible and may miss important associations. Bias can be easily reproduced and can be found in every step of the 16S rRNA gene sequencing workflow [@Amir2017; @preservation_Luo_2016; @contamination_Salter2014; @taq_Gohl_2016]. This study shows that specific diversity metrics used to measure the microbial community consistently vary based on polymerase. Standardizing multiple workflows to one specific polymerase could be detrimental since some polymerases may work better in certain situations over others. Arguably, the degree of workflow standardization across studies and research group needs to be approached on a study by study basis and not every project needs to use the exact same approach. All aspects of the 16S rRNA gene sequencing workflow need to be customized for the specific microbial community that will be sampled. Although a diversity of approaches may make reproducibility and replicability more difficult it will help to avoid systematic biases from occuring due to widespread standardization of approaches. 




```{r, echo=FALSE, message=FALSE, warning=FALSE}
#Results have an impact on downstream diversity measurements due to effects on total OTUs


#Need to cover that different types of polymerase may affect outcome
#- Accuprime HiFi from recombinant A-Family Taq DNA polymerase, Pyrococcus spp GB-D polymerase (proof-reading), platinum Taq Antibody (inhibits room temp activity)

#- Kappa HiFi from B-family DNA polymerase (proof-reading baked in), propietary antibody (inhibit room temp activity)

#- Q5 HiFi from B-family thermostable DNA polymerase with 3´→ 5´ exonuclease activity, fused to a processivity-enhancing Sso7d domain to support robust DNA amplification

#-Phusion HiFi B-family DNA polymerase a novel Pyrococcus-like enzyme fused with a processivity-enhancing domain, generate blunt-ended products.

#- Platinum HiFi recombinant A-Family Taq DNA polymerase, Pyrococcus species GB-D polymeras, Platinum Taq Antibody

```



\newpage

## Conclusion

Our observations fill a gap in knowledge on the bias introduced to 16S rRNA gene sequencing results due to differences in polymerases. We found that the number of OTUs and the chimera prevalence is dependent on both polymerase and cycle number chosen. Care should be taken when choosing a polymerase for 16S rRNA gene surveys because their intrinsic differences can change the number of OTUs observed and influence diversity based metrics that do not down-weight rare observations. Knowing the inherent bias associated with different polymerases allows for better interpretation of the relationsip between an individual study to their respective field of research.

\newpage



## Acknowledgements

The authors would like to thank all the study participants in ERIN whose samples were utilized. We would also like to thank Judy Opp and April Cockburn for their effort in sequencing the samples as part of the Microbiome Core Facility at the University of Michigan. Salary support for Marc A. Sze came from the Canadian Institute of Health Research and NIH grant UL1TR002240. Salary support for Patrick D. Schloss came from NIH grants P30DK034933 and 1R01CA215574.   

\newpage

## References

<div id="refs"></div>

\newpage


**Figure 1: Total Number of OTUs in Fecal Samples by Number of Cycles.** The x-axis represents the different number of cycles used and the y-axis is the number of OTUs. The legend represents the color code for the 5 different HiFi DNA polymerases used. The range in the number of OTUs detected in the different fecal samples increased as cycle number increased. This range was also larger for specific HiFi DNA polymerases. 

**Figure 2:  Total Number of OTUs in Mock Samples by Number of Cycles.** The x-axis represents the different number of cycles used and the y-axis is the number of OTUs. The legend represents the color code for the 5 different HiFi DNA polymerases used. The dotted black line represents the number of OTUs detected when only the references sequences are clustered. The range in the number of OTUs detected in the Mock samples increased as cycle number increased. This range was also larger for specific HiFi DNA polymerases.

**Figure 3: Bray-Curtis Community Differences by Five-Cycle Intervals.** A) within person differences based on the next 5-cycle PCR interval in fecal samples. B) Within replicate difference based on the next 5-cycle PCR interval in Mock samples.

**Figure 4: HiFi DNA Polymerase Error Rate in Mock Samples.** The error bars represent the 75% interquartile range of the median error rate.  

**Figure 5: HiFi DNA Polymerase Chimera Prevalence in Mock Samples.** A) Precentage of chimeric sequences without the removal of chimeras with VSEARCH. C) Percentage of chimeric sequences with the removal of chimeras with VSEARCH. C) The total percent of chimeric sequences removed with VSEARCH by cycle number. The error bars represent the 75% interquartile range of the median. 

**Figure 6: The Correlation between Number of OTUs and Chimeras in Mock Samples.**   


\newpage


**Figure S1: HiFi DNA Polymerase Nucleotide Subsitutions in Mock Samples.**  


