---
title: ''
csl: peerj.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---


# Assessing the Differences in 16S rRNA Gene Sequencing Due to High Fidelity DNA Polymerase

# Influence of DNA Polymerase on PCR bias, chimera formation, and errors when sequencing 16S rRNA genes from complex communities


\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, amplification, and sequencing steps. At each of these stages error can be introduced that will change the underlying bacterial community composition results. In this study, we focus on the amplification step's contribution to this overall error. To accomplish this we assessed 16S rRNA gene sequencing results in mock community and human fecal samples after using different  amplification cycles and high fidelity DNA polymerases.  

**Methods.** Amplification of mock communities (technical replicates n=4) consisting of previously isolated whole genomes of 8 different bacteria were amplifeid at 15, 20, 25, 30, and 35 cycles using Accuprime, Kappa, Phusion, Platinum, or Q5 high fidelity DNA polymerase. For fecal samples (n=4) we first extracted DNA using a PowerMag DNA extraction kit with a 10 minute bead beating step and then used the same approach as the mock community for the PCR amplification step. We first assessed GC dependent differences, error rate, sequence error prevalence, chimera prevalence, and correlation between chimera prevalence and number of Operational Taxonomic Units (OTUs) by polymerase and number of cycles. Next, differences in the number of OTUs was examined based on the polymerase and number of cycles used. Additionally, differences in the bacterial community composition was assessed based on polymerase and number of cycles using the Bray-Curtis index. Based on these findings individual taxa differences based on polymerase and number of cycles was also investigated. Finally, Random Forest models were created to test whether the bacterial community was better at classifying polymerases, number of cycles, or individual donor. We also assessed whether the most important taxa in the polymerase and number of cycle Random Forest models were also important in the model for individual donors.    

**Results.** Predictably, we found noticeable differences in relative abundance based on high and low GC content (P-value $\leqslant$ `r round(max(filter(gc_data, bh < 0.05) %>% pull(bh)), digits = 2)`). Chimera prevalence in mock communities varied by polymerase with differences being most notable at 35 cycles (Kappa = `r round((filter(error_summary, cycles == "35x", taq == "K") %>% pull(chimera_prevalence_median))*100, digits = 2)`% (median) versus Platinum = `r round((filter(error_summary, cycles == "35x", taq == "PL") %>% pull(chimera_prevalence_median))*100, digits = 2)`%) and this variation persisted after chimera removal using VSEARCH. We also observed positive correlations between chimera prevalence and the number of OTUs with Platinum having the highest (R^2^ = `r max(pull(r2_table, r2))`) and Kappa having the worst (R^2^ = `r min(pull(r2_table, r2))`). When analyzing mock community samples the variation in the number of OTUs detected by the polymerases was observable as early as 20 cycles (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`). When analyzing fecal samples we observed smaller differences in the number of OTUs between polymerases. The median number of OTUs only varied by high fidelity DNA polymerase used at 35 cycles and not at 20 cycles like the mock community (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001). Random Forest models were most successful at classifying individual donor samples rather than polymerase or number of cycles used (P-value $\leqslant$ `r format(max(filter(dunn_rf_data, str_detect(var1, "fecal"), str_detect(var2, "fecal")) %>% pull(bh)), scientific = T, digits = 3)`). Additionally, the most important OTUs in the polymerase and number of cycle models were not the most important in the individual donor model.

**Conclusions.** Although there were 16S rRNA gene sequencing differences based on polymerase and number of cycles used, bacterial community composition differences were mostly only detectable in mock communities. Collectively, these results provide evidence that smaller biological differences between groups, based on 16S rRNA gene sequencing of human fecal samples, can be consistently detected regardless of polymerase and number of cycles used.  



\newpage

https://www.hindawi.com/journals/mbi/2014/287430/
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0169774
https://academic.oup.com/nar/article/24/18/3546/2359477



## Introduction

16S rRNA gene sequencing is a powerful and widely used tool for surveying the structure of microbial communities. This approach has exploded in popularity with the advent of high throughput sequencing where it is possible to characterize numerous samples with thousands of sequences per sample. Numerous factors can impact how a natural community is represented by the sequencing data including the method of acquiring samples, storage conditions, extraction methods, amplification conditions, sequencing method, and analytical pipeline. The increased sampling depth that is now available relative to previous Sanger-based methods is expected to compound the impacts of an investigator's choices and the interpretation of their results.

One step in the generation of 16S rRNA gene sequence data that has been long known to have a significant impact on the description of microbial communities is the choice of conditions for PCR amplification. Factors such as the choice of primers have an obvious impact on which populations will be amplified. However, a variety of PCR artifacts can also impact the perception of a community including the formation of chimeras, misincorporation of nucleotides, preferential amplification of populations leading to bias, and accumulation of random amplification events leading to PCR drift. Many bioinformatic tools have been developed to identify chimeras, there are significant sensitivity and specificity tradeoffs. Others have attempted to account for PCR bias using modeling approaches; however, these have been developed for idealized situations. Laboratory-based solutions to minimize chimera formation have also been proposed such as minimizing the amount of template DNA in the PCR, minimizing the number of rounds of PCR, minimizing the amount of shearing in the template DNA, and using DNA polymerases that have a proof-reading ability. To minimize PCR drift, some investigators will pool technical replicate PCRs hoping to average out the drift. Other factors that have been shown to impact the formation of PCR artifacts are outside the control of an investigator including the fraction of DNA bases that are guanines or cytosines, the variation in the length of the targeted region across the community, the sequence of the DNA that flanks the template, and the genetic diversity of the community. Early investigations of the factors that lead to the formation of PCR artifacts focused on analyzing binary mixtures of genomic DNA and 16S rRNA gene fragments to explore PCR biases and chimera formation. Although these studies were instrumental in forcing researchers to be cautious about the interpretation of their results, we have a poor understanding of how these factors affect the formation of PCR artifacts in more complex communities.

The influence that the choice of DNA polymerase has on the formation of PCR artifacts has not been well studied. There has been a lot of recent interest in how the choice of the hypervariable region and data analysis pipelines impact the sequencing error rate; however, these studies use the same DNA polymerase in the PCR step and assume that the rate of misincorporation of nucleotides due to PCR are significantly smaller than those from the sequencing phase. There has been more limited interest in the impact that DNA polymerase choice has on the formation of chimeras. A recent study found differences in the number of OTUs and chimeras between normal and high fidelity DNA polymerases [@taq_Gohl_2016]. The authors of this study could reduce the difference between two polymerases by optimizing the annealing and extension steps within the PCR protocol [@taq_Gohl_2016]. Yet this optimization was specific for the community they were analyzing (i.e. captive and semi-captive red-shanked doucs) and assumes that if the two polymerases generate the same community structure that the community structure is correct. In fact, the community structure generated by both methods is not free of artifacts, but has the same artifacts. A challenge in these types of experiments is a priori knowledge of the true community representation. A mock community with known composition allows researchers to quantify the sequencing error rate, fraction of chimeras, and bias; however, mock communities have a limited phylogenetic diversity relative to natural communities. Natural communities, in contrast, have an unknown community composition making absolute measurements impossible. They can be used to validate results from mock communities and to understand the relative impacts of artifacts on the ability to differentiate biological and methodological sources of variation. Given the large number of DNA polymerases available to researchers it is unlikely that a specific recommendation is possible. Rather, the development of general best practices and understanding the impact of PCR artifacts on an analysis are desired.

This study investigated the impact of choice of high fidelity DNA polymerase and the number of rounds of amplification on the formation of PCR artifacts using a mock community and human stool samples. These PCR artifacts included (i) the effect of the polymerase on the error rate of the bases represented in the final sequences, (ii) the fraction of sequences that appeared to be chimeras and the ability to detect those chimeras, (iii) the bias of preferentially amplifying one fragment over another in a mixed pool of templates, and (iv) inter-sample variation in community structure of samples amplified with the same polymerase across the amplification process. To characterize these factors we sequenced a mock community of 8 organisms with known sequences and community structure and human fecal samples with unknown sequences and community structures. We sequenced the V4 region of the 16S rRNA genes from a mock community by generating paired 250 nt reads on the Illumina MiSeq platform. This region and sequencing approach was used because it has been shown to result in a relatively low sequencing error rate and is a widely used protocol.
To better understand the impact of DNA polymerase choice on PCR artifacts, we selected five high fidelity DNA polymerases and amplified the communities using 15, 20, 25, 30, and 35 rounds of amplification (Table 1). Collectively, our results suggest that the number of rounds and to a lesser degree the choice of DNA polymerase used in PCR impact the sequence data, the effects are consistent and are smaller than the biological differences between individuals.

\newpage


## Results

***Sequencing errors vary by the number of cycles and the DNA polymerase used in PCR.*** The presence of sequence errors can confound the ability to accurately classify 16S rRNA gene sequences and group sequences into operational taxonomic units (OTUs). More importantly, sequencing errors themselves can alter the representation of the community. Therefore, it is important to minimize the number of sequencing errors in an analysis. Using a widely-used approach that generates the lowest reported error rate, we quantified the error rate by sequencing the V4 region of the 16S rRNA genes from an 8 member mock community. We also removed any contigs that were at least three bases more similar to a chimera of two references than to a single reference sequence [ref]. Regardless of the polymerase, the error rate increased with the number of rounds of amplification. Using 30 rounds of PCR is a common approach across diverse types of samples. Among the data generated using 30 rounds of PCR the Accuprime polymerase had the highest error rate (i.e. 0.XXX%) followed by the Platinum (i.e. 0.XXX%), Phusion (i.e. 0.XXX%), Kappa (i.e. 0.XXX%), and Q5 (i.e. 0.XXX%) polymerases (Figure 1). The order of the error rates using these polymerases is consistent with previously documented reports of their per cycle error rates. When we applied the pre.clustering denoising step, which merges the counts of reads within 2 nt of a more abundant sequence, the error rates dropped considerably such that the Platinum polymerase had the highest error rate (i.e. 0.XXX%) followed by the Accuprime (i.e. 0.XXX%), Phusion (i.e. 0.XXX%), Kappa (i.e. 0.XXX%), and Q5 (i.e. 0.XXX%) polymerases (Figure 1). Although specific recommendations are difficult to make because the impact of the actual community structure and concentration of the initial DNA template are likely to have an impact on the results, it is clear that using as few PCR cycles as necessary is a good guide to minimizing the impact of polymerase on the error rate. Furthermore, using a polymerase with the lowest possible error rate is preferred.


***The fraction of sequences identified as being chimeric varies by the number of cycles and the DNA polymerase used in PCR.*** Chimeric PCR products can significantly confound downstream analyses. Although numerous bioinformatic tools exist to identify and remove chimeric sequences with high specificity, their sensitivity is relatively low and can be reduced by the presence of sequencing errors. We identified those contigs that were at least three bases more similar to a chimera of two references than to a single reference sequence. As expected from previous studies, the number of chimeras increased with rounds of amplification. Interestingly, the fraction of chimeric sequences from the mock community varied by the type of polymerase used. After 30 rounds of PCR, the Platinum polymerase had the highest chimera rate (i.e. XX%) followed by the Q5 (i.e. XX%), Phusion (i.e. XX%), Kappa (i.e. XX%), and Accuprime (i.e. XX%) polymerases. We used the UChime algorithm to detect chimeras in our pre-clustered mock community data and calculate the algorithm's sensitivity and specificity. For all polymerases, the specificity was above XX% and showed a slight association with the number of cycles used. There was considerable inter-polymerase and inter-round of amplification variation in the sensitivity of UChime to detect the chimeras from the mock community. This suggests that the residual error rate after pre-clustering the sequence data did not compromise the sensitivity. In general, the sensitivity of UChime varied between 50 and 90%. These results could be affected by the use of a single mock community with limited genetic diversity. Although we did not know the true chimera rate for our four human stool samples, we were able to calculate the fraction of sequences that UChime identified as being chimeric. These results followed those from the mock communities - additional rounds of amplification significantly increased the rate of chimeras and there was variation between the polymerases that we used. Although it is difficult to identify a feature of a polymerase that resulted in higher rates of chimeras, it is clear that using the smallest number of PCR cycles possible will minimize the impacts of chimeras.


***The bias of PCR amplification is inconsistent and subtle.*** Since researchers began using PCR to sequence 16S rRNA genes there has been concern that amplifying fragments from a mixed template pool could lead to a biased representation in the pool of products and would confound downstream analyses. The mock community was generated by mixing equal amounts of genomic DNA from each of the 8 bacteria resulting in uneven representation of the *rrn* operons across the bacteria. The vendor of the mock community subjects each lot of genomic DNA to shotgun sequencing to more accurately quantify the actual abundance of each organism in the community. We compared the expected relative abundance of the 16S rRNA genes from each bacterium in the mock community to the data we generated across rounds of amplification and polymerase. Interestingly, for some bacteria, there representation became less biased with additional rounds of PCR (e.g. *L. fermentum*), while others became more biased (e.g. *E. faecalis*), and others had little change (e.g. *B. subtilis*). The percentage of bases in the V4 region that were guanines or cytosines was not predictive of the amount of bias. Across the bacteria there was no variation in the length of their V4 regions and they each had the same sequence in the region that the primers annealed. One of the bacteria represented in the mock community, *S. enterica*, had 6 identical copies of the V4 region and 1 copy that differed from those by one nucleotide. We used the sequence data to calculate the ratio of the dominant to rare variants from *S. enterica* (Figure. XXX). With the exception of the Kappa polymerase, the other four polymerases converged to a ratio of 5.XXX. Based on the relative abundance data from the mock community it was not possible to discern the factors that result in amplification bias. As an additional test, we quantified the Bray-Curtis distance between the communities observed after 25 rounds of amplification and those at 30 and 35 rounds of amplification. For the mock community data, the distances between 25 and 35 rounds were higher than between 25 and 30 rounds for each of the polymerases by an average of 0.XX units. The Platinum polymerase varied the most across rounds of amplification (25 vs 30 rounds: 0.XX; 25 vs 35 rounds: 0.XX). In contrast, when we characterized the variation between amplification rounds using human stool samples, the distance between the 25 and 30 rounds and 25 and 35 rounds varied considerably between samples and polymerases. In general the inter-round variation was lowest for the data generated using the Kappa and Accuprime polymerases. The data generated using the Platinum polymerase was consistent across rounds, but it was more biased than these polymerases. Considering the average distance across the four samples varied between 0.XX and 0.XX, regardless of the polymerases and number of rounds of amplification, any bias due to amplification is unlikely to obscure community-wide differences between samples. It is difficult to develop a specific recommendation based on the level of bias across rounds of PCR or polymerases; however, the general suggestion is to use as few rounds of amplification as possible.


***There is little evidence of a relationship between polymerase or number of rounds of amplification on PCR drift.*** There have been concerns that the same template DNA subjected to the same PCR conditions could result in different representations of communities because of random drift. To test this, we determined the average Bray-Curtis distance between replicate reactions using the same polymerase and number of rounds of amplification. Using the mock community data there were no obvious trends. The average Bray-Curtis distance within a set of conditions varied by 0.XX to 0.XX units. Although we did not obtain replicates of each of the stool samples, the intra-sample variation for each set of conditions was consistent and varied between 0.XX and 0.XX units. These data suggest that amplicon sequencing is robust to random variation in amplification and that any differences are likely to be smaller than what is considered biologically relevant.


***The choice of polymerase or the number of rounds of amplification have little impact on the relative interpretation of community-wide metrics of diversity.*** The biases that we observed at the population and community levels appeared to be small relative to the overall differences between samples. To study this further, we calculated alpha and beta-diversity metrics using the human stool samples for each of the polymerases and rounds of amplification. We calculated the number of observed OTUs, Shannon diversity, inverse Simpson diversity index for each condition and donor. Although there were clear differences between conditions, the relative ordering of the stool samples did not meaningfully vary across conditions. We also performed a principle coordinates analysis of the Bray-Curtis distances showed distinct clusters by donor. Within each cluster there were no obvious patterns related to the polymerase or number of rounds of PCR. However, when we performed the same analysis using the mock community, there were clear shifts in the represented community structures as the number of rounds of PCR increased. Using adonis, there were statistically significant differences in the community structures for the mock and stool community data (all P<0.001). Within the mock community data, the number of rounds of PCR explained the most variation (R^2^=0.XXX) followed by the choice of polymerase (R^2^=0.XXX). Similarly, within the stool community data, the subject explained the most variation (R^2^=0.XXX), followed by the number of rounds of PCR (R^2^=0.XXX) and the choice of polymerase (R^2^=0.XXX). Together, these results indicate that for a coarse analysis of communities, the choice of number of rounds of amplification or polymerase are not important, but that they must be consistent across samples.


\newpage

## Discussion

Our results suggest that the number of rounds of PCR and to a lesser degree the choice of DNA polymerase impact the analysis of 16S rRNA gene sequence data from bacterial communities. Although it was not possible to make direct connections between PCR conditions and specific sources of bias, we were able to identify general recommendations that reduce the amount of error, chimera formation, and bias. Researchers should strive to minimize the number of rounds of PCR and should use a high fidelity polymerase. Although specific PCR conditions impact the precise interpretation of the data, the effects were consistent and were smaller than the biological differences between individuals. Based on these observations, within a study, amplicons must be generated by consistent protocols to yield meaningful comparisons. When comparing across studies, values like richness, diversity, and relative abundances must be made in relative and not absolute terms.

The observed sequencing error rates and alpha diversity metrics followed the manufacturers' measurements of their polymerases' fidelity. Accuprime and Platinum have fidelity that are approximately 10-times higher than that of Taq whereas the fidelity of Phusion, Q5, and Kappa are more than 100 times higher. Among these polymerases, the Kappa polymerase resulted in the the lowest error rate, lowest chimera rate, and least bias across rounds of PCR. Considering polymerase development is an active area of commercial development and it likely that new polymerases will come to market, it is important for researchers to understand how changing the polymerase impacts downstream analyses for their type of samples.


### PCR bias
* GC, sequence length non-issues
* Other factors to investigate include impact of phylogenetic diversity, shearing, interactions of factors probably precludes ability to make definitive decisions.
* Minimize number of rounds of amplification - bias could be due to mis-determination of relative abundances by zymo

Previous studies have reported GC dependent amplification differences [@polz_bias_1998] and we confirm that these differences still persist in high fidelity DNA polymerase when analyzing mock community samples [Figure 1]. Additionally, these specific polymerases have different sequence error rates and chimera prevalence [Figure 2]. Chimera prevalence in particular was strongly associated with the total number of OTUs detected in our mock community and varied by polymerase  [Figure 2E & 3A]. Additionally, the bacterial community composition varied by number of cycles and polymerase in these mock community samples. Clearly, it is possible to identify a large number of differences based on number of cycles and polymerase used when analyzing mock community samples. However, these differences may not be a major concern when analyzing more complex community samples.


### Conclusion
* Use multiple methods to triangulate on reality/validate claims

Although care should always be taken when choosing a polymerase for 16S rRNA gene sequencing, our observations show that the differences between a variety of polymerases are smaller than the actual biological variation in fecal communities between individuals. Therefore, if the biological signal of interest is similar to differences in fecal bacterial communities found between individuals, then the type of high fidelity DNA polymerase used will only minimally change the results.





\newpage


## Materials & Methods

#Each condition was replicated four times and reads for each condition were pooled. Paired 250-nt sequencing reads were generated on an Illumina MiSeq sequencing platform and the reads were assembled using the make.contigs command in mothur. Any contigs that contained an ambiguous base call, mapped to the incorrect region of the 16S rRNA gene, or appeared to be a contaminant were removed from subsequent analyses. The sequencing error rates of the resulting contigs were quantified using the seq.error command in mothur. This command uses the true sequences from the mock community to generate all possible chimeras and removes any contigs that were at least three bases more similar to a chimera than to a reference sequence. The command then counts the number of substitutions, insertions, and deletions in the contig relative to the reference sequence.


***Mock community and human samples.***
The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used for mock communities and the bacterial component was made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards). The theoretical abundance of 0.12 used as a comparison point for our relative abundance was based on this equal genomic DNA composition of the mock community. Fecal samples were obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN). The processing and storage of these samples were previously published [@erin_seekatz_2016]. Other than confirmation that none of these individuals had a *Clostridium difficle* infection, clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA).

***PCR protocol.***
The five different high fidelity DNA polymerases that were tested included AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA). The polymerases activation time was 2 minutes, unless a different activation was specified by the manufacturer. The annealing and extension time for Platinum and Accuprime followed a previously published protocol [@protocol_Kozich_2013] (https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md). For Kappa and Q5, the annealing and extension time also were from a previously published optimized protocol [@taq_Gohl_2016]. For Phusion, the company defined activation and annealing times were used while the extension time followed the Accuprime and Platinum settings.

The number of cycles in the PCR for mock and fecal samples started at 15 and increased by 5 up to 35 cycles, with amplicons used at each 5-step increase for sequencing. The mock communities had 4 replicates at 15, 20, 25, and 35 cycles and 10 replicates for 30 cycles for all polymerases (total samples n=130). No mock community sample had enough PCR product at 15 cycles for adequate 16S rRNA gene sequencing. The PCR of fecal DNA samples consisted of all 4 samples at 15, 20, 25, 30, and 35 cycles for each polymerase (total sample n=100).     

***Sequence processing.***
The mothur software program was used for all sequence processing steps [@mothur_schloss_2009]. The protocol has been previously published [@protocol_Kozich_2013] (https://www.mothur.org/wiki/MiSeq_SOP). Two major differences from the published protocol were the use of VSEARCH instead of UCHIME for chimera detection and the use of the OptiClust algorithm instead of average neighbor for OTU generation at 97% similarity [@vsearch_Rognes_2016; @uchime_Edgar_2011; @opticlust_Westcott_2017]. Sequence error was determined using the 'seq.error' command on mock samples to compare back to the reference 16S sequences of *P. aeruginosa*, *E. coli*, *S. enterica*, *L. fermentum*, *E. faecalis*, *S. aureus*, *L. monocytogenes*, and *B. subtilis* [@mothur_schloss_2009; @vsearch_Rognes_2016; @rdp_Cole_2013].

***Analysis workflow.***
To adjust for unequal sequencing, all samples were rarefied to 1000 sequences for downstream analysis. The analysis of the mock community of each polymerase for GC-based amplification differences, sequence error rate, number of sequences with an error, base substitution, and numbers of chimeras before and after chimera removal with VSEARCH was assessed. Additionally, the correlation between the number of chimeras and the number of OTUs was also assessed. The total number of OTUs, taxa differences, and Bray-Curtis indices were analyzed for both the fecal and mock community samples. Finally, Random Forest models were created to assess whether classification of polymerase, number of cycles, or individual could be performed best using 16S rRNA gene sequencing data. For the model used to classify individuals, all samples were included from all number of cycles and polymerases but only the individual label was used. Both number of cycles and polymerase models included all samples but only the number of cycles or polymerase label was used for each respective model. Additionally, overlap between the most important OTUs to the three models was assessed using mean decrease in accuracy (MDA).  

***Statistical analysis.***
All analysis was done with the R (v `r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. Data transformation and graphing was completed using the tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) and colors selected using the viridis package (v `r paste(packageVersion("viridis")[1])`) [@tidyverse_2017; @viridis_citation_2017]. High and low GC content was determined based on the median GC percentage of either the V4 region or the average GC content of the whole genome of the bacterial species used in the mock community. Differences in the total number of OTUs were analyzed using an ANOVA with a tukey post-hoc test. For the comparison of the number of OTUS in fecal samples the data was normalized to each individual by cycle number to account for the biological variation. Bray-Curtis distance matrices were generated using mothur. The distance matrix data was analyzed using PERMANOVA with the vegan package (v `r paste(packageVersion("vegan")[1])`) [@vegan_citation] and Kruskal-Wallis tests within R. The Random Forest models were run using the caret package (v `r paste(packageVersion("caret")[1])`) [@caret_citation]. A total of 100 10-fold CV runs on different 80/20 splits of the data were run to generate a range of the Logloss value. The probability of a correct call was obtained from this Logloss value by taking the negative natural logarithm. For both error and chimera analysis, samples were tested using Kruskal-Wallis with a Dunns post-hoc test. Where applicable, correction for multiple comparison utilized the Benjamini-Hochberg method [@benjamini_controlling_1995].    

***Reproducible methods.***
The code and analysis for this study can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. The raw sequences can be found on the SRA at the following accession number SRP132931.


\newpage



## Acknowledgements

The authors would like to thank all the study participants in ERIN whose samples were utilized. We also would like to thank Judy Opp and April Cockburn for their effort in sequencing the samples as part of the Microbiome Core Facility at the University of Michigan. Additional thanks to members of the Schloss lab and Dr. Marcy Balunas for reading earlier drafts of the manuscript and providing helpful critiques. Salary support for Marc A. Sze came from the Canadian Institute of Health Research and NIH grant UL1TR002240. Salary support for Patrick D. Schloss came from NIH grants P30DK034933 and 1R01CA215574.   

\newpage

## References

<div id="refs"></div>

\newpage


**Figure 1: Relative abundance differences due to GC is consistent across polymerases.** At each number of cycle the points and lines represent the median and with the minimum and maximum relative abundance respectively. Regardless of the number of cycle used a consistent difference between high and low GC bacteria was observed. The dotted line represents the actual relative abundance that each bacterium should be at within the mock community sample.  

**Figure 2: Error rate and chimera prevalence vary by polymerase and affect the number of observed OTUs.** A) The error bars represent the 75% interquartile range of the median error rate. B) Percentage of chimeric sequences without the removal of chimeras with VSEARCH. C) Percentage of chimeric sequences with the removal of chimeras with VSEARCH. D) The total percent of chimeric sequences removed with VSEARCH by cycle number. The error bars represent the 75% interquartile range of the median. E) Chimera prevalence and the the observed number of OTUs are strongly correlated in mock community samples.   

**Figure 3: Subtle differences  based on number of cycles and polymerase used are detected in bacterial community composition of mock samples.** A) The range in the number of OTUs detected in the mock samples increased as cycle number increased. This range was larger for specific HiFi DNA polymerases. The points represent the median number of OTUs and the lines represent the range from the minimum to maximum number of OTUs detected within the four technical replicates. The dotted black line represents the number of OTUs detected when only the references sequences for the mock community are clustered. A) Within replicate difference based on the next 5-cycle PCR interval in mock samples. The lines represent the range of the minimum and maximum Bray-Curtis index value for each PCR 5-cycle increment comparison. The closer a sample is to a Bray-Curtis index of 1.00 the more dissimilar the bacterial community is of the two compared number of cycles.

**Figure 4: Subtle differences based on number of cycles and polymerase used are detected in bacterial community composition of fecal samples.** A) The range in the number of OTUs detected in the different fecal samples increased as cycle number increased. This range was larger for specific HiFi DNA polymerases. The points represent the median number of OTUs and the lines represent the range from the minimum to maximum number of OTUs detected within the four fecal samples. B) Within person differences based on the next 5-cycle PCR interval in fecal samples. The points represent the median Bray-Curtis index for the samples. The lines represent the range of the minimum and maximum Bray-Curtis index value for each PCR 5-cycle increment comparison. The closer a sample is to a Bray-Curtis index of 1.00 the more dissimilar the bacterial community is of the two compared number of cycles.

**Figure 5: Important OTUs for the number of cycles and polymerase models are not the most important OTUs in the model classifying individuals.** Color highlights the top 10 OTUs in number of cycles and polymerase used and where they fall in the model used to classify individuals. The majority of these top 10 OTUs have 0 importance to the model used to classify individuals.    



\newpage


**Figure S1: No preference for specific base subsitutions was observed across the polymerases in mock community samples.**  
