---
title: ''
csl: peerj.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr", "FSA"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read in the meta data file
metadata <- read_csv("../data/process/tables/meta_data.csv")

# Read in fecal count data
fecal_numOTU_data <- read_csv("../data/process/tables/fecal_sub_sample_1000_count_table.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5")), 
         cycles = factor(cycles, 
                            levels = c("15x", "20x", "25x", "30x", "35x"), 
                            labels = c("15x", "20x", "25x", "30x", "35x"))) %>% 
  group_by(taq, cycles) %>% 
  summarise(taq_median = median(numOTUs, na.rm = T), 
            taq_min = min(numOTUs, na.rm = T), 
            taq_max = max(numOTUs, na.rm = T)) %>% 
  mutate(range_diff = taq_max - taq_min)
  
# Read in fecal sample number of OTU results
fs_anova_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_anova_results.csv"))
fs_tukey_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_tukey_results.csv")) %>% 
  rename(padj = `p adj`)


# Read in the Mock count data
mock_num_otu_data <- read_csv("../data/process/tables/mock_sub_sample_1000_count_table.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5"))) %>% 
  group_by(taq, cycles) %>% 
  summarise(taq_median = median(numOTUs, na.rm = T), 
            taq_min = min(numOTUs, na.rm = T), 
            taq_max = max(numOTUs, na.rm = T)) %>% 
  mutate(range_diff = taq_max - taq_min)

ref_mock_otu_num <- read_tsv("../data/process/zymo_mock_only.shared") %>% 
  select(numOtus)


# Read in mock sample number of OTU results
mock_anova_summary <- as.data.frame(read_csv("../data/process/tables/mock_overall_anova_results.csv"))
mock_tukey_summary <- read_csv("../data/process/tables/mock_overall_tukey_results.csv")

# Read in error rate results
krsukal_error_rate <- read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")
dunn_error_rate <- read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")

# Read in sequence error count results
kruskal_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")
dunn_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")

# Read in chimera prevalence results
kruskal_chimera_prev <- as.data.frame(read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv"))
dunn_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")

# Read in results from PERMANOVA and bray analysis
bray_sim_to_prev_cycle_kruskal_results <- as.data.frame(read_csv("../data/process/tables/bray_sim_to_prev_cycle_kruskal_results.csv"))
bray_permanova_by_taq_results <- as.data.frame(read_csv("../data/process/tables/bray_permanova_by_taq_results.csv"))

# Read in actual error data
error_data <- read_csv("../data/process/tables/mock_chimera_error_summary.csv") %>% 
                       filter(total_seqs >= 1000)

error_summary <- error_data %>% 
  group_by(taq, cycles) %>% 
  summarise_at(vars(mean_error, chimera_prevalence, seq_error_prevalence), funs(median, min, max)) %>% 
  select(taq, cycles, contains("chimera"), contains("mean_error"), contains("seq_error"))
  

# Vector of sub samples used
dist_data <- read_csv("../data/process/tables/bray_5_cycle_dist_1000_data.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5"))) %>% 
  group_by(taq, cycle_compare) %>% 
  summarise(dist_median = median(distance, na.rm = T), 
            dist_min = min(distance, na.rm = T), 
            dist_max = max(distance, na.rm = T)) %>% 
  mutate(range_diff = dist_max - dist_min) %>% 
  ungroup() %>% 
  filter(cycle_compare != "25to35")


fecal_kruskal_test_by_cycle_compare_summary <- dist_data %>% 
  nest() %>% 
  mutate(kruskal_analysis = map(data, ~kruskal.test(dist_median ~ factor(cycle_compare), data = .x)), 
         summary_data = map(kruskal_analysis, broom::tidy)) %>% 
  select(summary_data) %>% unnest(summary_data)
  
dunns_test_by_cycle_compare_summary <- dist_data %>% 
  nest() %>% 
  mutate(dunn_analysis = map(data, ~dunnTest(dist_median ~ factor(cycle_compare), data = .x, method = "bh")[["res"]])) %>% 
  select(dunn_analysis) %>% unnest() %>% 
  arrange(Comparison)

mock_dist_data <- read_csv("../data/process/tables/mock_bray_5_cycle_dist_1000_data.csv") %>% 
  mutate(taq = factor(taq, 
                      levels = c("ACC", "K", "PHU", "PL", "Q5"), 
                      labels = c("Accuprime", "Kappa", "Phusion", "Platinum", "Q5"))) %>% 
  filter(!is.na(distance)) %>% 
  group_by(taq, cycle_compare) %>% 
  summarise(dist_median = median(distance, na.rm = T), 
            dist_min = min(distance, na.rm = T), 
            dist_max = max(distance, na.rm = T)) %>% 
  mutate(range_diff = dist_max - dist_min) %>% 
  ungroup() %>% 
  filter(cycle_compare != "25to35")

mock_kruskal_test_by_cycle_compare_summary <- mock_dist_data %>% 
  nest() %>% 
  mutate(kruskal_analysis = map(data, ~kruskal.test(dist_median ~ factor(cycle_compare), data = .x)), 
         summary_data = map(kruskal_analysis, broom::tidy)) %>% 
  select(summary_data) %>% unnest(summary_data)

# Read in the regression data
r2_table <- read_csv("../data/process/tables/r2_otu_vs_chimera_summary.csv")

# Read in GC data
gc_data <- read_csv("../data/process/tables/v4_gc_wilcox_test_summary.csv") %>% 
  filter(cycles != "fifteen") %>% 
  mutate(bh = p.adjust(p.value, method = "BH"))

gc_genome_data <- read_csv("../data/process/tables/whole_genome_gc_wilcox_test_summary.csv") %>% 
  filter(cycles != "fifteen") %>% 
  mutate(bh = p.adjust(p.value, method = "BH"))

gc_summary <- read_csv("../data/process/tables/gc_summary_table.csv")

# Read in the taxa data
fecal_otu_tests <- read_csv("../data/process/tables/kruskal_otu_polymerase_fecal_sample_diffs_summary.csv") %>% 
  arrange(p.value)

mock_otu_tests_kruskal <- read_csv("../data/process/tables/kruskal_otu_polymerase_mock_diffs_summary.csv") %>% 
  arrange(bh)

mock_otu_tests_dunns <- read_csv("../data/process/tables/dunns_sig_otu_polymerase_mock_diffs_summary.csv")

# Read in RF model data
dunn_rf_data <- read_csv("../data/process/tables/dunn_rf_model_test_summary.csv")

```



# Assessing the Differences in 16S rRNA Gene Sequencing Due to High Fidelity DNA Polymerase 

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, amplification, and sequencing steps. At each of these stages error can be introduced that will change the underlying bacterial community composition results. In this study, we focus on the amplification step's contribution to this overall error. To accomplish this we assessed 16S rRNA gene sequencing results in human fecal and mock community samples after using different high fidelity (HiFi) DNA polymerases and number of amplification cycles.  

**Methods.** We extracted DNA from fecal samples (n=4) using a PowerMag DNA extraction kit with a 10 minute bead beating step and amplified at 15, 20, 25, 30, and 35 cycles using Accuprime, Kappa, Phusion, Platinum, or Q5 HiFi DNA polymerase. Amplification of mock communities (technical replicates n=4) consisting of previously isolated whole genomes of 8 different bacteria used the same approach. We first assessed GC dependent differences, error rate, sequence error prevalence, chimera prevalence, and correlation between chimera prevalence and number of Operational Taxonomic Units (OTUs) by polymerase and number of cycles. Next, differences in the number of OTUs was examined based on the polymerase and number of cycles used. Additionally, differences in the bacterial community composition by the Bray-Curtis index also was assessed based on polymerase and number of cycles. Based on these findings individual taxa differences based on polymerase and number of cycles was investigated. Finally, Random Forest models were created to test whether the bacterial community was better at classifying polymerases, number of cycles, or individual donor. We also assessed whether the most important taxa in the polymerase and number of cycle Random Forest models were also important in the model for individual donors.    

**Results.** Predictably, we found noticeable differences in relative abundance based on high and low GC content (P-value $\leqslant$ `r round(max(filter(gc_data, bh < 0.05) %>% pull(bh)), digits = 2)`). Chimera prevalence in mock communities varied by polymerase with differences being most notable at 35 cycles (Kappa = `r round((filter(error_summary, cycles == "35x", taq == "K") %>% pull(chimera_prevalence_median))*100, digits = 2)`% (median) versus Platinum = `r round((filter(error_summary, cycles == "35x", taq == "PL") %>% pull(chimera_prevalence_median))*100, digits = 2)`%) and this variation persisted after chimera removal using VSEARCH. We also observed positive correlations between chimera prevalence and the number of OTUs with Platinum having the highest (R^2^ = `r max(pull(r2_table, r2))`) and Kappa having the worst (R^2^ = `r min(pull(r2_table, r2))`). When analyzing mock community samples the variation in the number of OTUs detected by the polymerases was observable as early as 20 cycles (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`). There also was a large range in the number of OTUs amplified by the polymerases at 35 cycles (Accuprime = `r filter(mock_num_otu_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_min)` - `r filter(mock_num_otu_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_max)` versus Phusion = `r filter(mock_num_otu_data, cycles == "35x", taq == "Phusion") %>% pull(taq_min)` - `r filter(mock_num_otu_data, cycles == "35x", taq == "Phusion") %>% pull(taq_max)`). When analyzing fecal samples we observed similar differences in the number of OTUs. However, the median number of OTUs only varied by HiFi DNA polymerase used at 35 cycles and not at 20 cycles like the mock community (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001). Random Forest models were most successful at classifying individual donor samples rather than polymerase or number of cycles used (P-value $\leqslant$ `r format(max(filter(dunn_rf_data, str_detect(var1, "fecal"), str_detect(var2, "fecal")) %>% pull(bh)), scientific = T, digits = 3)`). Additionally, the most important OTUs in the polymerase and number of cycle models were not the most important in the individual donor sample model.

**Conclusions.** Although there are 16S rRNA gene sequencing differences based on polymerase and number of cycles used, they are small with respect to the biological differences between fecal bacterial communities within indivdiuals. Collectively, these results provide evidence that a real biological difference between groups, based on 16S rRNA gene sequencing of fecal samples, should be detectable regardless of polymerase and number of cycles used.  



\newpage


## Introduction

The bacterial community is reported to vary between case and control for a number of diseases [@Turnbaugh2008; @Baxter2016; @Sze2015; @Bonfili2017]. However, for diseases like obesity, the taxa identified have varied widely depending on the study [@Turnbaugh2008; @Zupancic2012]. Some of this variation could be due to error introduced during the 16S rRNA gene sequencing workflow. A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, PCR, and sequencing steps. The preservation and extraction stages of the 16S rRNA gene sequencing workflow have been the most extensively studied [@contamination_Salter2014; @preservation_Song_2016; @storage_Bassis_2017; @review_Kim_2017]. For preservation and extraction stages of the workflow, it has been consistently found that there are errors based on the kits used, but that these differences are smaller than the overall biological difference measured between samples with different kits [@preservation_Song_2016; @storage_Bassis_2017]. Since these studies use the same PCR approach while varying preservation or extraction method, the contribution of PCR error to this overall workflow is not well characterized.

There is a large body of literature that shows there are errors due to primer and number of cycles chosen for the PCR stage of 16S rRNA gene sequencing [@Eckert1991; @Burkardt2000]. Primers have variable region dependent binding affinities which causes an inability to detect specific bacteria (e.g. V1-V3 does not detect *Haemophilus influenzae* and V3-V5 does not detect *Propionibacterium acnes*) [@Meisel2016; @Sze2015 (Table S4)]. Another source of error is the selective amplification of AT-rich over GC-rich sequences which exaggerate the difference between 16S rRNA genes higher in AT versus those higher in GC content [@polz_bias_1998]. Many of these sources of error are made worse as the number of cycles increases [@Wang1996; @Haas2011; @Kebschull2015]. For example, both amplification error and non-specific amplification (e.g. incorrect amplicon size products) also can increase as the number of cycles increases. This will increase the number of Operational Taxonomic Units (OTUs) observed and drastically change the values obtained from commonly used diversity measures [@BautistadelosSantos2016; @Acinas2005]. Additionally, as the number of cycles increases more chimeras can form from an aborted extension step that causes a priming error and subsequent secondary extension [@Haas2011]. These chimeras will artificially increase community diversity by increasing the number of OTUs that are observed [@Haas2011]. In addition to these sources of errors, there also are multiple families of DNA polymerases that have their own error rate and proof reading capacity [@polymerase_Ishino_2014]. Interestingly, the influence that these different DNA polymerases can have on the observed 16S rRNA gene sequencing results have not been well studied like some of the other sources of PCR-based bias.     

A recent study found differences in the number of OTUS and chimeras between normal and high fidelity DNA polymerases [@taq_Gohl_2016]. The authors could reduce the difference between the two polymerases by optimizing the annealing and extension steps within the PCR protocol [@taq_Gohl_2016]. Yet, within this study there was no comparison made between different high fidelity DNA polymerases. Additionally, the more complex fecal samples that were analyzed had different methodolgies applied to the two different groups tested, which could exagerate the actual biological differences detected. Due to this gap, it is natural to extend this line of inquiry. First, for detecting whether there are high fidelity polymerase and number of cycle dependent error. Second, for examining whether these differences are enough to obscure smaller biological signals than what has been previously reported [@taq_Gohl_2016]. Thus, this study will investigate whether differences in high fidelity DNA polymerases are due to error and whether these differences obscure the biological variation in the fecal bacterial community composition between individuals. We will accomplish this by examining GC-based amplification error, sequence error (e.g. subsitutions), chimera prevalence, the number of OTUs, Bray-Curtis index, and OTU-based differences at varying number of cycles in five different high fidelity DNA polymerases. Additionally, we also used Random Forest models to classify number of cycles, polymerase, and individuals to assess which model was best when using 16S rRNA gene sequencing data.     


\newpage

## Materials & Methods

***Human and mock samples.***
Fecal samples were obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN). The processing and storage of these samples were previously published [@erin_seekatz_2016]. Other than confirmation that none of these individuals had a *Clostridium difficle* infection, clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used for mock communities and was made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards).

***PCR protocol.***
The five different high fidelity DNA polymerases (hereto referred to as polymerases) that were tested included AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA). The polymerases activation time was 2 minutes, unless a different activation was specified by the manufacturer. The annealing and extension time for Platinum and Accuprime followed a previously published protocol [@protocol_Kozich_2013] (https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md). For Kappa and Q5, the annealing and extension time also were from a previously published protocol [@taq_Gohl_2016]. For Phusion, the company defined activation and annealing times were used while the extension time followed the Accuprime and Platinum settings. 

The number of cycles in the PCR for fecal and mock samples started at 15 and increased by 5 up to 35 cycles, with amplicons used at each 5-step increase for sequencing. The PCR of fecal DNA samples consisted of all 4 samples at 15, 20, 25, 30, and 35 cycles for each polymerase (total sample n=100). The mock communities had 4 replicates at 15, 20, 25, and 35 cycles and 10 replicates for 30 cycles for all polymerases (total samples n=130). No mock community sample had enough PCR product at 15 cycles for adequate 16S rRNA gene sequencing.    

***Sequence processing.***
The mothur software program was used for all sequence processing steps [@mothur_schloss_2009]. The protocol has been previously published [@protocol_Kozich_2013] (https://www.mothur.org/wiki/MiSeq_SOP). Two major differences from the published protocol were the use of VSEARCH instead of UCHIME for chimera detection and the use of the OptiClust algorithm instead of average neighbor for OTU generation at 97% similarity [@vsearch_Rognes_2016; @uchime_Edgar_2011; @opticlust_Westcott_2017]. Sequence error was determined using the 'seq.error' command on mock samples to compare back to the reference 16S sequences of *P. aeruginosa*, *E. coli*, *S. enterica*, *L. fermentum*, *E. faecalis*, *S. aureus*, *L. monocytogenes*, and *B. subtilis* [@mothur_schloss_2009; @vsearch_Rognes_2016; @rdp_Cole_2013]. For simplicity, the use of the term 'error' within the context of this study moving forward encompasses changes in the downstream sequencing results due to the amplification step (e.g. higher GC content genomes, base subsitutions, deletions, insertions, non-specific amplification, chimeras, etc.).  

***Analysis workflow.***
To adjust for unequal sequencing, all samples were rarefied to 1000 sequences for downstream analysis. Analysis of the mock community of each polymerase for GC-based amplification differences, sequence error rate, number of sequences with an error, base substitution, and numbers of chimeras before and after chimera removal with VSEARCH was assessed. Additionally, the correlation between the number of chimeras and the number of OTUs was also assessed. The total number of OTUs, taxa differences, and Bray-Curtis indices were analyzed for both the fecal and mock community samples. Finally, Random Forest models were created to assess whether classification of polymerase, cycles, or individual could be performed best using 16S rRNA gene sequencing data. Additionally, overlap between the most important OTUs to the three models was assessed using mean decrease in accuracy (MDA).  

***Statistical analysis.***
All analysis was done with the R (v `r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. Data transformation and graphing was completed using the tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) and colors selected using the viridis package (v `r paste(packageVersion("viridis")[1])`) [@tidyverse_2017; @viridis_citation_2017]. High and low GC content was determined based on the median GC percentage of either the V4 region or the whole genome of the bacterial species used in the mock community. Differences in the total number of OTUs were analyzed using an ANOVA with a tukey post-hoc test. For the comparison of the number of OTUS in fecal samples the data was normalized to each individual by cycle number to account for the biological variation. Bray-Curtis distance matrices were generated using mothur. The distance matrix data was analyzed using PERMANOVA with the vegan package (v `r paste(packageVersion("vegan")[1])`) [@vegan_citation] and Kruskal-Wallis tests within R. The Random Forest models were run using the caret package (v `r paste(packageVersion("caret")[1])`) [@caret_citation]. A total of 100 10-fold CV runs on different 80/20 splits of the data was run to generate a range of the Logloss value. The probability of a correct call was obtained from this Logloss value by taking the negative natural logrithim. For both error and chimera analysis, samples were tested using Kruskal-Wallis with a Dunns post-hoc test. Where applicable, correction for multiple comparison utilized the Benjamini-Hochberg method [@benjamini_controlling_1995].    

***Reproducible methods.***
The code and analysis can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. The raw sequences can be found on the SRA at the following accession number SRP132931.


\newpage


## Results

***Mock community relative abundance differs by GC content.***
There was a significant difference in relative abundance between high/low GC content based on either the V4 16S rRNA gene region or the whole genome [Figure 1 & Table S1]. However, using only the GC content of the V4 region resulted in less differences than when using the whole genome [Table S1]. The differences between high and low GC content groups were significant as early as 20 cycles in all polymerases except Q5 [Table S1]. The highest (*Staphylococcus*) and lowest (*Pseudomonas*) GC content bacteria were the most divergent  from the expected relative abundance of 0.12 [Figure 1]. These observations suggest that the high fidelity DNA polymerases tested do not prevent GC-based amplification error.   


***Sequence error varies by polymerase and is consistent across the number of cycles used.***
The median error rate varied by polymerase, with Kappa having the highest error rate of all the polymerases across the number of cycles [Figure 2A & Table S2]. Thus, the majority of the differences observed across the number of cycles was between Kappa and other polymerases [Figure 2A & Table S3]. The total sequences that contained at least one error was also polymerase dependent, with the majority of differences being between Kappa or Accuprime and the other polymerases [Table S2 & S3]. These differences in error rates were not due to polymerase dependent differences in base substitution rate [Figure S1]. Collectively, the results suggest that sequence error is dependent on polymerase, persists across the number of cycles used, and are not due to any bias towards a specific base substitution.



***Prevalence of chimeric sequences are polymerase dependent and correlate with the number of OTUs.***  Based on the previous results, we examined whether chimeras also were dependent on polymerase and whether this could affect the total number of OTUs. We observed significant differences in the chimera prevalence based on polymerase across the number of cycles used (P-value `r ifelse(max(filter(kruskal_chimera_prev)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S2]. Differences in chimera prevalence between Platinum and all other polymerases accounted for the majority of these differences [Table S3]. Accuprime^TM^ had the lowest chimera prevalence of all polymerases regardless of whether chimera removal with VSEARCH was used [Figure 2B & 2C]. The number of cycles used clearly increased chimera prevalence for all polymerases but the rate of increase differed [Figure 2B & 2C]. Additionally, there was a plateau in the total percent of chimeras that were removed that was similar for all polymerases [Figure 2D]. A positive correlation was observed between chimeric sequences and the number of OTUs for all polymerases [Figure 2E]. This positive correlation was strongest for Accuprime^TM^, Platinum, and Phusion [Figure 2E]. This data suggests that chimera prevalence depends on polymerase, is made worse by increasing the number of cycles, and confirms that the number of OTUs is dependent on the prevalence of these chimeric sequences. 



***The number of OTUs generated are dependent on polymerase used.***
In addition to the sequence error and chimera differences between polymerases, we also investigated whether there were polymerase or number of cycle dependent effects on the observed number of OTUs. There was a polymerase dependent difference in the range of the number of OTUs in the mock community samples [Figure 3A]. The closest any of the polymerases came to a total of `r pull(ref_mock_otu_num, numOtus)` OTUs, created by running the mock reference 16S sequences through mothur processing, was at 25 and 30 cycles [Figure 3A]. Differences between polymerases for the number of OTUs created were observed as early as 20 cycles in the mock community (F-stat = `r round(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "F.value"], digits = 2)`, P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`) [Table S4]. Using a Tukey post-hoc test, the majority of differences for the number of OTUs detected in the mock community was largely due to Kappa and Platinum versus the other polymerases across the number of cycles used [Table S5]. Similarily, differences in the range of the number of OTUs detected for fecal samples was dependent on the polymerase used (e.g. Accuprime at 35 cycles = `r filter(fecal_numOTU_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "35x", taq == "Accuprime") %>% pull(taq_max)` versus Phusion at 35 cycles = `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_max)`) [Figure 3B]. There was also a trend for lower number of cycles (15-20) to result in a reduced range in the number of OTUs versus higher number of cycles (25, 30, and 35) for all polymerases (e.g. Phusion at 15 cycles = `r filter(fecal_numOTU_data, cycles == "15x", taq == "Phusion") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "15x", taq == "Phusion") %>% pull(taq_max)` versus Phusion at 35 cycles = `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_min)` - `r filter(fecal_numOTU_data, cycles == "35x", taq == "Phusion") %>% pull(taq_max)`) [Figure 3B]. Unlike mock community samples, an overall difference in the number of OTUs was only detected between polymerases at 35 cycles in fecal samples (F-stat > `r round(min(filter(fs_anova_summary, cycle == "35x", sub_sample_level == 1000)[, "F.value"]), digits = 2)`, P-value = `r format(filter(fs_anova_summary, cycle == "35x", sub_sample_level == 1000)[, "bh"], digits = 2, nsmall = 2, scientific = T)`) [Table S4], and a Tukey post-hoc test failed to identify any differences between specific polymerase groups (P-value `r ifelse(min(filter(fs_tukey_summary, sub_sample_level == 1000)[, "padj"]) > 0.05,  invisible(">"), invisible("check data"))` 0.05) [Table S5]. Overall, mock communities showed a noticeably larger difference between polymerases in the number of OTUs than actual fecal samples [Figure 3 & Table S4-S5]. This was consistent across the number of amplification cycles tested [Figure 3]. Based on these observations, it is clear that larger differences in total number of OTUs based on polymerases are found in mock versus fecal communities.    


***The bacterial communities generated by the polymerases were similar but varied by number of cycles.***
Next, we investigated the effect polymerases and number of cycles had on the bacterial community composition in more detail by using the Bray-Curtis index. We observed no difference in the bacterial community composition between any measure tested when using the mock community (P-value `r ifelse(pull(mock_kruskal_test_by_cycle_compare_summary, p.value) < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 4A]. With respect to fecal samples, within each respective 5-cycle increment comparison there was no difference in Bray-Curtis index between the polymerases (P-value `r ifelse(min(filter(bray_sim_to_prev_cycle_kruskal_results, sub_sample_level == 1000)[, "bh"], na.rm = T) > 0.05, invisible(">"), invisible("check data"))` 0.05) [Figure 4B]. Using PERMANOVA to test for community differences based on any of the number of cycles within polymerases, only Phusion had cycle dependent differences (P-value = `r format(filter(bray_permanova_by_taq_results, sub_sample_level == 1000, taq == "PHU")[, "bh"], digits = 1, nsmall = 1)`). Additionally, Phusion was one of two polymerases that had enough sequences to be rarefied to 1000 sequences at 15 cycles. There was an overall decrease in Bray-Curtis index by cycle comparison group (e.g. the 15 cycle versus 20 cycle group compared to the 30 cycle versus 35 cycle group) (P-value `r ifelse(pull(fecal_kruskal_test_by_cycle_compare_summary, p.value) < 0.01, invisible("<"), invisible(">"))` 0.01) [Figure 4B]. Using a Dunn's post-hoc test the 15 cycle versus 20 cycle and 20 cycle versus 25 cycle comparison groups had a higher Bray-Curtis index than the 30 cycle versus 35 cycle comparison group (P-value `r ifelse(min(pull(dunns_test_by_cycle_compare_summary, P.adj)) < 0.05, invisible("<"), invisible(">"))` 0.05). Overall, these data suggest that the number of cycles can change the bacterial community independent of the polymerase used to generate the sequences.


***Polymerase dependent differences in taxa are more pronounced in mock versus fecal samples.***
Finding minimal differences in overall bacterial community composition, we next examined if specific taxa vary depending on polymerase and number of cycles. Further investigation of mock community samples found that OTUs taxonomically classified to *Salmonella*, *Escherichia*, *Enterococcus*, and *Staphylococcus* differed based on polymerase used [Table S6 & S7]. These specific OTUs were all members of the mock community and the differences were most pronounced at 25 and 30 cycles [Table S7]. The majority of the differences between taxa were due to differences between Kappa and the other polymerases [Table S7]. When differences in OTUs based on polymerase were investigated for fecal samples, no differences could be found at any number of cycles (P-value $\geqslant$ `r round(min(pull(fecal_otu_tests, p.value)), digits = 2)`, corrected P-value = `r round(min(pull(fecal_otu_tests, bh)), digits = 2)`). These results suggest that taxa specific differences in mock community samples do not carry over to actual fecal community samples.   


***Variation due to polymerase and number of cycles is dwarfed by the biological variation of an individual.***
Using 16S rRNA gene sequencing data there was a large disparity in a model's ability to correctly classify samples based on polymerase used, number of cycles, and individual [Figure 5A]. The model classification success for an individual was significantly better than all other models tested [Figure 5A & Table S8]. Additionally, the top 10 most impotant variables to the polymerase and number of cycle models were not the most important variables in the model classifying individuals [Figure 5B]. This observation shows that OTUs that change due to the number of cycles and polymerase used are not the same as OTUs that differentiate between indvidiuals [Figure 5B]. Although there are a variety of errors associated with the amplification process, they are smaller than the bacterial community variation between indviduals.   


\newpage

## Discussion

Previous studies have reported GC dependent amplification differences [@polz_bias_1998] and we confirm that these differences still persist in high fidelity DNA polymerase when analyzing mock community samples [Figure 1]. Additionally, these specific polymerases have different sequence error rates and chimera prevalence [Figure 2]. Chimera prevalence in particular was strongly associated with the total number of OTUs detected in a mock community [Figure 2E]. This total number of OTUs also varied by polymerase used for these mock community samples [Fiugre 3A]. The range in the number of OTUs observed increased with the number of cycles and the change in range was dependent on the polymerase for both mock community and fecal samples [Figure 3]. However, these differences did not translate to large bacterial community composition changes based on polymerase [Figure 4]. Even though there were bacterial community composition changes based on the number of cycles they occured independently of the polymerase used [Figure 4]. Additionally, there were no OTU-based differences observed within the number of cycles between any of the polymerases for fecal samples. When Random Forest models were created to classify number of cycles, polymerase, and individual based on the 16S rRNA gene sequencing, the model performance for individuals was significantly greater than all other models tested [Figure 5A]. In addition to the better model performance of individuals, the most important OTUs to this model were not the most important OTUs for the other models tested [Figure 5B]. Our observations show that differences between high fidelity DNA polymerases can be found. However, when using 16S rRNA gene sequencing these technical differences are dwarfed by biological differences between fecal bacterial communities within individuals.    

The small differences observed due to error rate and chimera prevalence may be due to the actual DNA polymerase family being used within the PCR mix. DNA polymerases from distinct families have different binding affinities and error correction capacity [@polymerase_Ishino_2014]. However, based on the observations within our study this is unlikely to be the only contributor. Within our study the highest and lowest chimera prevalence both belonged to a family A polymerase (Platinum and Accuprime^TM^ respectively) [@polymerase_Ishino_2014]. Additionally, based on the information supplied by the respective manufacturers, the differences between the two PCR mixtures are not immediately apparent. Both PCR mixtures contain a recombinant *Taq* DNA polymerase, a *Pyrococcus* spp GB-D polymerase and a platinum *Taq* antibody. Since it is not possible to know everything about the mixture beyond what was willingly provided by the manufacturer, it is possible that differences in how the recombinant *Taq* was generated or other compounds within the PCR mixture could be a contributing factor for the differences in error rate and chimera prevalence. Beyond the choice of the type of polymerase, there may be other ways to reduce the affect of polymerase dependent error rates and chimera prevalence on the downstream results. 

Based on polymerase used, our observations generally found no difference when using the Bray-Curtis index to measure the bacterial community. Specifically, there was no difference in distance between successive 5-cycle increments within samples between the polymerases when using the Bray-Curtis index [Figure 4]. One possible reason for this outcome is that many of the OTUs generated by polymerase dependent error rates and chimera prevalence are likely not highly abundant, allowing the Bray-Curtis index to be able to successfully down-weight these OTUs [@bc_index_Minchin1987]. The choice of downstream diversity metric could be an important consideration in helping to mitigate some of the observed polymerase dependent differences in error rate and chimera prevalence. Metrics that solely use presence/absence of OTUs (e.g. Jaccard [@Real1996], richness) may be less robust to polymerase dependent error rates and chimera prevalence. When choosing a distance metric, careful consideration of the biases introduced from the PCR step of the 16S rRNA gene sequencing workflow need to be taken into account. One possible way to better choose polymerases may be based on the DNA polymerase family used in the PCR mixture.

Collectively, our data shows that using distinct high fidelity DNA polymerases will result in a different number of OTUs being detected. The differences in the number of OTUs are primarily due to chimera prevalence and sequence error rates that are distinct to the specific polymerase. Despite this variation in the number of OTUs, our Random Forest models were able to correctly classify individuals regardless of the number of cycles and polymerase used. These results suggest that even though there is a large variation in the high fidelity DNA polymerases used across studies, biological variation similar in effect to fecal bacterial community composition differences between individuals will still be consistent.    




```{r, echo=FALSE, message=FALSE, warning=FALSE}
#Results have an impact on downstream diversity measurements due to effects on total OTUs


#Need to cover that different types of polymerase may affect outcome
#- Accuprime HiFi from recombinant A-Family Taq DNA polymerase, Pyrococcus spp GB-D polymerase (proof-reading), platinum Taq Antibody (inhibits room temp activity)

#- Kappa HiFi from B-family DNA polymerase (proof-reading baked in), propietary antibody (inhibit room temp activity)

#- Q5 HiFi from B-family thermostable DNA polymerase with 3´→ 5´ exonuclease activity, fused to a processivity-enhancing Sso7d domain to support robust DNA amplification

#-Phusion HiFi B-family DNA polymerase a novel Pyrococcus-like enzyme fused with a processivity-enhancing domain, generate blunt-ended products.

#- Platinum HiFi recombinant A-Family Taq DNA polymerase, Pyrococcus species GB-D polymeras, Platinum Taq Antibody

```



\newpage

## Conclusion

Although care should always be taken when choosing a polymerase for 16S rRNA gene sequencing, our observations show that the differences between a variety of polymerases are dwarfed by the actual biological variation in fecal communities between individuals. Therefore, if the biological signal of interest is similar to differences in fecal bacterial communities found between individuals, then the type of high fidelity DNA polymerase used will only minimally change the results. 

\newpage



## Acknowledgements

The authors would like to thank all the study participants in ERIN whose samples were utilized. We also would like to thank Judy Opp and April Cockburn for their effort in sequencing the samples as part of the Microbiome Core Facility at the University of Michigan. Additional thanks to members of the Schloss lab and Dr. Marcy Balunas for reading earlier drafts of the manuscript and providing helpful critiques. Salary support for Marc A. Sze came from the Canadian Institute of Health Research and NIH grant UL1TR002240. Salary support for Patrick D. Schloss came from NIH grants P30DK034933 and 1R01CA215574.   

\newpage

## References

<div id="refs"></div>

\newpage


**Figure 1: Relative abundance differences due to GC is consistent across polymerases.** At each number of cycle the points and lines represent the median and with the minimum and maximum relative abundance respectively. Regardless of the number of cycle used a consistent difference between high and low GC bacteria was observed. The dotted line represents the actual relative abundance that each bacterium should be at within the mock community sample.  

**Figure 2: Error rate and chimera prevalence vary by polymerase and affect the number of observed OTUs.** A) The error bars represent the 75% interquartile range of the median error rate. B) Percentage of chimeric sequences without the removal of chimeras with VSEARCH. C) Percentage of chimeric sequences with the removal of chimeras with VSEARCH. D) The total percent of chimeric sequences removed with VSEARCH by cycle number. The error bars represent the 75% interquartile range of the median. E) Chimera prevalence and the the observed number of OTUs are strongly correlated in mock community samples.   

**Figure 3: The number and range of OTUs in mock and fecal samples varies by polymerase.** The points represent the median number of OTUs for the mock samples. The lines represent the range of the minimum and maximum number of OTUs detected within the four fecal samples. The dotted black line represents the number of OTUs detected when only the references sequences for the mock community are clustered. A) The range in the number of OTUs detected in the mock samples increased as cycle number increased. This range was also larger for specific HiFi DNA polymerases. B) The range in the number of OTUs detected in the different fecal samples increased as cycle number increased. This increased range also was larger for specific HiFi DNA polymerases.


**Figure 4: Lower five-cycle intervals have larger differences than higher five-cycle intervals using Bray-Curtis.** A) Within replicate difference based on the next 5-cycle PCR interval in mock samples. B) Within person differences based on the next 5-cycle PCR interval in fecal samples. The points represent the median Bray-Curtis index for the samples. The lines represent the range of the minimum and maximum Bray-Curtis index value for each PCR 5-cycle increment comparison. The closer a sample is to a Bray-Curtis index of 1.00 the more dissimilar the bacterial community is of the two compared number of cycles. 

**Figure 5: Biological variation in bacterial community composition due to individual was larger than number of cycles and polymerase used.** A) Using 16S rRNA gene sequencing, the classification of individuals was higher than any other model tested. The lines represent the median probability of correct classification and the gray points represent the individual probability from one run out of the 100 10-fold CV runs. B) The most important OTUs to the model classifying individuals are not the same as those that are most important for the number of cycles or polymerase used.    



\newpage


**Figure S1: No preference for specific base subsitutions was observed across the polymerases in mock community samples.**  


