---
title: ''
csl: peerj.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read in the meta data file
metadata <- read_csv("../data/process/tables/meta_data.csv")

# Read in fecal sample number of OTU results
fs_anova_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_anova_results.csv"))
fs_tukey_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_tukey_results.csv")) %>% 
  rename(padj = `p adj`)

# Read in mock sample number of OTU results
mock_anova_summary <- as.data.frame(read_csv("../data/process/tables/mock_overall_anova_results.csv"))
mock_tukey_summary <- read_csv("../data/process/tables/mock_overall_tukey_results.csv")

# Read in error rate results
krsukal_error_rate <- read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")
dunn_error_rate <- read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")

# Read in sequence error count results
kruskal_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")
dunn_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")

# Read in chimera prevalence results
kruskal_chimera_prev <- as.data.frame(read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv"))
dunn_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")

```



# What the Taq? The Influence of Different Hi-Fidelity Taq Polymerase on 16S rRNA Gene Sequencing

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** 

**Methods.**

**Results.** 

**Conclusions.** 




\newpage


## Introduction




\newpage

## Materials & Methods

***Human and Mock Samples:***
A single fecal sample was obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN) and the processing and storage of these samples have been published previously [@erin_seekatz_2016]. Clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used in this study and is made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance  (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards).

***PCR Protocol:***
The five different high fidelity (HiFi) Taq DNA polymerase that were tested were AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA). The PCR cycle conditions for Platinum and Accuprime followed a previously published protocol [@protocol_Kozich_2013] (https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md). If the HiFi Taq had a specific activation time that was different then 2 minutes that was used instead. For Kappa and Q5 the protocol previously published by Gohl and colleagues was used [@taq_Gohl_2016]. For Phusion the company defined conditions were used but the same extension time as that used for Accuprime and Platinum was used. 

The 30 cycle default was used but the cycle conditions started at 15 and increased by 5 up to 35 cycles and was used for both fecal and mock samples. The fecal PCR consisted of all 4 samples at 15, 20, 25, 30, and 35 cycles for each Taq (total samples = 100). Although, the mock communities also had 4 replicates used for 15, 20, 25, and 35 cycles, 10 replicates were used for 30 cycles for all Taq (total samples = 130). For all the mock community samples there was not enough PCR product at 15 cycles for adequate sequencing.    

***Sequence Processing:***
The mothur software program was utilized for all sequence processing steps [@mothur_schloss_2009]. The protocol followed was similar to what has been previously publsihed [@protocol_Kozich_2013] (https://www.mothur.org/wiki/MiSeq_SOP). Two major differences from the stated protocol were the use VSEARCH instead of UCHIME for chimera detection and the use of the OptiClust algorithmn instead of average neighbor for Operational Taxonomic Unit (OTU) generation [@vsearch_Rognes_2016; @uchime_Edgar_2011; @opticlust_Westcott_2017]. Sequence error was determined using the seq.error command on mock samples after chimera removal and classification to the RDP to remove non-bacterial sequences [@mothur_schloss_2009; @vsearch_Rognes_2016; @rdp_Cole_2013].  

***Statistical Analysis:***
All analysis was done with the R (v `r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. Data tranformation and graphing was completed using the tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) and colors selected using the viridis package (v `r paste(packageVersion("viridis")[1])`) [@tidyverse_2017; @viridis_citation_2017]. The total number of OTUs were analyzed using an ANOVA with a tukey post-hoc test. For the fecal samples the data was normalized to each individual by cycle number to account for the biological variation between different people. For both error and chimera analysis, samples were tested using Kruskal-Wallis with a Dunns post-hoc test. Where applicable correction for multiple comparison utilized the Benjamini-Hochberg method [@benjamini_controlling_1995].    

***Analysis Workflow:***
The total number of OTUs after sub-sampling was analyzed for both the fecal and mock community samples. From these observations we wanted to next analyze potential reasons as to why some of these differences may have occured. First, analysis of general sequence error rate, number of sequences with an error, and base subsitution were assessed in the mock community for each Taq. After assessing these errors, the total number of chimeras was determined after sequence processing. The fecal samples were analyzed at 4 different sub-sampling levels, 1000, 5000, 10000, and 15000 while the mock community samples were analysed at 3 levels, 1000, 5000, 10000.

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. The raw sequences can be found in the SRA at the following accesssion number **need to upload still**.


\newpage


## Results

***The Number of OTUs is Dependent on HiFi Taq Used:***
After normalization by individual, for each cycle number, we observed that for fecal samples the number of OTUs identified was dependent upon the HiFi Taq used and this difference increased as the depth of sub-sampling increased [Figure 1]. Lower cycle numbers (15-20) resulted in less differences between Taq while cycle numbers of 25, 30, and 35 had larger clearer defined differences [Figure 1]. Only 35 cycles had HiFi Taq differences that were significantly different at all sub-sampling levels (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001) [Table S1]. At sub-sampling depth of 5000 or higher 25 and 30 cycles had HiFi Taq differences (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "30x", sub_sample_level != 1000)[, "bh"]) < 0.05 & max(filter(fs_anova_summary, cycle == "25x", sub_sample_level != 1000)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S1]. Using a Tukey post-hoc test only at 35 cycles were significant differences found to be mainly driven by Platinum being different than all other Taq across sub-sampling depth (P-value `r ifelse(max(filter(fs_tukey_summary, padj < 0.05)[, "padj"]) < 0.05,  invisible("<"), invisible("check data"))` 0.05) [Table S2].  

This Taq dependent difference in the number of OTUs was also observed in the mock community samples with the same Taq polymerases being high (Platinum) and low (Accuprime) respectively [Figure 2 & Table S3]. Conversely, differences between HiFi Taq were observed as early as 20 cycles and a sub-sampling depth of 1000 sequences (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`) [Table S3]. Using a Tukey post-hoc test differences between Platinum and the other HiFi Taqs was the major driver of the differences seen at different cycle numbers and sub-sampling depths [Table S4]. Both fecal and mock samples consistently showed that across sub-sampling depth and cycle number the lowest number of OTUs identified was from Accuprime^TM^ while the highest was from Platinum for both fecal and mock samples [Figure 1 & 2].

***Sequence Error is Dependent on both Taq and Cycle Number Used:***
Differences by HiFi Taq in the median average per base error varied without a clear pattern across sub-sampling depth [Table S5]. Generally, the highest values were for the Kappa HiFI Taq [Figure 3A]. Sub-sampling depth seems to have little effect on this rate with both 5000 and 10000 sub-sampled sequences showing similar results [Figure 3B-C]. There were small differences between the various HiFi Taq at lower cycle number but larger differences at higher cycle number with Platinum having large differences between all other HiFi Taq [Figure 3B-C and Table S6]. 

The total seqeunces with at least one error had multiple differences at different cycle numbers and sub-sampling depth driven by large differences in Accuprime^TM^ and Platinum versus the other HiFi Taq tested [Figure S1 & Table S7 & S8]. Although Accuprime^TM^ had the lowest per base error rate it had either the largest or second largest number of sequences with at least one error regardless of cycle number or sub-sampling depth [Figure S1]. Investigation of whether there were Taq dependent effects on base subsitution found that there was no clear bias and this was indpendent of sub-sampling depth [Figure S2-S4]. Further, the variation in subsitution error seems to reduce as the sub-sampling depth increases [Figure S2-S4].


***Chimeric Sequences Corelate with OTUs and are HiFi Taq Dependent:***
After chimera removal using VSEARCH and removal of sequences that did not classify as bacteria we assessed the percentage of sequences that were still chimeric within our mock community. At all levels of sub-sampling and cycle number there were significant differences between the HiFi Taq used (P-value `r ifelse(max(filter(kruskal_chimera_prev, sub_sample_level > 500)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S9]. Using a Dunn's post-hoc test the vast majority of these differences were driven by Platinum being different then all other HiFi Taq across cycle number and sub-sampling depth [Table S10]. Generally, across sub-sampling depth and cycle number Accuprime^TM^ had the lowest chimera prevalence of all the HiFi Taq [Figure 4]. 

For all Taqs a positive correlation was observed between chimeric sequences and number of OTUs, with this correlation being strongest for Platinum and Phusion HiFi Taq [Figure 5]. In general, the R^2^ value between the number of OTUs and chimeric sequences became stronger as sub-sampling depth increased [Figure 5]. Taken together this data suggests that a strong correlation exists between the number of OTUs and chimera sequence prevalence. Interestingly, Accuprime^TM^ not only had one of the lowest prevalence of chimeric sequences but also, consistently, had the lowest correlations between the number of OTUs and chimeric sequences across sub-sampling depth [Figure 5]. 

\newpage

## Discussion



\newpage

## Conclusion



\newpage



## Acknowledgements

The authors would like to thank all the study participants ERIN whose samples were utilized. We would also like to thank Judy Opp and April Cockburn for their effort in sequencing the samples as part of the Microbiome Core Facility at the University of Michigan. Salary support for Marc Sze came from the Canadian Institue of Health Research and the Michigan Institute for Clinical and Health Research Postdoctoral Translational Scholar Program.

\newpage

## References

<div id="refs"></div>

\newpage


**Table 1: **

\footnotesize

```{r table1, echo=FALSE, message=FALSE, warning=FALSE}


```

\normalsize
\newpage

**Table 2: **

\footnotesize

```{r table2, echo=FALSE, message=FALSE, warning=FALSE}


```

\normalsize
\newpage



**Figure 1: .** 

**Figure 2: .** 

**Figure 3: .**

**Figure 4: .** 

**Figure 5: .** 

**Figure 6: .** 

**Figure 7: .**  


\newpage


**Figure S1: .**

**Figure S2: .** 

**Figure S3: .** 

**Figure S4: .** 

**Figure S5: .** 

**Figure S6: .** 

**Figure S7: .** 

\newpage
