---
title: ''
csl: peerj.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown", "stringr"))

```

```{r rdata_tables, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

#Read in the meta data file
metadata <- read_csv("../data/process/tables/meta_data.csv")

# Read in fecal sample number of OTU results
fs_anova_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_anova_results.csv"))
fs_tukey_summary <- as.data.frame(read_csv("../data/process/tables/fecal_overall_tukey_results.csv")) %>% 
  rename(padj = `p adj`)

# Read in mock sample number of OTU results
mock_anova_summary <- as.data.frame(read_csv("../data/process/tables/mock_overall_anova_results.csv"))
mock_tukey_summary <- read_csv("../data/process/tables/mock_overall_tukey_results.csv")

# Read in error rate results
krsukal_error_rate <- read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")
dunn_error_rate <- read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")

# Read in sequence error count results
kruskal_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")
dunn_seq_error_count <- read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")

# Read in chimera prevalence results
kruskal_chimera_prev <- as.data.frame(read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv"))
dunn_chimera_prev <- read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")

# Read in results from PERMANOVA and bray analysis
bray_sim_to_prev_cycle_kruskal_results <- as.data.frame(read_csv("../data/process/tables/bray_sim_to_prev_cycle_kruskal_results.csv"))
bray_permanova_by_taq_results <- as.data.frame(read_csv("../data/process/tables/bray_permanova_by_taq_results.csv"))

# Vector of sub samples used
sub_sample_level <- c("1000", "5000", "10000")

dist_data <- sapply(sub_sample_level, 
                    function(x) as.data.frame(read_csv(paste("../data/process/tables/bray_5_cycle_dist_",                                                x, "_data.csv", sep = ""))), simplify = F)

mock_dist_data <- sapply(sub_sample_level, 
                    function(x) as.data.frame(read_csv(paste("../data/process/tables/mock_bray_5_cycle_dist_", 
                                               x, "_data.csv", sep = ""))), simplify = F)

```



# High Fidelity DNA Polymerase Introduces Bias into 16S rRNA Gene Sequencing Results

\begin{center}
\vspace{25mm}

Marc A Sze${^1}$ and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI




\end{center}

Co-author e-mails:

* marcsze@med.umich.edu



\newpage
\linenumbers

## Abstract

**Background.** Using different reagents and kits at different steps of the 16S rRNA gene sequencing workflow can introduce bias by changing the observed microbial community. Although cycle number and high fidelity (HiFi) DNA polymerase are studied less often, they are still important sources of bias in this workflow. Here, we examine how both cycle number and HiFi DNA polymerase  can change the bacterial community and introduce bias to the final obtained results.  

**Methods.** DNA from fecal samples (n=4) were extracted using a PowerMag DNA extraction kit with a 10 minute bead beating step and amplified at 15, 20, 25, 30, and 35 cycles using Accuprime, Kappa, Phusion, Platinum, or Q5 HiFi DNA polymerase. Mock communities (technical replicates n=4) consisting of previously isolated whole genomes of 8 different bacteria were also amplified using the same approach. The number of OTUs (Operational Taxonomic Units) were examined for both fecal samples and mock communities. Next, Bray-Curtis index, the error rate, sequence error prevalence, and chimera prevalence were investigated. Finally, the chimera prevalence correlation with number of OTUs was assessed. 

**Results.** When analyzing fecal samples a different number of OTUs was observed between HiFi DNA polymerases at 35 cycles (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001). These HiFi dependent differences in the number of OTUs were identified as early as 20 cycles in the mock communities (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`). Chimera prevalence varied by HiFi DNA polymerase and this variation was still present after chimera removal using VSEARCH. The chimera prevalence was positively correlated with the number of OTUs and was also not affected by chimera removal with VSEARCH.   

**Conclusions.** HiFi DNA polymerase dependent differences in the number of OTUs and chimera prevalence makes comparison across studies difficult. Care should be exercised when choosing both HiFi DNA polymerase and cycle number to be used in 16S rRNA gene sequencing studies. 


\newpage


## Introduction

This study will specifically address how cycle number and high fidelity (HiFi) DNA polymerases can bias observed bacterial community results derived from 16S rRNA gene sequencing. First, it is important to differentiate between bias, reproducibility, and standardization since often times these three can be confused and used interchangeably with each other. Bias can change the observed results in a way that is reproducible and standardized. For example, if one group uses one brand of DNA extraction kit for their 16S rRNA gene sequencing, their results may be biased versus another group not using the same brand kit but within their group they can still have reproducible results. Therefore, standardization of 16S rRNA gene sequencing methods to increase reproducibility can still be problematic due to bias. Determining how different aspects of a 16S rRNA gene sequencing workflow could bias the observed results is critical for the interpretation of specific studies in the borader context of the overall field. 

A typical 16S rRNA gene sequencing workflow can be divided into preservation, extraction, PCR, and sequencing steps. Generally, not using a preservation media and leaving samples at room temperature has been shown to cause overgrowth of low abundance members of the fecal bacterial community [@Amir2017]. Similarily, this overgrowth could still occur if the preservation media does not adequately inhibit growth [@preservation_Luo_2016; @preservation_Song_2016]. Reports have also shown that changes in specific community members might occur due to differing susceptibility to freeze thaw cycles amongst microbes [@Gorzelak2015]. Additionally, reagent contamination has been shown to add community members and the contribution of these contaminant members grows larger with lower biomass samples [@contamination_Salter2014]. Recent studies have shown that the majority of these biases due to either preservation or extraction tend to be smaller than the overall biological signal being measured [@preservation_Song_2016; @storage_Bassis_2017]. However, the contribution of PCR bias to this overall workflow is not well characterized since these studies use the same PCR approach while varying preservation or extraction method. 

Identifying the biases in the PCR stage of 16S rRNA gene sequencing is important because a large body of literature shows that there are a variety of steps during PCR that can change the observed results [@Eckert1991; @Burkardt2000]. Many of these sources of biases are made worse as cycle number increases [@Wang1996; @Haas2011; @Kebschull2015]. For example, the selective amplification of AT-rich over GC-rich sequences can increase the difference between these types of 16S rRNA genes and over-emphasizing the actual difference [@polz_bias_1998]. Both amplification error and non-specific amplification (e.g. incorrect amplicon size products) can also increase as cycle number increases which can drastically change commonly used  diversity measures [@BautistadelosSantos2016; @Acinas2005]. Additionally, chimeras can form from an aborted extension followed by subsequent priming error and secondary extension and even if it has not been explicitly shown to increase with cycle number it is easy to postulate that this does happen and will also artificially increase community diversity [@Haas2011]. 

There are also intrinsic properties to primers and DNA polymerases chosen that can introduce bias. Primers have variable region dependent binding affinities for different bacteria and depending on the primer pair do not detect specific bacteria (e.g. V1-V3 does not detect *Haemophilus influenzae* and V3-V5 does not detect *Propionibacterium acnes*) [@Meisel2016; @Sze2015 (Table S4)]. Additionally, there are multiple families of DNA polymerases that have their own error rate and proof reading capcity [@polymerase_Ishino_2014]. Interestingly, the influence that these different DNA polymerases can have on the observed results have not been well studied like some of the other previously mentioned sources of PCR-based bias.     

A recent study found clear differences between normal and high fidelity (HiFi) DNA polymerase and that optimization of the PCR protocol could reduce error and chimera generation [@taq_Gohl_2016]. This study also found that regardless of DNA polymerase, the number of Operational Taxonomic Units (OTUs) or taxa generated were not easily reduced using the authors chosen bioinformatic pipeline [@taq_Gohl_2016]. It is natural to extend this line of inquiry and ask if biases in the number of OTUs and chimeras are also dependent on HiFi DNA polymerases. There is some reason to think that this may be the case since many of these HiFi DNA polymerases come from different families (e.g. *Taq* belongs to the family A polymerases) and may intrinsically have different error rates that cannot be completely removed with modifications [@polymerase_Ishino_2014]. In this study, we critically examine if any of five different HiFi DNA polymerases introduce significant biases into 16S rRNA gene surveys, if this is a cycle dependent phenomenon, and whether they can be removed using a standard bioinformatic pipeline.


\newpage

## Materials & Methods

***Human and Mock Samples:***
A single fecal sample was obtained from 4 individuals who were part of the Enterics Research Investigational Network (ERIN). The processing and storage of these samples have been previously published [@erin_seekatz_2016]. Other than confirmation that none of these individuals had a *Clostridium difficle* infection, clinical data and other types of meta data were not utilized or accessed for this study. All samples were extracted using the MOBIO^TM^ PowerMag Microbiome RNA/DNA extraction kit (now Qiagen, MD, USA). The ZymoBIOMICS^TM^ Microbial Community DNA Standard (Zymo, CA, USA) was used in this study and is made up of *Pseudomonas aeruginosa*, *Escherichia coli*, *Salmonella enterica*, *Lactobacillus fermentum*, *Enterococcus faecalis*, *Staphylococcus aureus*, *Listeria monocytogenes*, and *Bacillus subtilis* at equal genomic DNA abundance (http://www.zymoresearch.com/microbiomics/microbial-standards/zymobiomics-microbial-community-standards).

***PCR Protocol:***
The five different HiFi DNA polymerases that were tested included AccuPrime^TM^ (ThermoFisher, MA, USA), KAPA HIFI (Roche, IN, USA), Phusion (ThermoFisher, MA, USA), Platinum (ThermoFisher, MA, USA), and Q5 (New England Biolabs, MA, USA). The PCR cycle conditions for Platinum and Accuprime followed a previously published protocol [@protocol_Kozich_2013] (https://github.com/SchlossLab/MiSeq_WetLab_SOP/blob/master/MiSeq_WetLab_SOP_v4.md). The HiFi DNA polymerase activation time was 2 minutes, unless a different activation was specified. For Kappa and Q5, the protocol previously published by Gohl and colleagues was used [@taq_Gohl_2016]. For Phusion, the company defined conditions were used except for extension time, where the Accuprime and Platinum settings were used. 

Both fecal and mock samples cycle conditions started at 15 and increased by 5 up to 35 cycles with amplicons used at each 5-step increase for sequencing. The fecal PCR consisted of all 4 samples at 15, 20, 25, 30, and 35 cycles for each HiFi DNA polymerase (total samples = 100). The mock communities had 4 replicates at 15, 20, 25, and 35 cycles and 10 replicates for 30 cycles for all HiFi DNA polymerase (total samples = 130). No mock community sample had enough PCR product at 15 cycles for adequate 16S rRNA gene sequencing.    

***Sequence Processing:***
The mothur software program was utilized for all sequence processing steps [@mothur_schloss_2009]. Generally, the protocol followed what has been previously published [@protocol_Kozich_2013] (https://www.mothur.org/wiki/MiSeq_SOP). Two major differences from the stated protocol were the use of VSEARCH instead of UCHIME for chimera detection and the use of the OptiClust algorithm instead of average neighbor for OTU generation at 97% similarity [@vsearch_Rognes_2016; @uchime_Edgar_2011; @opticlust_Westcott_2017]. Sequence error was determined using the 'seq.error' command on mock samples after chimera removal and classification to the RDP to remove non-bacterial sequences [@mothur_schloss_2009; @vsearch_Rognes_2016; @rdp_Cole_2013].  

***Analysis Workflow:***
The total number of OTUs after sub-sampling was analyzed for both the fecal and mock community samples. Cycle dependent affects on Bray-Curtis indices were next assessed for the fecal samples looking at both overall cycle differences and within individual differences for the previous cycle (e.g. 20 versus 25, 25 versus 30, etc.). For these community based measures, the fecal samples were analyzed at 4 different sub-sampling sequence depths (1000, 5000, 10000, and 15000) while the mock community samples were analysed at 3 levels (1000, 5000, 10000). Next, based on these observations we wanted to analyze potential reasons for these differences. First, analysis of general sequence error rate, number of sequences with an error, and base substitution were assessed in the mock community for each HiFi DNA polymerase. After assessing these errors, the total number of chimeras was determined after sequence processing and correlations to the number of OTUs also assessed. 

***Statistical Analysis:***
All analysis was done with the R (v `r paste(version$major, ".", version$minor, sep = "")`) software package [@r_citation_2017]. Data transformation and graphing was completed using the tidyverse package (v `r paste(packageVersion("tidyverse")[1])`) and colors selected using the viridis package (v `r paste(packageVersion("viridis")[1])`) [@tidyverse_2017; @viridis_citation_2017]. Differences in the total number of OTUs were analyzed using an ANOVA with a tukey post-hoc test. For the fecal samples the data was normalized to each individual by cycle number to account for the biological variation between people. Bray-Curtis distance matrices were generated using mothur after 100 sub-samplings at 1000, 5000, 10000, and 15000 sequence depth. The distance matrix data was analyzed using PERMANOVA with the vegan package (v `r paste(packageVersion("vegan")[1])`) [@vegan_citation] and Kruskal-Wallis tests within R. For both error and chimera analysis, samples were tested using Kruskal-Wallis with a Dunns post-hoc test. Where applicable correction for multiple comparison utilized the Benjamini-Hochberg method [@benjamini_controlling_1995].    

***Reproducible Methods:***
The code and analysis can be found here https://github.com/SchlossLab/Sze_PCRSeqEffects_XXXX_2017. The raw sequences can be found in the SRA at the following accession number SRP132931.


\newpage


## Results

***The Number of OTUs is Dependent on HiFi DNA Polymerase:***
There was a HiFi DNA polymerase dependent difference that was consistent across sub-sampling [Figure 1]. Lower cycle numbers (15-20) tended to result in less differences between HiFI DNA polymerase versus higher cycle numbers (25, 30, and 35)  [Figure 1]. At 35 cycles, all sub-sampling levels had significant differences between HiFi DNA polymerases (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "35x")[, "bh"]) < 0.0001, invisible("<"), invisible("check data"))` 0.0001) [Table S1]. Some differences between HiFi DNA polymerase we identifed at 25 and 30 cycles but the sub-sampling depth had to be 5000 or higher (P-value `r ifelse(max(filter(fs_anova_summary, cycle == "30x", sub_sample_level != 1000)[, "bh"]) < 0.05 & max(filter(fs_anova_summary, cycle == "25x", sub_sample_level != 1000)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S1]. Most of the differences observed at 35 cycles were between Platinum and other HiFi DNA polymerases, based on a Tukey post-hoc test (P-value `r ifelse(max(filter(fs_tukey_summary, padj < 0.05)[, "padj"]) < 0.05,  invisible("<"), invisible("check data"))` 0.05) [Table S2]


This HiFi DNA polymerase dependent difference in the number of OTUs was also observed in the mock community samples with the same DNA polymerases having high (Platinum) and low (Accuprime) number of OTUs [Figure 2 & Table S3]. In contrast to fecal samples, differences between HiFi DNA polymerase were observed as early as 20 cycles and as low as a sub-sampling depth of 1000 sequences (P-value = `r format(filter(mock_anova_summary, sub_sample_level == 1000, cycle == "20x")[, "bh"], digits = 1, nsmall = 1)`) [Table S3]. For both different cycle numbers and sub-sampling depths, the majority of differences were between Platinum and the other HiFi DNA polymerases [Table S4]. The lowest number of OTUs identified was from Accuprime^TM^ while the highest was from Platinum and this was consistent for both fecal and mock samples across sub-sampling depth and cycle number [Figure 1 & 2]. Based on these observations it is clear that different HiFi DNA polymerase result in different total number of OTUs observed within a sample.


***Minimal Bray-Curtis Differences are Detected by Cycle Number:***
Overall, there were a few sub-sampling dependent differences in bacterial communities between cycle number . First, independent of HiFi DNA polymerase there were differences for the same fecal sample between 20 versus 25 cycles that was dependent on sub-sampling depth (sub-sampled to 1000 = `r round(median(filter(dist_data[["1000"]], cycle_compare == "20to25")[, "distance"]), digits = 2)` (`r round(quantile(filter(dist_data[["1000"]], cycle_compare == "20to25")[, "distance"])["25%"], digits = 2)` - `r round(quantile(filter(dist_data[["1000"]], cycle_compare == "20to25")[, "distance"])["75%"], digits = 2)`) (median (25% - 75% quantile)), sub-sampled to 5000 = `r round(median(filter(dist_data[["5000"]], cycle_compare == "20to25")[, "distance"]), digits = 2)` (`r round(quantile(filter(dist_data[["5000"]], cycle_compare == "20to25")[, "distance"])["25%"], digits = 2)` - `r round(quantile(filter(dist_data[["5000"]], cycle_compare == "20to25")[, "distance"])["75%"], digits = 2)`), sub-sampled to 10000 = `r round(median(filter(dist_data[["10000"]], cycle_compare == "20to25")[, "distance"]), digits = 2)` (`r round(quantile(filter(dist_data[["10000"]], cycle_compare == "20to25")[, "distance"])["25%"], digits = 2)` - `r round(quantile(filter(dist_data[["10000"]], cycle_compare == "20to25")[, "distance"])["75%"], digits = 2)`)) [Figure 3A]. Second, where data is available for the mock communities, there were larger difference between 20 and 25 cycles (sub-sampled to 1000 = `r round(median(filter(mock_dist_data[["1000"]], cycle_compare == "20to25")[, "distance"]), digits = 2)` (`r round(quantile(filter(mock_dist_data[["1000"]], cycle_compare == "20to25")[, "distance"])["25%"], digits = 2)` - `r round(quantile(filter(mock_dist_data[["1000"]], cycle_compare == "20to25")[, "distance"])["75%"], digits = 2)`)) [Figure 3B]. Third, using PERMANOVA to test for differences within HiFi DNA polymerase groups based on cycle number, only Phusion had cycle dependent differences at 1000 and 5000 sub-sampling depth (P-value = `r format(filter(bray_permanova_by_taq_results, sub_sample_level == 1000, taq == "PHU")[, "bh"], digits = 1, nsmall = 1)` and `r format(filter(bray_permanova_by_taq_results, sub_sample_level == 5000, taq == "PHU")[, "bh"], digits = 1, nsmall = 1)`, respectively). Phusion was one of only two HiFi DNA polymerases that that had fecal samples for the 1000 sub-sampling depth at 15 cycles. These stated differences between the next 5-cycle increment do not persist once 25 cycles are reached [Figure 3].


Next, we assessed whether there were any differences between 5-cycle increments within each individual or sample replicate. For both fecal and mock samples, we found that there was no detectable difference in Bray-Curtis index when comparing to the previous 5-cycle increment (P-value `r ifelse(min(bray_sim_to_prev_cycle_kruskal_results$bh, na.rm = T) > 0.05, invisible(">"), invisible("check data"))` 0.05). However, Phusion at 1000 sub-sampling depth had a P-value = `r format(filter(bray_sim_to_prev_cycle_kruskal_results, sub_sample_level == 1000, taq == "PHU")[, "pvalue"], digits = 1, nsmall = 1)` before multiple comparison correction. Similar to the PERMANOVA analysis, at higher sub-sampling depths these differences in Bray-Curtis indices disappear. Overall, HiFi DNA polymerase differences in Bray-Curtis index are dependent on both sub-sampling depth and cycle number.     


***Sequence Error is Dependent on both HiFi DNA Polymerase and Cycle Number:***
Differences by HiFi DNA polymerase in the median average per base error varied without a clear pattern across sub-sampling depth [Table S5]. The highest per base median average error rates were for the Kappa HiFi DNA polymerase [Figure 4]. This error rate was minimally affected by both the 'pre.cluster' step and chimera removal by VSEARCH [Figure 4]. The differences in the per base error rate between the various HiFi DNA polymerase was cycle dependent with Platinum having the largest differences versus other HiFi DNA polymerases [Figure 4B-C and Table S6]. The total sequences with at least one error had was also cycle number dependent and mostly alleviated by the use of the 'pre.cluster' step [Figure S1]. Differences in sequences with an error before this 'pre.cluster' step were due to large differences in Accuprime^TM^ and Platinum versus the other HiFi DNA polymerases [Figure S1 & Table S7 & S8]. Investigation of whether there were HiFi DNA polymerase dependent effects on base substitution found that there were no biases in the types of substitution made [Figure S2]. Although though sequence error is dependent on HiFi DNA polymerase it can be corrected using exsiting bioinformatic approaches. 


***Chimeric Sequences are HiFi DNA Polymerase Dependent and Correlate with Number of OTUs:***
At all levels of sub-sampling and cycle numbers there were significant differences in the chimera prevalence based on HiFi DNA polymerase used (P-value `r ifelse(max(filter(kruskal_chimera_prev, sub_sample_level > 500)[, "bh"]) < 0.05, invisible("<"), invisible("check data"))` 0.05) [Table S9]. Differences between Platinum and all other HiFi DNA polymerases accounted for the majority of these differences and was independent of cycle number and sub-sampling depth [Table S10]. Across sub-sampling depth and cycle number Accuprime^TM^ had the lowest chimera prevalence of all the HiFi DNA Polymerases regardless of whether 'pre.cluster' or chimera removal with VSEARCH was used [Figure 5]. 

For all HiFi DNA polymerases, a positive correlation was observed between chimeric sequences and number of OTUs, with this correlation being strongest for Accuprime, Platinum and Phusion HiFi DNA Polymerase [Figure 6]. The R^2^ value between the number of OTUs and chimeric sequences did not change with the use of 'pre.cluster' or with the removal of chimeras using VSEARCH [Figure 6]. Taken together, this data suggests that a strong correlation exists between the number of OTUs and the prevalence of chimeric sequences. 

\newpage

## Discussion

These observations build upon previous studies [@taq_Gohl_2016] by showing that different HiFi DNA polymerases have significant differences in the number of OTUs and that changes in total OTUs correlate with chimeras not removed after sequence processing [Figure 1-2 & 5]. HiFi DNA polymerase dependent differences in total number of OTUs are important to consider since many diversity metrics rely on this measure as part of their calculations. Our observations show that HiFi DNA polymerase can have a noticeable affect on the OTUs generated and these differences are consistent across sub-sampling depth and PCR cycle number [Figure 2-4]. Importantly, high biomass samples may underestimate the biases present within low biomass samples. An indication that this may be the case is the earlier detection of differences in total number of OTUs between HiFi DNA polymerases in the mock versus fecal samples.        

Although we did not observe strong differences in the Bray-Curtis index the data suggests that there may be differences between 15 and 20 cycles versus higher cycle numbers, such as 30 cycles, that are commonly used. There was few differences within individuals between corresponding 5-cycle increments (e.g. 15 to 20, 20 to 25, etc.). However, there is a clear trend that suggests that the community at 20 cycles is different then the community at 25 cycles [Figure 3]. These findings, in conjunction with the PERMANOVA results, suggest that cycle number can change bacterial community calculations but that these differences are minimal once 25 cycles are reached. Further, for some HiFI DNA polymerases, increasing the sub-sampling depth may reduce some of these observed community differences at lower cycle numbers.   

Increasing the cycle number also exacerbated chimera prevalence differences between the different HiFi DNA polymerases [Figure 5]. The chimera prevalence was strongly correlated with the number of OTUs which is relied upon heavily for many community metric calculations. However, Bray-Curtis analysis with PERMANOVA showed few differences based on HiFi DNA polymerase. It is possible that many of the increased number of OTUs are not highly abundant allowing the Bray-Curtis index to be able to successfully down-weight these respective OTUs [@bc_index_Minchin1987]. The choice of downstream diversity metric could be an important consideration in helping to mitigate these observed changes due to high chimera prevalence in specific HiFi DNA polymerases.  

Our observations suggest that there are clear HiFi DNA polymerase dependent differences in both per base error rate and chimera prevalence that cannot be removed using bioinformatic approaches [Figure 4 & 5]. Although it may be a natural assumption that the variation may be due to the DNA polymerase family, the highest chimera rate, from Platinum, was a family A polymerase while the lowest, from Accuprime, was also an A family polymerase [@polymerase_Ishino_2014]. In fact, from the material safety data sheet (MSDS), the differences between the two is not immediately apparent. Both Accuprime and Platinum contain a recombinant *Taq* DNA polymerase, a *Pyrococcus* spp GB-D polymerase and a platinum *Taq* antibody. It is possible that differences in how the recombinant *Taq* was generated could be the main reason for the differences in chimera rate.



```{r, echo=FALSE, message=FALSE, warning=FALSE}
#Results have an impact on downstream diversity measurements due to effects on total OTUs


#Need to cover that different types of polymerase may affect outcome
#- Accuprime HiFi from recombinant A-Family Taq DNA polymerase, Pyrococcus spp GB-D polymerase (proof-reading), platinum Taq Antibody (inhibits room temp activity)

#- Kappa HiFi from B-family DNA polymerase (proof-reading baked in), propietary antibody (inhibit room temp activity)

#- Q5 HiFi from B-family thermostable DNA polymerase with 3´→ 5´ exonuclease activity, fused to a processivity-enhancing Sso7d domain to support robust DNA amplification

#-Phusion HiFi B-family DNA polymerase a novel Pyrococcus-like enzyme fused with a processivity-enhancing domain, generate blunt-ended products.

#- Platinum HiFi recombinant A-Family Taq DNA polymerase, Pyrococcus species GB-D polymeras, Platinum Taq Antibody

```



\newpage

## Conclusion

Our findings show that the number of OTUs will be dependent on both HiFi DNA polymerase and cycle number chosen. Care should be taken when choosing a HiFi DNA polymerase for 16S rRNA gene surveys since intrinsic differences can change the number of OTUs observed and influence diversity based metrics that do not down-weight rare observations.

\newpage



## Acknowledgements

The authors would like to thank all the study participants in ERIN whose samples were utilized. We would also like to thank Judy Opp and April Cockburn for their effort in sequencing the samples as part of the Microbiome Core Facility at the University of Michigan. Salary support for Marc Sze came from the Canadian Institute of Health Research and the Michigan Institute for Clinical and Health Research Postdoctoral Translational Scholar Program.

\newpage

## References

<div id="refs"></div>

\newpage


**Figure 1: Normalized Fecal Number of OTUs.** The x-axis represents the different sub-sampling depths used and the y-axis is the normalized within individual number of OTUs. The red line represents the overall mean Z-score normalized number of OTUs for each respective HiFi DNA polymerase. The dashed black line represents the overall Z-score normalized mean number of OTUs. 

**Figure 2:  Mock Sample Variability in Number of OTUs based on HiFi DNA Polymerase.** A) Sub-sampled to 1000 reads. B) Sub-sampled to 5000 reads. C) Sub-sampled to 10000 reads. The dotted line represents the number of OTUs generated when the mock reference sequences are run through the pipeline. 

**Figure 3: Community Differences by Five-Cycle Intervals and Sub-sampling Depth.** A) Fecal samples within person difference based on the next 5-cycle PCR interval. B) Mock samples within replicate difference based on the next 5-cycle PCR interval.

**Figure 4: HiFi DNA Polymerase Per Base Error Rate in Mock Samples.** A) Error rate before the merger of sequences with pre.cluster and the removal of chimeras with VSEARCH. B) Error rate before the removal of chimeras with VSEARCH. C) Full pipeline. The error bars represent the 75% interquartile range of the median.  

**Figure 5: HiFi DNA Polymerase Chimera Prevalence in Mock Samples.** A) Chimera sequence percentage before the merger of sequences with pre.cluster and the removal of chimeras with VSEARCH. B) Chimera sequence percentage before the removal of chimeras with VSEARCH. C) Full pipeline. The error bars represent the 75% interquartile range of the median. 

**Figure 6: The Correlation between Number of OTUs and Chimeras.** A) Correlation before the merger of sequences with pre.cluster and the removal of chimeras with VSEARCH. B) Correlation before the removal of chimeras with VSEARCH. C) Correlation with full pipeline.  


\newpage


**Figure S1: HiFi DNA Polymerase Sequence Error Prevalence in Mock Samples.** A) Sequence error prevalence before the merger of sequences with pre.cluster and the removal of chimeras with VSEARCH. B) Sequence error prevalence before the removal of chimeras with VSEARCH. C) Full pipeline. The error bars represent the 75% interquartile range of the median. 

**Figure S2: HiFi DNA Polymerase Nucleotide Subsitutions in Mock Samples.**  


