\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{longtable,booktabs}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{}
  \pretitle{\vspace{\droptitle}}
  \posttitle{}
  \author{}
  \preauthor{}\postauthor{}
  \date{}
  \predate{}\postdate{}

\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\usepackage{amsmath}

\usepackage{mathtools}

\begin{document}

\section{Assessing the Differences in 16S rRNA Gene Sequencing Due to
High Fidelity DNA
Polymerase}\label{assessing-the-differences-in-16s-rrna-gene-sequencing-due-to-high-fidelity-dna-polymerase}

\vspace{10mm}

\begin{center}
Marc A Sze and Patrick D Schloss

\vspace{10mm}

Supplemental
\end{center}

\newpage

\textbf{Table S1: Wilcoxson Rank Sum Results of High versus Low GC
groups based on the median percent of the V4 hypervariable region or the
whole genome}

\begin{longtable}[]{@{}cccccc@{}}
\toprule
Number of Cycles & Polymerase & Rank Sum Statistic & P-value & BH & Test
Type\tabularnewline
\midrule
\endhead
30x & PL & 404.0 & 9.7e-05 & 2.4e-03 & V4 Region\tabularnewline
35x & PL & 56.0 & 5.8e-03 & 5.4e-02 & V4 Region\tabularnewline
20x & PHU & 59.0 & 8.5e-03 & 5.4e-02 & V4 Region\tabularnewline
30x & Q5 & 532.0 & 9.5e-03 & 5.4e-02 & V4 Region\tabularnewline
25x & PL & 61.0 & 1.1e-02 & 5.4e-02 & V4 Region\tabularnewline
30x & K & 546.0 & 1.5e-02 & 5.4e-02 & V4 Region\tabularnewline
25x & PHU & 64.0 & 1.5e-02 & 5.4e-02 & V4 Region\tabularnewline
20x & PL & 69.0 & 2.6e-02 & 8.1e-02 & V4 Region\tabularnewline
30x & PHU & 74.0 & 4.3e-02 & 1.1e-01 & V4 Region\tabularnewline
35x & PHU & 77.0 & 5.6e-02 & 1.3e-01 & V4 Region\tabularnewline
25x & ACC & 82.5 & 9.0e-02 & 1.9e-01 & V4 Region\tabularnewline
25x & K & 85.0 & 1.1e-01 & 1.9e-01 & V4 Region\tabularnewline
35x & ACC & 88.0 & 1.4e-01 & 2.3e-01 & V4 Region\tabularnewline
20x & ACC & 90.0 & 1.6e-01 & 2.5e-01 & V4 Region\tabularnewline
25x & Q5 & 96.0 & 2.4e-01 & 3.3e-01 & V4 Region\tabularnewline
20x & Q5 & 98.0 & 2.7e-01 & 3.6e-01 & V4 Region\tabularnewline
35x & Q5 & 99.0 & 2.9e-01 & 3.6e-01 & V4 Region\tabularnewline
35x & K & 102.5 & 3.5e-01 & 4.1e-01 & V4 Region\tabularnewline
20x & K & 108.0 & 4.6e-01 & 5.3e-01 & V4 Region\tabularnewline
30x & ACC & 139.0 & 7.0e-01 & 7.6e-01 & V4 Region\tabularnewline
30x & Q5 & 1340.0 & 4.1e-08 & 1.0e-06 & Whole Genome\tabularnewline
30x & K & 1354.5 & 9.8e-08 & 1.2e-06 & Whole Genome\tabularnewline
20x & PL & 249.0 & 1.5e-07 & 1.2e-06 & Whole Genome\tabularnewline
25x & PHU & 237.0 & 6.9e-06 & 2.9e-05 & Whole Genome\tabularnewline
25x & PL & 237.0 & 6.9e-06 & 2.9e-05 & Whole Genome\tabularnewline
25x & Q5 & 227.0 & 7.1e-05 & 2.5e-04 & Whole Genome\tabularnewline
20x & Q5 & 224.0 & 1.3e-04 & 4.0e-04 & Whole Genome\tabularnewline
20x & PHU & 220.0 & 2.7e-04 & 7.6e-04 & Whole Genome\tabularnewline
25x & ACC & 223.5 & 3.4e-04 & 7.8e-04 & Whole Genome\tabularnewline
35x & K & 223.5 & 3.4e-04 & 7.8e-04 & Whole Genome\tabularnewline
30x & ACC & 217.0 & 4.7e-04 & 9.7e-04 & Whole Genome\tabularnewline
25x & K & 210.0 & 2.1e-03 & 4.1e-03 & Whole Genome\tabularnewline
30x & PHU & 206.0 & 2.6e-03 & 4.4e-03 & Whole Genome\tabularnewline
20x & K & 201.0 & 6.3e-03 & 9.8e-03 & Whole Genome\tabularnewline
20x & ACC & 194.0 & 1.4e-02 & 2.0e-02 & Whole Genome\tabularnewline
35x & PHU & 186.0 & 2.9e-02 & 4.0e-02 & Whole Genome\tabularnewline
30x & PL & 961.0 & 1.2e-01 & 1.5e-01 & Whole Genome\tabularnewline
35x & Q5 & 168.0 & 1.4e-01 & 1.6e-01 & Whole Genome\tabularnewline
35x & ACC & 152.0 & 3.8e-01 & 4.1e-01 & Whole Genome\tabularnewline
35x & PL & 115.0 & 6.4e-01 & 6.7e-01 & Whole Genome\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S2: Kruskal-Wallis results of differences in error rate,
sequences with an error, and chimera prevalence between polymerases in
mock samples}

\begin{longtable}[]{@{}cccccc@{}}
\toprule
Rank Sum Statistic & DF & P-value & BH & Cycle & Test
Type\tabularnewline
\midrule
\endhead
7.85 & 2 & 2.0e-02 & 2.0e-02 & 20x & Error Rate\tabularnewline
12.95 & 4 & 1.2e-02 & 1.5e-02 & 25x & Error Rate\tabularnewline
28.31 & 4 & 1.1e-05 & 4.3e-05 & 30x & Error Rate\tabularnewline
15.00 & 4 & 4.7e-03 & 9.4e-03 & 35x & Error Rate\tabularnewline
7.85 & 2 & 2.0e-02 & 2.0e-02 & 20x & Sequences with Error\tabularnewline
15.98 & 4 & 3.0e-03 & 5.2e-03 & 25x & Sequences with
Error\tabularnewline
29.22 & 4 & 7.0e-06 & 2.8e-05 & 30x & Sequences with
Error\tabularnewline
15.44 & 4 & 3.9e-03 & 5.2e-03 & 35x & Sequences with
Error\tabularnewline
8.91 & 2 & 1.2e-02 & 1.2e-02 & 20x & Chimera Prevalence\tabularnewline
15.54 & 4 & 3.7e-03 & 4.9e-03 & 25x & Chimera Prevalence\tabularnewline
21.41 & 4 & 2.6e-04 & 1.1e-03 & 30x & Chimera Prevalence\tabularnewline
16.28 & 4 & 2.7e-03 & 4.9e-03 & 35x & Chimera Prevalence\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S3: Dunn's Post-Hoc results of differences in error rate,
sequences with an error, and chimera prevalence between polymerases in
mock samples}

\begin{longtable}[]{@{}cccccc@{}}
\toprule
Chi Squared & Z Statistic & P Adjusted & Comparison & Cycle & Test
Type\tabularnewline
\midrule
\endhead
7.85 & 2.77 & 2.8e-03 & PHU - PL & 20x & Error Rate\tabularnewline
7.85 & -1.64 & 5.0e-02 & PL - Q5 & 20x & Error Rate\tabularnewline
12.95 & 3.26 & 5.6e-04 & K - Q5 & 25x & Error Rate\tabularnewline
12.95 & 2.68 & 3.7e-03 & K - PHU & 25x & Error Rate\tabularnewline
12.95 & 2.01 & 2.2e-02 & ACC - Q5 & 25x & Error Rate\tabularnewline
12.95 & 1.88 & 3.0e-02 & PL - Q5 & 25x & Error Rate\tabularnewline
28.31 & -4.46 & 4.1e-06 & ACC - K & 30x & Error Rate\tabularnewline
28.31 & 4.32 & 7.7e-06 & K - PL & 30x & Error Rate\tabularnewline
28.31 & -2.69 & 3.6e-03 & ACC - Q5 & 30x & Error Rate\tabularnewline
28.31 & 2.65 & 4.0e-03 & K - PHU & 30x & Error Rate\tabularnewline
28.31 & 2.37 & 8.8e-03 & K - Q5 & 30x & Error Rate\tabularnewline
28.31 & -2.00 & 2.3e-02 & PL - Q5 & 30x & Error Rate\tabularnewline
15.00 & 3.14 & 8.6e-04 & K - PHU & 35x & Error Rate\tabularnewline
15.00 & -3.05 & 1.2e-03 & PHU - PL & 35x & Error Rate\tabularnewline
15.00 & -2.33 & 1.0e-02 & ACC - K & 35x & Error Rate\tabularnewline
15.00 & -2.05 & 2.0e-02 & ACC - PL & 35x & Error Rate\tabularnewline
15.00 & -1.79 & 3.7e-02 & PHU - Q5 & 35x & Error Rate\tabularnewline
15.00 & 1.68 & 4.7e-02 & K - Q5 & 35x & Error Rate\tabularnewline
7.85 & 2.77 & 2.8e-03 & PHU - PL & 20x & Sequences with
Error\tabularnewline
7.85 & -1.64 & 5.0e-02 & PL - Q5 & 20x & Sequences with
Error\tabularnewline
15.98 & 3.37 & 3.7e-04 & K - Q5 & 25x & Sequences with
Error\tabularnewline
15.98 & 2.79 & 2.6e-03 & K - PHU & 25x & Sequences with
Error\tabularnewline
15.98 & 2.76 & 2.9e-03 & ACC - Q5 & 25x & Sequences with
Error\tabularnewline
15.98 & 2.14 & 1.6e-02 & ACC - PHU & 25x & Sequences with
Error\tabularnewline
15.98 & 1.86 & 3.1e-02 & K - PL & 25x & Sequences with
Error\tabularnewline
29.22 & 4.17 & 1.5e-05 & K - Q5 & 30x & Sequences with
Error\tabularnewline
29.22 & 3.89 & 5.0e-05 & K - PHU & 30x & Sequences with
Error\tabularnewline
29.22 & 3.57 & 1.8e-04 & PL - Q5 & 30x & Sequences with
Error\tabularnewline
29.22 & -3.41 & 3.2e-04 & PHU - PL & 30x & Sequences with
Error\tabularnewline
29.22 & -1.81 & 3.5e-02 & ACC - K & 30x & Sequences with
Error\tabularnewline
29.22 & 1.76 & 3.9e-02 & ACC - PHU & 30x & Sequences with
Error\tabularnewline
15.44 & -3.71 & 1.0e-04 & PHU - PL & 35x & Sequences with
Error\tabularnewline
15.44 & 2.38 & 8.6e-03 & PL - Q5 & 35x & Sequences with
Error\tabularnewline
15.44 & 2.38 & 8.7e-03 & K - PHU & 35x & Sequences with
Error\tabularnewline
15.44 & 1.85 & 3.2e-02 & ACC - PHU & 35x & Sequences with
Error\tabularnewline
15.44 & -1.85 & 3.2e-02 & ACC - PL & 35x & Sequences with
Error\tabularnewline
8.91 & 2.98 & 1.4e-03 & PHU - PL & 20x & Chimera
Prevalence\tabularnewline
15.54 & 3.37 & 3.7e-04 & K - PHU & 25x & Chimera
Prevalence\tabularnewline
15.54 & -2.76 & 2.9e-03 & PHU - PL & 25x & Chimera
Prevalence\tabularnewline
15.54 & -2.68 & 3.7e-03 & ACC - K & 25x & Chimera
Prevalence\tabularnewline
15.54 & -2.01 & 2.2e-02 & ACC - PL & 25x & Chimera
Prevalence\tabularnewline
15.54 & 1.98 & 2.4e-02 & K - Q5 & 25x & Chimera
Prevalence\tabularnewline
21.41 & -4.23 & 1.2e-05 & ACC - PL & 30x & Chimera
Prevalence\tabularnewline
21.41 & 2.89 & 1.9e-03 & PL - Q5 & 30x & Chimera
Prevalence\tabularnewline
21.41 & -2.84 & 2.2e-03 & ACC - K & 30x & Chimera
Prevalence\tabularnewline
21.41 & -2.59 & 4.8e-03 & PHU - PL & 30x & Chimera
Prevalence\tabularnewline
21.41 & -2.05 & 2.0e-02 & ACC - Q5 & 30x & Chimera
Prevalence\tabularnewline
21.41 & -1.73 & 4.2e-02 & K - PL & 30x & Chimera
Prevalence\tabularnewline
16.28 & -3.71 & 1.0e-04 & ACC - PL & 35x & Chimera
Prevalence\tabularnewline
16.28 & -2.65 & 4.0e-03 & PHU - PL & 35x & Chimera
Prevalence\tabularnewline
16.28 & -2.38 & 8.7e-03 & ACC - K & 35x & Chimera
Prevalence\tabularnewline
16.28 & -2.12 & 1.7e-02 & ACC - Q5 & 35x & Chimera
Prevalence\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S4: ANOVA results of differences in number of OTUs between
polymerases in fecal and mock samples}

\begin{longtable}[]{@{}cccccccc@{}}
\toprule
DF & Sum Squares & Mean Squares & F value & P-value & BH & Cycle &
Sample Type\tabularnewline
\midrule
\endhead
4 & 3.66 & 0.91 & 1.25 & 3.5e-01 & 3.5e-01 & 20x & Fecal
Sample\tabularnewline
4 & 6.90 & 1.73 & 2.84 & 6.1e-02 & 1.2e-01 & 25x & Fecal
Sample\tabularnewline
3 & 4.67 & 1.56 & 2.55 & 1.0e-01 & 1.4e-01 & 30x & Fecal
Sample\tabularnewline
4 & 13.01 & 3.25 & 16.35 & 2.4e-05 & 9.7e-05 & 35x & Fecal
Sample\tabularnewline
2 & 364.56 & 182.28 & 15.82 & 1.7e-03 & 2.2e-03 & 20x & Mock
Sample\tabularnewline
4 & 3478.53 & 869.63 & 75.15 & 2.7e-09 & 1.1e-08 & 25x & Mock
Sample\tabularnewline
4 & 2206.58 & 551.64 & 10.28 & 1.8e-05 & 3.6e-05 & 30x & Mock
Sample\tabularnewline
4 & 1971.92 & 492.98 & 2.31 & 1.0e-01 & 1.0e-01 & 35x & Mock
Sample\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S5: Tukey post-hoc results of differences in number of
OTUs between polymerases in fecal and mock samples}

\begin{longtable}[]{@{}ccccccc@{}}
\toprule
Difference & Lower & Upper & P Adjusted & Comparison & Cycle & Sample
Type\tabularnewline
\midrule
\endhead
25.00 & -17.32 & 67.32 & 0.40 & PL-ACC & 35x & Fecal
Sample\tabularnewline
24.25 & -18.07 & 66.57 & 0.43 & PL-K & 35x & Fecal Sample\tabularnewline
20.50 & -21.82 & 62.82 & 0.58 & Q5-ACC & 35x & Fecal
Sample\tabularnewline
19.75 & -22.57 & 62.07 & 0.61 & Q5-K & 35x & Fecal Sample\tabularnewline
13.50 & -28.82 & 55.82 & 0.86 & PHU-ACC & 35x & Fecal
Sample\tabularnewline
12.75 & -29.57 & 55.07 & 0.88 & PHU-K & 35x & Fecal
Sample\tabularnewline
11.50 & -30.82 & 53.82 & 0.91 & PL-PHU & 35x & Fecal
Sample\tabularnewline
7.00 & -35.32 & 49.32 & 0.98 & Q5-PHU & 35x & Fecal
Sample\tabularnewline
-4.50 & -46.82 & 37.82 & 1.00 & Q5-PL & 35x & Fecal
Sample\tabularnewline
0.75 & -41.57 & 43.07 & 1.00 & K-ACC & 35x & Fecal Sample\tabularnewline
-13.50 & -20.36 & -6.64 & 1.3e-03 & PL-PHU & 20x & Mock
Sample\tabularnewline
-38.50 & -46.60 & -30.40 & 5.1e-09 & Q5-K & 25x & Mock
Sample\tabularnewline
-37.50 & -45.60 & -29.40 & 7.3e-09 & PHU-K & 25x & Mock
Sample\tabularnewline
36.50 & 28.40 & 44.60 & 1.1e-08 & K-ACC & 25x & Mock
Sample\tabularnewline
-35.50 & -43.60 & -27.40 & 1.5e-08 & PL-K & 25x & Mock
Sample\tabularnewline
22.03 & 9.31 & 34.74 & 1.8e-04 & K-ACC & 30x & Mock
Sample\tabularnewline
-15.48 & -25.20 & -5.75 & 5.7e-04 & Q5-K & 30x & Mock
Sample\tabularnewline
-18.78 & -31.49 & -6.06 & 1.5e-03 & PHU-K & 30x & Mock
Sample\tabularnewline
15.55 & 3.03 & 28.07 & 9.0e-03 & PL-ACC & 30x & Mock
Sample\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S6: Kruskal-Wallis results of significant differences in
OTU in mock community samples}

\begin{longtable}[]{@{}ccccccc@{}}
\toprule
Cycles & OTU & Lowest Taxa ID & Statistic & P-value & BH & Sample
Type\tabularnewline
\midrule
\endhead
25x & Otu0033 & Clostridiales & 18.00000 & 1.2e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0039 & Phascolarctobacterium & 18.00000 & 1.2e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0024 & Dorea & 17.88235 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0026 & Roseburia & 17.88235 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0031 & Erysipelotrichaceae & 17.88235 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0068 & Blautia & 17.88235 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0006 & Roseburia & 17.84348 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0009 & Ruminococcaceae & 17.84348 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0012 & Blautia & 17.84348 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
25x & Otu0030 & Enterococcus & 17.84348 & 1.3e-03 & 1.6e-02 &
mock\tabularnewline
30x & Otu0185 & Streptococcus & 30.54780 & 3.8e-06 & 1.1e-03 &
mock\tabularnewline
30x & Otu0020 & Pseudomonas & 28.11429 & 1.2e-05 & 1.7e-03 &
mock\tabularnewline
30x & Otu0023 & Bacteroides & 26.88809 & 2.1e-05 & 2.0e-03 &
mock\tabularnewline
30x & Otu0007 & Escherichia/Shigella & 23.06969 & 1.2e-04 & 8.6e-03 &
mock\tabularnewline
30x & Otu0011 & Bacillales & 21.61181 & 2.4e-04 & 1.3e-02 &
mock\tabularnewline
30x & Otu0017 & Lactobacillus & 21.11598 & 3.0e-04 & 1.4e-02 &
mock\tabularnewline
30x & Otu0004 & Staphylococcus & 20.10917 & 4.8e-04 & 1.6e-02 &
mock\tabularnewline
30x & Otu0001 & Salmonella & 19.96365 & 5.1e-04 & 1.6e-02 &
mock\tabularnewline
30x & Otu0162 & Bacillales & 19.74476 & 5.6e-04 & 1.6e-02 &
mock\tabularnewline
30x & Otu0029 & Bacteroides & 19.02599 & 7.8e-04 & 1.6e-02 &
mock\tabularnewline
30x & Otu0030 & Enterococcus & 18.17396 & 1.1e-03 & 1.6e-02 &
mock\tabularnewline
30x & Otu0056 & Bacilli & 18.09753 & 1.2e-03 & 1.6e-02 &
mock\tabularnewline
30x & Otu0083 & Alistipes & 16.91062 & 2.0e-03 & 2.3e-02 &
mock\tabularnewline
35x & Otu0031 & Erysipelotrichaceae & 19.95000 & 5.1e-04 & 1.6e-02 &
mock\tabularnewline
\bottomrule
\end{longtable}

\newpage

\textbf{Table S7: Significant Dunns post-hoc results of differences
between polymerases for specific taxa}

\footnotesize

\begin{longtable}[]{@{}ccccccccc@{}}
\toprule
Chi Sq & Z & P-value & BH & Comparison & Cycles & OTU & Taxa & Part of
Mock\tabularnewline
\midrule
\endhead
2.0e+01 & -4.2e+00 & 1.1e-05 & 1.1e-04 & ACC - PL & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.3e+01 & 4.0e+00 & 3.8e-05 & 3.8e-04 & PL - Q5 & 30x & Otu0007 &
Escherichia/Shigella & yes\tabularnewline
2.3e+01 & -3.7e+00 & 9.9e-05 & 5.0e-04 & K - PL & 30x & Otu0007 &
Escherichia/Shigella & yes\tabularnewline
1.8e+01 & -3.5e+00 & 2.5e-04 & 6.3e-04 & ACC - K & 25x & Otu0030 &
Enterococcus & yes\tabularnewline
2.3e+01 & -3.5e+00 & 2.0e-04 & 6.8e-04 & ACC - PL & 30x & Otu0007 &
Escherichia/Shigella & yes\tabularnewline
1.8e+01 & 3.5e+00 & 2.5e-04 & 8.4e-04 & K - PHU & 25x & Otu0030 &
Enterococcus & yes\tabularnewline
1.8e+01 & 3.7e+00 & 1.1e-04 & 1.1e-03 & K - Q5 & 30x & Otu0030 &
Enterococcus & yes\tabularnewline
1.8e+01 & 3.5e+00 & 2.5e-04 & 1.3e-03 & K - PL & 25x & Otu0030 &
Enterococcus & yes\tabularnewline
2.0e+01 & 3.6e+00 & 1.4e-04 & 1.4e-03 & ACC - PL & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
2.0e+01 & 3.4e+00 & 3.8e-04 & 1.9e-03 & K - PL & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
1.8e+01 & 3.5e+00 & 2.5e-04 & 2.5e-03 & K - Q5 & 25x & Otu0030 &
Enterococcus & yes\tabularnewline
1.8e+01 & 3.2e+00 & 6.6e-04 & 3.3e-03 & K - PL & 30x & Otu0030 &
Enterococcus & yes\tabularnewline
1.8e+01 & -2.8e+00 & 2.3e-03 & 5.9e-03 & ACC - K & 30x & Otu0030 &
Enterococcus & yes\tabularnewline
1.8e+01 & 2.8e+00 & 2.3e-03 & 7.8e-03 & K - PHU & 30x & Otu0030 &
Enterococcus & yes\tabularnewline
2.0e+01 & 2.7e+00 & 3.9e-03 & 1.3e-02 & ACC - PHU & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
2.0e+01 & 2.5e+00 & 5.4e-03 & 1.4e-02 & ACC - Q5 & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
2.0e+01 & -2.7e+00 & 3.0e-03 & 1.5e-02 & K - PL & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.0e+01 & -2.5e+00 & 5.4e-03 & 1.8e-02 & ACC - Q5 & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.0e+01 & 2.2e+00 & 1.2e-02 & 3.1e-02 & PL - Q5 & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.0e+01 & -2.1e+00 & 1.9e-02 & 3.2e-02 & ACC - K & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.0e+01 & 2.1e+00 & 1.7e-02 & 3.4e-02 & K - PHU & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
2.0e+01 & -2.1e+00 & 1.7e-02 & 3.5e-02 & PHU - PL & 30x & Otu0001 &
Salmonella & yes\tabularnewline
2.0e+01 & 2.0e+00 & 2.5e-02 & 4.2e-02 & K - Q5 & 30x & Otu0004 &
Staphylococcus & yes\tabularnewline
\bottomrule
\end{longtable}

\normalsize

\newpage

\textbf{Table S8: Dunns post-hoc results of differences in model
classification success using 16S rRNA gene sequencing data}

\begin{longtable}[]{@{}cccccccc@{}}
\toprule
Chi Sq & Z & P-value & BH & Sample Type 1 & Model 1 & Sample Type 2 &
Model 2\tabularnewline
\midrule
\endhead
4.8e+02 & 4.9e+00 & 4.9e-07 & 5.5e-07 & fecal & cycles & fecal &
polymerase\tabularnewline
4.8e+02 & -1.5e+01 & 4.2e-49 & 2.1e-48 & fecal & cycles & fecal &
individual\tabularnewline
4.8e+02 & -2.0e+01 & 1.2e-85 & 1.2e-84 & fecal & polymerase & fecal &
individual\tabularnewline
4.8e+02 & -9.8e+00 & 6.5e-23 & 1.1e-22 & fecal & cycles & mock &
cycles\tabularnewline
4.8e+02 & -1.5e+01 & 4.3e-49 & 1.4e-48 & fecal & polymerase & mock &
cycles\tabularnewline
4.8e+02 & 4.9e+00 & 4.9e-07 & 6.1e-07 & fecal & individual & mock &
cycles\tabularnewline
4.8e+02 & -4.9e+00 & 4.9e-07 & 7.0e-07 & fecal & cycles & mock &
polymerase\tabularnewline
4.8e+02 & -9.8e+00 & 6.2e-23 & 1.5e-22 & fecal & polymerase & mock &
polymerase\tabularnewline
4.8e+02 & 9.8e+00 & 6.5e-23 & 1.3e-22 & fecal & individual & mock &
polymerase\tabularnewline
4.8e+02 & 4.9e+00 & 5.1e-07 & 5.1e-07 & mock & cycles & mock &
polymerase\tabularnewline
\bottomrule
\end{longtable}

\newpage


\end{document}
