---
title: ''
csl: peerj.csl
fontsize: 11pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=1.0in
bibliography: null
---

# Assessing the Differences in 16S rRNA Gene Sequencing Due to High Fidelity DNA Polymerase

\vspace{10mm}

\begin{center}
Marc A Sze and Patrick D Schloss

\vspace{10mm}

Supplemental
\end{center}


\newpage

```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("tidyverse", "scales", "knitr", "rmarkdown"))

```



**Table S1: Wilcoxson Rank Sum Results of High versus Low GC groups based on the median percent of the V4 hypervariable region or the whole genome**

```{r supplemental_table1, echo=FALSE, message=FALSE, warning=FALSE}

mock_gc_wlcox <- as.data.frame(read_csv("../data/process/tables/v4_gc_wilcox_test_summary.csv")) %>% 
  arrange(p.value) %>% 
  mutate_at(vars(statistic), funs(round(., digits = 2))) %>% 
  mutate_at(vars(p.value:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
  mutate(test_type = "V4 Region") %>% 
  bind_rows(
    as.data.frame(
    read_csv("../data/process/tables/whole_genome_gc_wilcox_test_summary.csv")) %>% 
    arrange(p.value) %>% 
    mutate_at(vars(statistic), funs(round(., digits = 2))) %>% 
    mutate_at(vars(p.value:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    mutate(test_type = "Whole Genome")) %>% 
  select(cycles, taq, statistic, p.value, bh, test_type) %>% 
  filter(cycles != "fifteen") %>% 
  mutate(cycles = case_when(
    cycles == "twenty" ~ "20x", 
    cycles == "twenty_five" ~ "25x", 
    cycles == "thirty" ~ "30x", 
    cycles == "thirty_five" ~ "35x", 
    TRUE ~ "uh oh"))

colnames(mock_gc_wlcox) <- c("Number of Cycles", "Polymerase", "Rank Sum Statistic", "P-value", "BH", "Test Type")

kable(mock_gc_wlcox, format = "markdown", align = 'c', padding = 2)
```



\newpage

**Table S2: Kruskal-Wallis results of differences in error rate, sequences with an error, and chimera prevalence between polymerases in mock samples**

```{r supplemental_table2, echo=FALSE, message=FALSE, warning=FALSE}

mock_error_rate_kruskal <- as.data.frame(read_csv("../data/process/tables/mock_error_overall_kruskal_results.csv")) %>% 
  mutate_at(vars(statistic), funs(round(., digits = 2))) %>% 
  mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
  select(-sub_sample_level) %>% 
  mutate(test_type = "Error Rate") %>% 
  bind_rows(
    as.data.frame(
    read_csv("../data/process/tables/mock_error_count_overall_kruskal_results.csv")) %>% 
    mutate_at(vars(statistic), funs(round(., digits = 2))) %>% 
    mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    select(-sub_sample_level) %>% 
    mutate(test_type = "Sequences with Error")) %>% 
  bind_rows(
    as.data.frame(
    read_csv("../data/process/tables/mock_chimera_overall_kruskal_results.csv")) %>% 
    mutate_at(vars(statistic), funs(round(., digits = 2))) %>% 
    mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    select(-sub_sample_level) %>% 
    mutate(test_type = "Chimera Prevalence"))

colnames(mock_error_rate_kruskal) <- c("Rank Sum Statistic", "DF", "P-value", "BH", "Cycle", "Test Type")

kable(mock_error_rate_kruskal, format = "markdown", align = 'c', padding = 2)
```



\newpage


**Table S3: Dunn's Post-Hoc results of differences in error rate, sequences with an error, and chimera prevalence between polymerases in mock samples**

```{r supplemental_table3, echo=FALSE, message=FALSE, warning=FALSE}

mock_error_rate_dunn <- as.data.frame(read_csv("../data/process/tables/mock_error_overall_dunn_results.csv")) %>% 
  mutate_at(vars(chi2:Z), funs(round(., digits = 2))) %>% 
  filter(pvalue < 0.05) %>% 
  arrange(pvalue) %>% 
  mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
  select(-bh, -sub_sample_level) %>% 
  arrange(cycle) %>% 
  mutate(test_type = "Error Rate") %>% 
  bind_rows(
    as.data.frame(read_csv("../data/process/tables/mock_error_count_overall_dunn_results.csv")) %>% 
    mutate_at(vars(chi2:Z), funs(round(., digits = 2))) %>% 
    filter(pvalue < 0.05) %>% 
    arrange(pvalue) %>% 
    mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    select(-bh, -sub_sample_level) %>% 
    arrange(cycle) %>% 
    mutate(test_type = "Sequences with Error")) %>% 
  bind_rows(
    as.data.frame(read_csv("../data/process/tables/mock_chimera_overall_dunn_results.csv")) %>% 
    mutate_at(vars(chi2:Z), funs(round(., digits = 2))) %>% 
    filter(pvalue < 0.05) %>% 
    arrange(pvalue) %>% 
    mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    select(-bh, -sub_sample_level) %>% 
    arrange(cycle) %>% 
    mutate(test_type = "Chimera Prevalence"))
  

colnames(mock_error_rate_dunn) <- c("Chi Squared", "Z Statistic", "P Adjusted", "Comparison", "Cycle", "Test Type")

kable(mock_error_rate_dunn, format = "markdown", align = 'c', padding = 2)
```

\newpage


**Table S4: ANOVA results of differences in number of OTUs between polymerases in fecal and mock samples**

```{r supplemental_table4, echo=FALSE, message=FALSE, warning=FALSE}

fs_anova <- as.data.frame(read_csv("../data/process/tables/fecal_overall_anova_results.csv")) %>% 
  filter(sub_sample_level == 1000) %>% 
  select(-sub_sample_level) %>% 
  mutate_at(vars(Sum.Sq:F.value), funs(round(., digits = 2))) %>% 
  mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
  mutate(sample_type = "Fecal Sample") %>% 
  bind_rows(
    as.data.frame(read_csv("../data/process/tables/mock_overall_anova_results.csv")) %>% 
    mutate_at(vars(Sum.Sq:F.value), funs(round(., digits = 2))) %>% 
    mutate_at(vars(pvalue:bh), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    filter(sub_sample_level == 1000) %>% 
    select(-sub_sample_level) %>% 
    mutate(sample_type = "Mock Sample"))
  
colnames(fs_anova) <- c("DF", "Sum Squares", "Mean Squares", "F value", "P-value", "BH", "Cycle", "Sample Type")

kable(fs_anova, format = "markdown", align = 'c', padding = 2)
```


\newpage


**Table S5: Tukey post-hoc results of differences in number of OTUs between polymerases in fecal and mock samples**

```{r supplemental_table5, echo=FALSE, message=FALSE, warning=FALSE}

fs_tukey <- as.data.frame(read_csv("../data/process/tables/fecal_overall_tukey_results.csv")) %>% 
  mutate_at(vars(diff:upr), funs(round(., digits = 2))) %>% 
  filter(sub_sample_level == 1000) %>% 
  select(-sub_sample_level) %>% 
  rename(padj = `p adj`) %>% 
  mutate(padj = as.numeric(padj)) %>% 
  mutate_at(vars(padj), funs(format(., digits = 2, nsmall = 2))) %>% 
  mutate(sample_type = "Fecal Sample") %>% 
  bind_rows(
    as.data.frame(read_csv("../data/process/tables/mock_overall_tukey_results.csv")) %>% 
    mutate_at(vars(diff:upr), funs(round(., digits = 2))) %>% 
    filter(sub_sample_level == 1000) %>% 
    select(-sub_sample_level) %>% 
    rename(padj = `p adj`) %>% 
    mutate(padj = as.numeric(padj)) %>% 
    filter(padj < 0.05) %>% 
    arrange(padj) %>% 
    mutate_at(vars(padj), funs(format(., scientific = TRUE, digits = 2, nsmall = 2))) %>% 
    arrange(cycle) %>% 
    mutate(sample_type = "Mock Sample"))

colnames(fs_tukey) <- c("Difference", "Lower", "Upper", "P Adjusted", "Comparison", "Cycle", "Sample Type")

kable(fs_tukey, format = "markdown", align = 'c', padding = 2)

```


\newpage












